{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import pandas as pd\n",
    "import csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "trainV = pd.read_csv(\"training_variants.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Variation</th>\n",
       "      <th>Class</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>FAM58A</td>\n",
       "      <td>Truncating Mutations</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>CBL</td>\n",
       "      <td>W802*</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>CBL</td>\n",
       "      <td>Q249E</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>CBL</td>\n",
       "      <td>N454D</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>CBL</td>\n",
       "      <td>L399V</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   ID    Gene             Variation  Class\n",
       "0   0  FAM58A  Truncating Mutations      1\n",
       "1   1     CBL                 W802*      2\n",
       "2   2     CBL                 Q249E      2\n",
       "3   3     CBL                 N454D      3\n",
       "4   4     CBL                 L399V      4"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "trainV.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(3321, 4)"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "trainV.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/TheResearcher/Desktop/anaconda/lib/python2.7/site-packages/ipykernel_launcher.py:1: ParserWarning: Falling back to the 'python' engine because the 'c' engine does not support regex separators (separators > 1 char and different from '\\s+' are interpreted as regex); you can avoid this warning by specifying engine='python'.\n",
      "  \"\"\"Entry point for launching an IPython kernel.\n"
     ]
    }
   ],
   "source": [
    "trainT = pd.read_csv('training_text',sep=\"||\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ID\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'Text\\n'"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "fileHandle = open('training_text', 'r')\n",
    "\n",
    "line = fileHandle.next()\n",
    "fields = line.split(',')\n",
    "print fields[0]\n",
    "fields[1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "for line in fileHandle:\n",
    "    print line\n",
    "    fields = line.split('||')\n",
    "\n",
    "    #print fields[0] # prints the first fields value\n",
    "    #print fields[1] # prints the second fields value\n",
    "\n",
    "fileHandle.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>ID,Text</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0||Cyclin-dependent kinases (CDKs) regulate a ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1|| Abstract Background  Non-small cell lung c...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2|| Abstract Background  Non-small cell lung c...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3||Recent evidence has demonstrated that acqui...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4||Oncogenic mutations in the monomeric Casita...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>5||Oncogenic mutations in the monomeric Casita...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>6||Oncogenic mutations in the monomeric Casita...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>7||CBL is a negative regulator of activated re...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>8|| Abstract Juvenile myelomonocytic leukemia ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>9|| Abstract Juvenile myelomonocytic leukemia ...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                             ID,Text\n",
       "0  0||Cyclin-dependent kinases (CDKs) regulate a ...\n",
       "1  1|| Abstract Background  Non-small cell lung c...\n",
       "2  2|| Abstract Background  Non-small cell lung c...\n",
       "3  3||Recent evidence has demonstrated that acqui...\n",
       "4  4||Oncogenic mutations in the monomeric Casita...\n",
       "5  5||Oncogenic mutations in the monomeric Casita...\n",
       "6  6||Oncogenic mutations in the monomeric Casita...\n",
       "7  7||CBL is a negative regulator of activated re...\n",
       "8  8|| Abstract Juvenile myelomonocytic leukemia ...\n",
       "9  9|| Abstract Juvenile myelomonocytic leukemia ..."
      ]
     },
     "execution_count": 39,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "trainT.head(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {},
   "outputs": [],
   "source": [
    "cols = trainT.columns.values\n",
    "newcols = cols[0].split(',')\n",
    "data = {}\n",
    "for i in range(trainT.shape[0]):\n",
    "    l = trainT.iloc[i]['ID,Text'].split('||')\n",
    "    data[l[0]] = l[1]\n",
    "    l = []\n",
    "\n",
    "#del trainT\n",
    "#test = pd.DataFrame(data=data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "344\n",
      "E-cadherin is involved in the formation of cell-junctions and the maintenance of epithelial integrity. Direct evidence of E-cadherin mutations triggering tumorigenesis has come from the finding of inactivating germline mutations of the gene (CDH1) in hereditary diffuse gastric cancer (HDGC). We screened a series of 66 young gastric cancer probands for germline CDH1 mutations, and two novel missense alterations together with an intronic variant were identified. We then analysed the functional significance of the two exonic missense variants found here as well as a third germline missense variant that we previously identified in a HGDC family. cDNAs encoding either the wild-type protein or mutant forms of E-cadherin were stably transfected into CHO (Chinese hamster ovary) E-cadherin-negative cells. Transfected cell-lines were characterized in terms of aggregation, motility and invasion. We show that a proportion of apparently sporadic early-onset diffuse gastric carcinomas are associated with germline alterations of the E-cadherin gene. We also demonstrate that a proportion of missense variants are associated with significant functional consequences, suggesting that our cell model can be used as an adjunct in deciding on the potential pathogenic role of identified E-cadherin germline alterations.\n",
      "345\n",
      "Nonsyndromic orofacial cleft (NSOFC) is a complex disease of still unclear genetic etiology. To investigate the contribution of rare epithelial cadherin (CDH1) gene variants to NSOFC, we target sequenced 221 probands. Candidate variants were evaluated via in vitro, in silico, or segregation analyses. Three probably pathogenic variants (c.760G>A [p.Asp254Asn], c.1023T>G [p.Tyr341*], and c.2351G>A [p.Arg784His]) segregated according to autosomal dominant inheritance in four nonsyndromic cleft lip with or without cleft palate (NSCL/P) families (Lod score: 5.8 at θ = 0; 47% penetrance). A fourth possibly pathogenic variant (c.387+5G>A) was also found, but further functional analyses are needed (overall prevalence of CDH1 candidate variants: 2%; 15.4% among familial cases). CDH1 mutational burden was higher among probands from familial cases when compared to that of controls (P = 0.002). We concluded that CDH1 contributes to NSCL/P with mainly rare, moderately penetrant variants, and CDH1 haploinsufficiency is the likely etiological mechanism. Nonsyndromic cleft lip with or without cleft palate (NSCL/P) and nonsyndromic cleft palate only (NSCPO) are two complex disorders within the nonsyndromic orofacial cleft (NSOFC) spectrum [Gorlin et al., 2001]. While the genetic etiology of NSCPO is largely unclear, genetic loci have been systematically implicated in NSCL/P, such as common low-risk 8q24, 10q25, and IRF6 variants [Rahimov et al., 2008; Birnbaum et al., 2009; Mangold et al., 2010, Brito et al., 2012aa, 2012b]. However, given the high heritability attributed to NSCL/P [Hu et al., 1982; Calzolari et al., 1988; Brito et al., 2011], searching for alternative genetic variants or mechanisms is necessary to bridge the missing heritability gap of these malformations.  Rare germline variants in the gene encoding the adhesion molecule epithelial cadherin, CDH1 (MIM# 192090), have long been associated with diffuse gastric cancer and lobular breast cancer [van Roy and Berx, 2008]. Most recently, CDH1 mutations have been reported in OFC patients in association with gastric cancer [Frebourg et al., 2006; Kluijt, et al. 2012; Benusiglio et al., 2013] or not [Vogelaar et al., 2013; Bureau et al., 2014]. These findings raise the questions as to what the proportion of NSOFC cases underlain by CDH1 variants and their attributed penetrance is, and which types of mutations or mechanisms lead to OFC, cancer, or both phenotypes.  Here, we performed a variant screening for CDH1 (NM_004360.3) coding region in 221 NSOFC probands (affected by NSCL/P [n = 189] or NSCPO [n = 32], either from nonfamilial [n = 138] or familial cases [n = 83]; Supp. Table S1). Sequencing was performed by using next-generation sequencing (NGS, exome or targeted gene sequencing) and Sanger sequencing (SS), and applied for 65 and 156 probands, respectively. Additional NGS or SS was performed for extra members of familial cases, when available. When exome sequencing was performed in affected members of the same family, we filtered out variants with minor allele frequency greater than 1% in public databases (1000 Genomes Project, and NHLBI ESP exomes) and in our in-house database of 609 Brazilian control exomes (Supp. Methods). Among the 221 probands, we identified a total of 47 variants, of which 12 were absent in our controls (two missense, one nonsense, and nine noncoding or synonymous variants; Supp. Table S2). Variants were submitted to the LOVD database (at http://www.lovd.nl/CDH1).  The novel missense variant c.760G>A (p.Asp254Asn, exon 6) was the most likely causative variant among the main candidates detected by exome analysis (mean coverage of 60×; average of 25,140 variants called for each individual; Supp. Table S3) in families F3788 and F617 (four affected individuals sequenced in each family; Supp. Fig. S1a and b). Both families segregate NSCL/P, and haplotype analysis of the exome data did not support a close relationship between these families (data not shown). SS of additional two affected and eight unaffected members from these two families supported segregation in accordance with an autosomal dominant model with incomplete penetrance estimated at 53%. Assuming this penetrance, a Lod score of 4.8 was obtained at recombination fraction (θ) 0, under an allele frequency of 0.0001. The other novel missense variant, c.2351G>A (p.Arg784His, exon 15), was found in the proband of family F1387 through SS. Segregation with NSCL/P was evidenced by its presence in three affected relatives (Supp. Fig. S1c), and penetrance was estimated at 62%. Further, the loss-of-function variant c.1023T>G (exon 8), predicted to create a stop codon at position 341 of CDH1 (p.Tyr341*), was found in the proband of family F7618 (Supp. Fig. S1d) through SS. Segregation with NSCL/P was suggested by its presence in an affected first cousin once removed. Although this represents the first association between this variant and NSCL/P, an association with hereditary diffuse gastric cancer (HDGC) has been previously observed [Guilford et al., 2010]. Penetrance was estimated at 31% in this family. Considering the four pedigrees, an overall 47% penetrance of NSCL/P was estimated, with a maximum Lod score of 5.86 at θ = 0 (individual Lod scores: F3788: 2.3; F617: 2.5; F1387: 0.9; F7618: 0.2).  Among the remaining nine noncoding or synonymous novel variants, only two variants were significantly scored by in silico tools for pathogenicity prediction (Supp. Table S4). Variant c.387+5G>A, which possibly decreases exon 3 splice donor site recognition, was found in a NSCL/P proband from a nonfamilial case (parental DNA unavailable for testing whether it is a de novo variant). We considered this variant as possibly pathogenic, although further functional studies are necessary. Variant c.2514C>T (exon 16), present in two unrelated probands (one isolated and one familial case), was discarded as pathogenic as it did not segregate with NSCL/P in the familial case.  E-cadherin consists of three major domains: a short cytoplasmic, a single transmembrane, and a large extracellular domain, with five repetitive subdomains [Paredes et al., 2012]. The p.Asp254Asn substitution is located at a calcium-binding site comprised by the amino acid sequence Asp–Gln–Asn–Asp, at position 254–257 of CDH1 [Tepass et al., 2000]. This site, in turn, is located in the outmost extracellular subdomain, which plays a major role in the molecular adhesive properties between cadherin trans-dimers [Shapiro et al., 1995]. Calcium binding in the extracellular subdomains is necessary for the cis-dimerization of E-cadherin, and for conferring rigidity to the extracellular domain [Nagar et al., 1996; Pertz et al., 1999]. Importantly, an amino acid substitution in a nearby calcium-binding site has been reported to completely suppress the cellular adhesive properties of E-cadherin in vitro [Ozawa et al., 1990]. The p.Arg784His substitution is located in the cytoplasmic domain, which is important for the assembly of catenins and for promoting cellular signaling [Nelson and Nusse, 2004]. Analyses with in silico tools indicated that these two missense variants are located in highly conserved regions of E-cadherin and probably impair protein function (Supp. Table S4). To determine their pathogenic potential in vitro, we transiently transfected Chinese Hamster Ovary (ATCC number: CCL-61) cells, which are negative for E-cadherin expression, with vectors encoding the wild-type (WT) E-cadherin and variants p.Asp254Asn and p.Arg784His (Supp. Methods). As revealed by Western blot and immunocytochemistry analysis, Asp254Asn cells showed decreased total E-cadherin protein expression (P = 0.00053; Fig. 1A), as well as reduced amount of E-cadherin located in the plasma membrane (Fig. 1B), when compared with cells expressing the WT protein. Furthermore, mutant cells were unable to form cellular aggregates and exhibited a scattered phenotype, contrary to the WT cells, thus clearly indicating impaired adhesive function (Fig. 1C). Even though no structural impact was predicted in the mutated CDH1 protein (performed with FoldX, http://foldx.crg.es/: ΔΔG = –0.81 kcal/mol), our in vitro assays suggest that p.Asp254Asn may lead to premature degradation, as shown for other cancer-related CDH1 pathogenic variants [Simoes-Correia et al., 2008, 2012; Figueiredo et al., 2013]. The functional effect of this variant could also be related to disturbances in calcium ion binding, given its location. Arg784His cells, in turn, showed no observable difference from WT cells in total E-cadherin amount, its location in the plasma membrane, and its adhesive behavior (Fig. 1A–C). However, this result should not be sufficient to rule out the pathogenicity of this variant, since this in vitro assay may not be able to detect other types of functional effects, such as changes in interactions with other proteins, altering subsequent signaling pathways.  Figure 1.  Figure 1. Open in figure viewerDownload Powerpoint slide In vitro functional characterization of CDH1 missense variants p.Asp254Asn (D254N) and p.Arg784His (R784H). A: Protein expression levels of Chinese Hamster Ovary cells transfected with the WT E-cadherin, the variants D254N and R784H, and the empty vector (Mock). α-Tubulin was used as a loading control. Band intensity was quantified and normalized against WT E-cadherin-expressing cells. The graph shows the average ± SE of E-cadherin protein level in five independent experiments. B: Immunocytochemistry showing E-cadherin subcellular localization (green staining). Nuclei counterstained with DAPI (blue). Scale bar represents 100 μm. C: Cell–cell adhesive properties assessed by slow aggregation assays. D: Invasive ability of cells expressing WT or the D254N and R784H variants. The graph shows the number of invasive cells ± SE of five independent experiments. Cells expressing a known invasive mutation associated with gastric cancer (Q16H), used as a positive control, showed higher invasive potential than the studied mutations. Families F3788 (p.Asp254Asn) and F1387 (p.Arg784His), the only two we were able to reascertain for cancer family history, include mutation carriers aged up to 70 years without cancer. This observation suggests that, under certain circumstances, CDH1 variants might cause NSCL/P alone. The invasive potential of Asp254Asn and Arg784His cells, investigated by an in vitro Matrigel matrix invasion assay, was similar to that of WT cells (Fig. 1D). Thus, it is possible that some E-cadherin mutations increase the risk of NSCL/P alone, while others increase risk of gastric cancer (mutations associated with higher invasiveness). However, it is of note that the current landscape of CDH1 mutations associated with gastric cancer and CL/P does not suggest any preferential distribution of mutations along the E-cadherin molecule (Fig. 2).  Figure 2.  Figure 2. Open in figure viewerDownload Powerpoint slide Scheme of CDH1 protein showing the distribution of CDH1 germline mutations (missense × nonsense/splice site/frameshift) associated with gastric cancer [Corso et al., 2012], cleft lip/palate, and both phenotypes. Circles depict the proportion of mutations associated with each phenotype along the protein domains (signal, precursor, extracellular, transmembrane [TM], and cytoplasmic). The overall prevalence of rare, possibly pathogenic CDH1 variants here reported was 2% (5 out of 221 NSOFC probands). To date, the CDH1 mutational repertoire in the literature associated with OFC includes 10 different mutations. Six of these mutations have been reported in families also segregating gastric cancer (four affecting mRNA splicing, one nonsense, and one frameshift deletion) [Frebourg et al., 2006; Kluijt et al., 2012; Benusiglio et al., 2013] and four were found in individuals with uncertain history of gastric cancer (one nonsense [Bureau et al., 2014] and three missense in a European cohort [Vogelaar et al., 2013]). Revisiting the list of variants described in the European cohort, we observed that the missense variant c.88C>A, which was reported in two patients from that study, was also found in two of our Brazilian controls; after removing this variant, the prevalence of possibly pathogenic CDH1 mutations in the European cohort becomes 2%, instead of the previously reported 5% and now similar to our estimate. Furthermore, considering that two of the possibly pathogenic mutations here reported (p.Asp254Asn and p.Tyr341*) and most of the 10 above-mentioned mutations are predicted to cause CDH1 loss-of-function, haploinsufficiency in critical stages of embryonic development seems to be the most likely mechanism by which rare variants in CDH1 lead to OFC.  To investigate whether the group of 221 NSOFC probands presents a higher burden of CDH1 rare variants compared to that of 609 Brazilian controls, we performed a gene-based sequence kernel association test (SKAT) [Wu et al., 2011]. A complementary two-tailed Fisher's exact test was performed to compare the proportion of individuals carrying at least one rare CDH1 variant between probands and controls. Only variants with minor allele frequency <1% and with nonneutral prediction in at least one in silico tool were selected for these tests. To avoid methodological bias in the tests, we only included variants from regions that were covered by both SS and NGS (with minimum coverage of 25×; Supp. Table S5). No significant differences in variant enrichment were detected by SKAT when comparing the 221 NSOFC probands with our 609 control exomes (P = 0.25). Similarly, no significant difference was detected in the number of individuals carrying these variants (two-tailed Fisher's exact test P = 0.85; patients: 9/221 [4%]; controls: 28/609 [5%]). Most of the probably pathogenic variants here reported (p.Asp254Asn, p.Tyr341*, and p.Arg784His) were found in familial cases with at least two affected members aside from the probands (4 out of 26 families matching the same condition, or 15.4%). Considering only probands from families with at least two additional affected individuals, significant differences were detected by SKAT (P = 0.002) and by two-tailed Fisher's exact test (P = 0.002; patients: 6/26 [23%]; controls: 28/609 [5%]). These findings suggest that the most noteworthy CDH1 etiological contribution to NSOFC arises from the fraction of NSCL/P cases involving moderate penetrance, which is best represented by familial cases. Since the previously suggested association between common variants and NSCL/P [Letra et al., 2009; Hozyasz et al., 2014] has not been supported by a large meta-analysis with GWAS data [Ludwig et al., 2012], rare variants seem to be the major contribution of CDH1 to NSCL/P etiology.  Given the lack of correlation between type/location of CDH1 rare pathogenic variants and NSCL/P or HDGC (Fig. 2), we speculate that a common underlying molecular mechanism could explain both phenotypes. Penetrance of CDH1 germline mutations implicated in HDGC depends on a second hit, which frequently occurs via promoter hypermethylation of the nonmutated allele, possibly triggered by environmental factors [Oliveira et al., 2009; Zeng et al., 2015]. In this regard, a lifetime exposure to such factors would be in agreement with the higher penetrance in HDGC (80%) [Pharoah et al., 2001], as compared to NSCL/P (47%). Germline, pathogenic variants in CDH1 could determine the resultant phenotype (NSCL/P or gastric cancer) under the influence of the following factors: time (early development or later in life), tissue (craniofacial or gastric structures), and exposure to environmental factors. In addition, given the prevalence of CDH1 pathogenic variants found in this study, we believe that the NSCL/P-associated CDH1 mutations are currently underrepresented, and future research should focus on their identification.  In summary, our results indicate a consistent role of rare, loss-of-function, moderately penetrant CDH1 variants in NSCL/P etiology. To better comprehend the mechanisms linking CDH1 to NSCL/P, as well as the risk of gastric cancer among NSCL/P individuals with mutations in CDH1, further studies are needed. Finally, CDH1 testing in NSCL/P familial cases should be discussed for genetic counseling purposes.\n",
      "346\n",
      "E-cadherin is involved in the formation of cell-junctions and the maintenance of epithelial integrity. Direct evidence of E-cadherin mutations triggering tumorigenesis has come from the finding of inactivating germline mutations of the gene (CDH1) in hereditary diffuse gastric cancer (HDGC). We screened a series of 66 young gastric cancer probands for germline CDH1 mutations, and two novel missense alterations together with an intronic variant were identified. We then analysed the functional significance of the two exonic missense variants found here as well as a third germline missense variant that we previously identified in a HGDC family. cDNAs encoding either the wild-type protein or mutant forms of E-cadherin were stably transfected into CHO (Chinese hamster ovary) E-cadherin-negative cells. Transfected cell-lines were characterized in terms of aggregation, motility and invasion. We show that a proportion of apparently sporadic early-onset diffuse gastric carcinomas are associated with germline alterations of the E-cadherin gene. We also demonstrate that a proportion of missense variants are associated with significant functional consequences, suggesting that our cell model can be used as an adjunct in deciding on the potential pathogenic role of identified E-cadherin germline alterations.\n",
      "347\n",
      "To screen and characterize germline variants for E-cadherin (CDH1) in non-hereditary gastric cancer (GC) patients and in subjects at risk of GC.  Methods  59 GCs, 59 first degree relatives (FDRs) of GC, 20 autoimmune metaplastic atrophic gastritis (AMAGs) and 52 blood donors (BDs) were analyzed for CDH1 by direct sequencing, structural modelling and bioinformatics. Functional impact on splicing was assessed for intronic mutations. E-cadherin/β-catenin immunohistochemical staining and E-cadherin mRNA quantification using RT-PCR were performed.  Results  In GCs, 4 missense variants (p.G274S; p.A298T; p.T470I; p.A592T), 1 mutation in the 5′UTR (−71C>G) and 1 mutation in the intronic IVS12 (c.1937-13T>C) region were found. First pathogenic effect of p.A298T mutation was predicted by protein 3D modelling. The novel p.G274S mutation showed a no clear functional significance. Moreover, first, intronic IVS12 (c.1937-13T>C) mutation was demonstrated to lead to an aberrant CDH1 transcript with exon 11 deletion. This mutation was found in 2 GCs and in 1 BD. In FDRs, we identified 4 variants: the polymorphic (p.A592T) and 3 mutations in untranslated regions with unidentified functional role except for the 5′UTR (−54G>C) that had been found to decrease CDH1 transcription. In AMAGs, we detected 2 alterations: 1 missense (p.A592T) and 1 novel variant (IVS1 (c.48+7C>T)) without effect on CDH1 splicing. Several silent and polymorphic substitutions were found in all the groups studied.  Conclusions  Overall our study improves upon the current characterization of CDH1 mutations and their functional role in GC and in individuals at risk of GC. Mutations found in untranslated regions and data on splicing effects deserve a particular attention like associated with a reduced E-cadherin amount. The utility of CDH1 screening, in addition to the identification of other risk factors, could be useful for the early detection of GC in subjects at risk (i.e. FDRs and AMAGs), and warrants further study.  Go to: Introduction Gastric cancer (GC) remains the fourth most common malignancy worldwide, even though its incidence and associated mortality rates have decreased in recent decades. GC prognosis is closely related to the stage of disease at diagnosis [1]. Early onset gastric cancer (EOGC) is defined as GC presenting at the age of 45 or younger [2] and has a poor overall survival [3], [4]. Most GCs are sporadic and often develop following Helicobacter pylori (HP)-associated gastritis [5], [6]. However, familial aggregation studies also stress the importance of a genetic predisposition in the sporadic development of GC. Frequency of familial gastric aggregation is about 10%.  The most widely accepted GC histopathological classification (Lauren's classification) [7] distinguishes two types of GC: intestinal type and diffuse type. Diffuse GC shows a greater hereditary basis and a generally worse prognosis as compared with the intestinal subtype [8].  CDH1 gene coding for the E-cadherin has been identified to have a causative role in about 30%–50% of hereditary diffuse GC (HDGC), an autosomal dominant GC and lobular breast cancer susceptibility syndrome constituting 1–3% of familial clustering of GCs [9], [10] and in diffuse GC subtype [11]. CDH1 germline mutations (such a mutation is passed on every cell in the offspring's body) are specifically associated with HDGC (about 30%–40% of cases); large CDH1 deletions have been found in about 6.5% of cases [12]. Familial intestinal gastric cancer (FIGC) with a positive family history have also been described but so far, no germline CDH1 defects have been associated with FIGC or intestinal GCs. This lack of evidence of CDH1 mutations in the intestinal subtype has led to the hypothesis that familial clustering in these cases is determined by shared environmental factors, as opposed to an inherited genetic predisposition. However, recent data demonstrate that CDH1 somatic alterations (such alterations accumulate in the cancer cells of the body over a person's lifespan) are as frequent in intestinal as in diffuse GC [13], suggesting an important role of CDH1 in both the histotypes. Nonetheless, the exact prevalence of CDH1 germline alterations in intestinal GCs is still unknown. CDH1 promoter hypermethylation is the most common second genetic hit in the GC carcinogenic process [14], [15]. CDH1 mutations are also associated with an increased susceptibility to invasive and metastatic [16], [17] colon, bladder, prostatic, breast and gynaecological cancers [18]–[20]. E-cadherin is a transmembrane glycoprotein that plays a role in maintaining epithelial tissue architecture by involving Ca2+ dependent cell-cell interactions [21], [22]. E-cadherin comprises a cytoplasmic domain, a short transmembrane domain and five extracellular repeat cadherin-like domains (EC1-5) that span exons 4–13 and contain highly conserved calcium-binding regions [23], [24] and conserved cysteines likely to form disulfide bridges [25].  In this study, we analysed CDH1 germline mutations in a series of consecutive random GC cases and individuals at risk of GC; mainly first degree GC-Relatives (FDRs) and autoimmune metaplastic atrophic gastritis (AMAG) patients directed to our institute for gastrointestinal symptoms and endoscopic evaluation. To explore the role of E-cadherin expression, structural, functional and immunohistochemical analyses were performed in samples with a CDH1 germline mutation. The aim of the present study was to evaluate the prevalence and characterize CDH1 germline mutations in a series of consecutive sporadic GC patients lacking the criteria of HDGC classification, and in a selected population at risk of GC development, to test its utility as a marker to improve early tumor detection. Data obtained could be used to develop a tool that rapidly and cheaply detects CDH1 mutations mainly present in our population.  Go to: Results Patient characterization and CDH1 germline genetic screening  Clinical and histopathological features of GC, FDR and AMAG subjects are summarized in Tables 1 and and2.2. Among the 59 GC patients, 2 (3.4%) have a family history of GC (S15 is the brother of S16) without meeting the criteria for hereditary diffuse GC, as defined by the International Gastric Cancer Linkage Consortium (IGCLC) at the time of sample collection. In our GC series, 5 sporadic early GC patients (≤45 years old) were present, but no CDH1 alterations were found in these patients. The median age of the FDRs was 49 years (range, 28–78 years) and for AMAGs 56 years (range, 31–72 years). Among the 59 GC patients, 16 subjects had a first degree relative included in the study (16/59 FDRs). FDRs and AMAGs came to our institution for a gastroenterology visit and gastroscopy exam, they manifested various symptoms, but neither cancer nor intestinal metaplasia/dysplasia was present in these subjects.  Table 1 Table 1 Clinicopathological characteristics of patients with gastric cancer. Table 2 Table 2 Patients characteristics and clinical presentation. CDH1 genetic screening results are listed in Table 3 with new mutations: 1 intronic (ID 5), 1 missense (ID 10), and 2 silent (ID 13 and ID18). Overall we found 4 variants, which code for an amino acid (AA) substitution (1 novel (ID10) and 3 previously reported in other populations (ID 11, ID 12, ID 15), 1 in the 5′near gene region and 2 mutations in the untranslated (UTR) regulatory element (ID 1, ID 3, already reported) and 6 substitutions in intronic regions (three mutations: ID 5, ID 9, ID 17; three intronic polymorphic variants: ID 4, ID 6, ID 8, probably with no effect on GC cancerogenesis). No deletions or insertions were found in the exon boundaries.  Table 3 Table 3 Analysis of CDH1 germline mutations found in 190 individuals. Other alterations resulted common polymorphisms (frequency of at least 1% in the population) or silent mutations that code for the same amino acid than the original strand. None statistical association was observed among the four groups of patients tested for ID 4, ID 6 or ID 19 variants.  RefSNP (rs) numbers to identify genetic variants previously published as well as their reported frequencies (NIEHS Environmental Genome Project, Seattle, WA (URL: http://evs.gs.washington.edu/niehsExome/ Accessed August 2013); ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/paper/ Accessed December 2012) are reported in Table 3.  All CDH1 variants were in heterozygous state except for the ID 4 and ID 19 in which a homozygous state was also detected.  Frequency of mutations and variants were calculated in subjects without GC or AMAG disease (52BDs+59FDRs, n = 111). Sixteen FDRs were first degree relatives of our GC series; when one of the variant was present in GC and its related FDR case, we excluded the FDR individual from the frequency calculation. ID4 for example, was present in 5 FDRs related to our GC of our series; therefore the control population frequency changed from total 111 to 106 individuals (7FDRs+8BDs/106; 14.5%). Figure 1 illustrates sequencing chromatograms of the novel mutations we have found. We have previously reported the ID 10 chromatogram in another paper [26].  Figure 1 Figure 1 Sequencing chromatograms showing the novel CDH1 germline mutations. Bioinformatic predictive role and structural modelling results of missense variants found  The missense mutated residues we found are all localized to the E-cadherin extracellular domain. The codon position in the immature and mature (after the N-terminal cleavage) proteins and data from the PolyPhen-2 and SIFT in silico analyses are reported in Table 4. All four missense variants are potentially damaging by PolyPhen-2, but only the p.A298T (ID 11) and p.A592T (ID 15) substitutions may affect protein function by SIFT analysis (Table 4). The p.G274S (ID 10) that we recently described [26], however, does not perturb the local environment, but introduces a potential residue for phosphorylation and glycosylation that may have possible effects on the stability and integrity of E-cadherin as we hypothesized [26]. The pathogenetic effect of ID 11 substitution was previously established [27], but was here first demonstrated by structural analysis. As illustrated in Figure 2A, the AA change in exon 7 of the p.A298T (ID 11) is positioned near the interactive region between protomers EC1 and EC2. Thus, the alanine-threonine polar residue substitution may drive H-bond formation through its oxydrilic group and this may interfere with the local structure of the protein in a region that is fundamental for Ca2+ interactions. Threonine in position 144 is sterically demonstrated obtrusive because it interacts with two aspartic acid residues (Asp136 and Asp 138) that are directly involved in Ca2+ binding. Moreover, the bond lengths are particularly stressed, being less than 3 Å (Figure 2B).  Figure 2 Figure 2 Structural modelling of the extracellular domains of the E-cadherin protein. Table 4 Table 4 Summary of germline missense variants detected in CDH1 gene and predictedfunctional effect by SIFT and PolyPhen tools. As regard to the remaining two missense mutations, they have a less clear functional effect as also reported in Table 4. p.T470I (ID 12) substitution [28] changes the AA surface of extradomain EC3 in the mature protein (Figure 2C). In both the murine E-cadherin and N-cadherin sequence (PDB code: 3Q2W) threonine is usually found O-glycosylated suggesting an important role for this residue in the structure of the protein. However, as showed in the Figure 2D, the change to isoleucine, a non-polar AA with a hydrophobic side chain that cannot undergo post-translational modification, suggests no particular intermolecular tension. We hypothesize that in the extracellular medium, the presence of an isoleucine residue at the same position than threonine may favour protein-protein interactions, and this mutation could thus assume a protective significance. The last mutation reported, p.A592T (ID 15), was found in all groups tested (see Table 3), suggesting an improbable effect on GC pathogenesis. In this case, Alanine on the extradomain EC4 of the mature E-cadherin (Figure 2E) provides conformational freedom, even when in proximity of the Ca2+ binding sites. A threonine substitution here has a limited effect on the local structure and torsional angles of the protein. However, we can not exclude that the oxydrilic lateral chain could be post-translational modified in particular situation and thus influence the structure and function of the CDH1 (Figure 2F).  Transcript analysis of intronic germline mutations  To explore if intronic mutations detected in our GC series (Table 3) could potentially induce an effect on splicing, we performed CDH1 transcription analysis. Polymorphic and silent variants were excluded from this analysis since they probably have no pathogenic role. cDNA produced from peripheral blood of the selected GC individuals harbouring intronic ID 5, ID 9 or ID 17 mutations (Table 3) were compared to that from two healthy blood donors, one only having the same ID 17 mutation as GC patients (BD code S190), and another (BD code S189) without CDH1 mutation.  For the ID 5 and ID 7 intronic mutations, we amplified the region covering part of exon 1 to part of exon 5, for ID 17 mutation, exon 10 to 13 (Figure 3). The RT-PCR exon 1 to 5 fragments showed no differences when run on 4% agarose gel (Figure 3A) nor after bidirectional sequencing (data not showed); by converse ID 17 intronic variant could affect splicing leading to an abnormal smaller CDH1 transcript (Figure 3B). Upon isolation and sequencing, we found that the smaller band resulted in a skipped transcript lacking exon 11, with exon 10 directly joined to exon 12. This aberrant transcript was also detected in the BD S190 carrying the same germline substitution (Figure 3B).  Figure 3 Figure 3 Evaluation of potential effects on splicing on CDH1 gene of the intronic variants found. Analysis of CDH1 protein abundance and mRNA expression level in subjects showing CDH1 intronic mutations  A comparison of E-cadherin mRNA expression level was performed from EBV immortalized lymphocytes obtained from the peripheral blood of S10 and S190 (mutation ID 17), S97 (ID 9) and S189 (no CDH1 mutations) subjects. Subject S10 was affected by a gastric cancer, subject S97 is first degree relative of a patient with a gastric cancer (FDR), while S189 and S190 were both blood donors. We observed between the control blood donor (S189) having no CDH1 mutation and patients, a relative strong decrease in E-cadherin expression (about 60%, Figure 4) in patient S10 having both ID 17 mutation and a GC, while only about a 2% reduction in the blood donor S190 having the same ID 17 mutation (p<0.05, with respect to GC S10). For S97 (mutation ID 9, FDR subject), we observed a similar E-cadherin expression as that in the control S189.  Figure 4 Figure 4 Graphic presentation of relative E-cadherin quantification by quantitative real time RT-PCR. Immunohistochemical analysis on the tumor gastric tissue of intronic ID 17 case (patient code S10, Figure 5E) showed a reduced expression of membrane-bound E-cadherin in the signet ring tumor cells (black arrows), while both membrane and cytoplasmic staining were present in the normal epithelium. The same patient showed reduced β-catenin staining in the signet ring cells as compared with the strong expression of this protein in the normal adjacent cells (Figure 5H). The loss of both E-cadherin and β-catenin staining was also noticeable for the second patients (S46) having the same intronic ID 17 mutation and affected by GC too (Figure 5F and 5I, respectively for E-cadherin and β-catenin).  Figure 5 Figure 5 Hematoxylin and eosin stain gastric sections and immunohistochemical staining for E-cadherin and β-catenin. Go to: Discussion GC patients typically have a poor prognosis [29]. Identification of patients with an increased risk of developing GC and the early detection of GC are promising approaches to reduce the morbidity and mortality of GC. FDR of GC patients are known to have a 2–3 fold increased risk of GC, probably owing to exposure to the same environmental risk factors and/or to inherited susceptibility to cancer [30].  The parietal cells destruction found in AMAG combined with the important role of E-cadherin in epithelial polarity and gastric glandular architecture, suggests that germline alterations of CDH1 could be an additional risk factor for GC development in AMAG patient [31].  In 1998, Guilford and colleagues described for the first time germline mutations of the CDH1 gene [28]. Subsequently, different types of mutations have been reported in families of varying ethnicities with diffuse GC [32], [33]. The first CDH1 germline mutation was described in an Italian family in 2006, in a patient who met the IGCLC criteria for HDGC [34]. However, very few studies report CDH1 germline mutations in sporadic GC cases without familial aggregation or in subjects at risk of developing GC [35], [36]. Moreover, in these studies the functional effects of CDH1 variants often are not investigated.  The strength of our study is the collection of 59 Caucasian patients with sporadic GC, 59 FDRs and 20 AMAGs who attended our gastroenterology service in the last years for gastric symptoms and a diagnosis or exclusion of a GC after endoscopic and histological tissue evaluation.  As summarized in Table 3, various different germline CDH1 variants have been detected. In the 59 GC series, excluding the polymorphic and silent changes that probably have no pathogenic role, we found 6 different substitutions in 9 patients (9/59 GCs = 15.2%): 4 of the missense type (ID 10, ID 11, ID 12, ID 15) in 4 distinct patients (6.8%) and 2 of non-missense type (ID 2 and ID 17) in 5 distinct GCs (3.4%).  The ID 10 (p.G274S) is a novel missense mutation that we found in an old male with a GC mixed histotype. This variant was not detected in 187 free-cancer individuals (108BDs+59FDRs+20AMAGs) thus excluding a polymorphism. A pathogenic effect of ID 10 mutation was not supported after functional (aggregation and invasion) in vitro assays as we recently reported [26], nonetheless data from in silico characterization of the mutation and a reduction in β-catenin expression found in the tumor tissue cannot completely exclude the significance of this mutation in GC development. Thus, at today ID 10 remains a novel CDH1 mutation with a pathogenesis of an undetermined significance.  The ID 11 (p.A298T) substitution in exon 7 of CDH1 has already been described in a 36-year-old young Caucasian male in a HDGC family [27]. In our series, this variant was detected only in 1 male (S47) of 74-year-old with a mixed histotype. The potential pathogenic effect of this mutation has been confirmed through in vitro functional studies in different laboratories [27], [37], [38]. Here first modelling results (Figure 2A–B) by analyzing 3D protein-ligand binding interactions, strongly support the potential for altered protein function and lead to the possible molecular mechanism that sustain this process. The potential altered protein function was supported also from SIFT analysis (Table 4) with a good score. Moreover, a recent study, using the in silico protein design FoldX algorithmic approach [39], reasserts the pathogenic role of the ID 11 (p.A298T) substitution, based on a calculation of native-state stability changes (ΔΔG>0.08 kcal/mol) [40]. Authors characterized patients harbouring this missense mutation as having a younger age at diagnosis and a diffuse histotype. Our case highlighted that ID 11 can also be detected in an old patient with mixed GC.  The ID 12 (p.T470I) was found in a 57-year-old male (S39) with a diagnosis of GC. This change was first described in a family of Maori ethnicity with EOGC, but the subject showing this mutation was not affected by GC at the time of study [28]. Here, we found that the p.T470I AA change is tolerated by SIFT and also by modelling analysis. Unfortunately, the tumor bioptic tissue specimen was insufficient to perform E-cadherin IHC staining.  The ID 15 substitution (p.A592T) was detected in each clinical group tested, suggesting a probable polymorphic diffusion. Nonetheless, this variant has been previously reported associated with thyroid tumors and lobular breast cancers [41]–[43]. Our structural analysis and in vitro [35] and in silico studies [38], [40] do not support a pathogenic role for this variant in GC.  As recommended by recent clinical management guidelines [44], endoscopy surveillance should be performed annually in those individuals with mutations of undetermined significance (eg, missense). In our opinion, subjects harbouring ID 15 and also ID 10, must be followed for up to 10 years before excluding a role although weak for this alteration in the pathogenesis of GC.  In the ID 2 we identified a C-to-G change before the start codon (−71C>G, CDH1 5′UTR region), that represented the most common variant associated with GC in our series, occurring in three out of 59 GC patients (5.1%). This variant was also reported in a Finnish study [45] in 1 of 13 (7.7%) GC patients and in 2 of 51 controls (3.9%), and also in two EOGC patients of Northern American origin (3.4%) [46]. Overall data from these studies suggest that ID 2 is a quite common mutation but authors did not report data about ID 2 variant in relation to the E-cadherin expression status. ID 2 was found in our series in one intestinal, one mixed, and one diffuse GC histotypes. All these patients had over 50 years at diagnosis and were negative for HP infection. None of control subjects (n = 111) tested without GC, showed this mutation (Table 3). An in situ evaluation or a correlation between ID 2 and E-cadherin expression was unable to be performed due to a lack of tumor material. The potential pathogenic effect of this promoter variant on E-cadherin expression level deserves further studies.  Intronic ID 17 variant (IVS12 c.1937-13T>C) was found in 2 females with GC (2/59 GCs = 3.4%) both positive for HP infection, and it was found also in 1 BD (1/52 = 1.9%, Table 3). The same alteration was previously reported in lobular breast cancer with high frequency (12/53 = 23%) [47], in HDGC families (2/27 = 7.4%) [48] and in EOGC patients (7/79 = 8.9%) [46] but also in a relative control population [46]. Of note, we demonstrate for the first time that this substitution leads to an aberrant CDH1 transcript harbouring a deletion of the CDH1 exon 11. Exon 11, together with partial sequences of the flanking exons, codifies for the EC4 domain of the mature protein [25]; ID 17 is an out-of-frame deletion and leads to the formation of a premature stop codon at position 384 of the EC4 promoter. Consequently, the translated protein from CDH1 ID 17 strand could lack the transmembrane domain and the cytoplasmic tail that is involved in β-catenin binding. Both S10 and S46 GC patients, having the ID 17 mutation, showed a reduction in the expression of E-cadherin and β-catenin by IHC analyses (Figure 5); the GC S10 patient, with a signet ring cell carcinoma, was diagnosed at the age of 61 years, and the GC S46 patient, with a diffuse adenocarcinoma, was diagnosed at the age of 58 years. Moreover, evaluation of E-cadherin expression from the EBV immortalized B-lymphocytes showed a strong reduction (60%) in GC S10 harbouring ID 17 mutation as compared with the BD control (S189) without CDH1 alterations, but also as compared to a single blood donor (S190) carrying the same ID 17 variant. Since all subjects carrying the ID 17 mutation are heterozygous for the CDH1 gene, our data indicated that the S190 individual, but not tumor cells of S10 and S46 patients, may exploit some compensatory mechanism that counteracts the E-cadherin down-regulation. In tumors, E-cadherin under-expression is linked to enhanced β-catenin transcriptional activity, a main effector of the Wnt pathway [49]. The expression of a large number of genes related to tumor progression, including those for cyclin D1, c-myc, vascular endothelial growth factor, and survivin is controlled via the Wnt/β-catenin pathway [50]. E-cadherin binding to β-catenin prevents its translocation to the nucleus; accordingly, a reduction of E-cadherin expression may favour GC pathogenesis through an increased nuclear β-catenin accumulation. Since patients S10 and S46 with ID17 variant are both women showing a helicobacter pylori (HP) infection, we assume that ID 17 might be associated with a sex-specific prognostic factor (it is well known that the incidence of GC is higher for men than for women) and/or an HP infection. A deletion of exon 11 in the CDH1 gene was also described in a HDGC patient, but in this last case aberrant splicing was associated to a different intronic mutation (IVS11 c.1711+5G>A) [27]. Intriguingly, an alternatively spliced, non-functional E-cadherin transcript that lacks exon 11 of the gene had also been reported in some head and neck cancer cells [51] and chronic lymphocytic leukaemia cases (CLL) and although at a lower level compared to CLL, also in normal B cells [52]. In these cases no genetic alterations in exon 11 or in its flanking intronic regions were observed; the non-functional transcript has a premature termination codon and is degraded by the nonsense-mediated RNA degradation [52]. Splicing factors, binding in the region of exon 11 of CDH1, could have altered expression levels or states of activation in CLL cells compared with normal B-cells as recently demonstrated [53].  In the FDR group of subjects, with the exclusion of polymorphic and silent CDH1 mutations, we observed 3 substitutions (ID 1, ID3 and ID 9) which were not found in GC.  The ID 1 (5′near gene-176C>T) variant was detected in a 32-year-old female with an unknown HP infection status. This variant was already submitted in popular databases, but its significance is unknown.  The ID 3 (5′UTR-54G>C) variant was found in a 72-year-old man positive for HP infection. Of interest, this mutation was already detected in a healthy 41-year-old Japanese subject, with no clinically detectable tumor at the time of the enrolment, and described as a rare variant able to decrease the transcriptional activity of CDH1 [54]. We hypothesize that this mutation by introducing a CpG island in the CDH1 promoter region increases the probability of CDH1 hypermethylation, a well known event favouring the transcriptional inactivation, an early event in HP gastritis [55] and a key risk factor associated to GC development.  The ID 9 (intronic IVS4 c.532-18C>T) was found in a 41-year-old male subject negative for HP infection. ID9 was first reported in two EOGC patients from England and Portugal, respectively [56], in two HDGC German patients and in 1 control subject enrolled in the same study [35]. Recently, a non pathogenic role for this variant was proposed [2]. We did not notice any influence on CDH1 splicing.  AMAG patients have a 3-fold increased relative risk of developing GC and have been never investigated for CDH1 germline mutations until now. In AMAG series we found only polymorphic variants with the exception of ID 5, a new intronic mutation close to exon 1 (IVS1 c.48+7C>T). ID 5 was found in a female of 51-year-old with hypergastrinemia. We did not find any truncations or frameshifts in the production of the protein associated to this mutation. Although our series is limited (n = 20), these data seems not to support a relevant role of CDH1 genetic alterations associated with AMAG disease.  In conclusion, our results show that the well known pathogenic ID 11 mutation (p.A298T) can also be detected in sporadic GC patients without fulfilling the strict criteria for HDGC. Furthermore, we demonstrated a deleterious effect of ID 17 variant (IVS12 c.1937-13T>C) on CDH1 splicing and a related decrease in E-cadherin expression and also for β-catenin. The same ID 17 mutation and splicing effect found in 1 blood donor, but with a limited effect on E-cadherin mRNA level, is intriguing and deserves further studies. Considering the correlation among specific CDH1 germline alterations and the tumor histotype, we found that 8.3% (1 of 12 GCs) of mixed (ID11) and 7.7% (2 of 26 GCs) of diffuse (ID17) subtypes, carried a potential pathogenic mutation.  Finally, in a FDR individual at risk for GC, we found the ID 3 variant (5′UTR-54G>C) with a potential effect of increasing the hypermethylation status of CDH1, a well known risk event associated with GC development and progression.  North East of Italy presents high GC incidence and mortality rates although lower respect to central regions, like Tuscany and Marche [57]. Our findings show prevalence in missense CDH1 substitutions versus non-missense alterations, as reported in a recent metanalysis for middle-high GC risk areas like the Central Italy [58]. However, we can not excluded that the middle-high GC prevalence herein found might be slightly more likely than in the rest of Friuli geographic region since study was conducted in a Cancer Institute.  Moreover, variants found in subjects at risk for GC, particularly in FDRs and recently findings of novel mutations in sporadic GC patients in Chinese population [59], invite to screen for CDH1 genetic alterations in addition to other risk factors, to define a high-risk group of patients that would benefit from an early GC diagnosis.  Go to: Methods Patients and sample preparation  Fifty-nine patients at first GC diagnosis were consecutively recruited at the Gastroenterology Unit of Centro di Riferimento Oncologico (CRO), National Cancer Institute. Histopathological diagnoses were based on the WHO Classification [60] and Lauren's classification [7]. Clinicopathological characteristics of patients are reported in Table 1. Concurrently, 20 consecutive AMAGs (S119–S139) and 59 FDR (S60–S118) individuals (parents, children, siblings, and offspring of a relative with a GC) were recruited from the same centre; the participants inclusion criteria were patients who attended gastroenterology unit for gastric symptoms and with exclusion of a GC after endoscopic and histological tissue evaluation (Table 2). A random sampling of 52 (S139–S190) healthy blood donors is used to be representative of the general population (BDs). 56 additional BD controls were genotyped for the novel variants. For each participant, a peripheral blood sample was collected in acid citrate dextrose (ACD) tubes, and genomic DNA extracted using the EZ1 DNA Blood kit and the BioRobot EZ1 Workstation (QIAGEN Inc., Valencia, CA, USA). Multiple biopsies were collected for preservation and immunohistochemical analysis. All subjects freely gave their written informed consent. Ethical guidelines for research involving human subjects were respected and this study was approved by the CRO institutional review board (CRO: Ricerca corrente. Project n.4 linea n.1).  Germline CDH1 mutation screening  PCR CDH1 primer sequences for the amplification of all 16 coding exons were previously reported. PCR reactions were carried out in a volume of 10 µl containing 10 ng of genomic DNA template, 1 mM MgCl2, 1 mM dNTPs, 0.6 µM of each PCR primer, 5X Green Buffer and 0.25 U Go-Taq DNA Polymerase (Promega, Madison, WI, USA). Furthermore, 5% DMSO was added to PCR reactions for exon 1 and 2. Thirty cycles of 30 s at 94°C, 30 s at 60°C and 1 min at 72°C were performed in a programmable thermocycler (Eppendorf, Hamburg, Germany). A 2 µl aliquot of the PCR product was then purified using 0.5 µl of ExoSAP-IT kit (USB Corporation, Cleveland, OH, USA), and a 0.5 µl aliquot of this purified product was sequenced using the Big Dye Terminator kit (Applied Biosystems, Foster City, CA, USA) on an ABI PRISM capillary sequencer. Chromas and ClustalW software were used for multiple sequence alignment. Variants detected were confirmed using the genomic DNA sequence.  Characterization of the impact on splicing for intronic variants by RT-PCR  To detect intronic splice variants, RNA was isolated from the peripheral blood mononuclear cells (PBMC) of patients with a CDH1 germline alteration and that of control patients negative for the same variant using the EZ1 RNA Cell Mini Kit and the BioRobot EZ1 Workstation (QIAGEN Inc.). First-strand cDNA was synthesized from 0.8 µg total RNA with the High Capacity cDNA Reverse Transcription Kit (Applied Biosystems) according to the manufacturer's protocol. CDH1 transcripts were amplified using gene-specific forward (FP) and reverse (RP) primers: i) CDH1 exons 1–5 flanking the c.IVS1+7C>T and c.IVS4-18C>T sequence variants (FP: 5′-GGAAGTCAGTTCAGACTCCAGCC-3′ and RP: 5′-GTGGCAATGCGTTCTCTATCCAG-3′); ii) the CDH1 exons 10–13 flanking the c.IVS12-13T>C variant (FP: 5′-ACCGTCACCGTGGATGTGCT-3′ and RP: 5′-GAATCCCCAGAATGGCAGGAA-3′). 5% DMSO was added to exons 1–5 PCRs. PCR product sizes were checked against a DNA ladder (Marker VIII, Roche Applied Science, Indianapolis, IN, USA) on a 4% agarose gel, and then sequenced [2].  Immunohistochemistry  Formalin-fixed, paraffin-embedded tissue blocks (tumor and non-tumor) from patients carrying a CDH1 germline mutation were cut into 5 µm-thick sections for H&E staining and immunostaining. Immunohistochemistry (IHC) was performed using the mouse monoclonal antibody against human E-cadherin (clone 36, Ventana Medical System, Tucson, AZ, USA), and β-catenin (clone 17C2 Novocastra, Newcastle upon Tyne, UK). H&E staining was performed according to standard protocols. Appropriate positive and negative control samples were included with each staining series.  Structural modelling and in silico characterization of missense variants  Structural studies on the effect of the missense variants were performed using Pymol (http://pymol.sourceforge.net/ Accessed 16 January 2012) and WinCoot [61]. For the p.A298T in the EC2 domain, we used the scaffold of the crystal structure of the corresponding human wild-type E-cadherin protein (PDB code: 2O72) [25] For the two missense variants (p.T470I and p.A592T located in EC3 and EC4, respectively), we used the murine crystallized sequence (PDB code: 3Q2V) [62].  To test a prediction value of the phenotypic effect of the genetic mutations, an in silico analysis was performed using the SIFT (Sorting Intolerant From Tolerant, http://sift.jcvi.org/ Accessed 21 April 2011) algorithm [63] and the polymorphism phenotyping Polyphen-2 tool (http://genetics.bwh.harvard.edu/pph2/ Accessed September 1, 2011). E-cadherin protein sequence (GI: 31073) was used for alignment comparisons and only mutations with a score below 0.05 were considered to be intolerant for SIFT. The A8K1U7_HUMAN feature (UniProtKB/TrEMBL) was used with the Polyphen tool.  Cell lines and relative quantitative real-time RT-PCR  Lymphoblastoid cell line (LCLs) from four subjects (GC S10, FDR S97, BD189 and BD190) were generated by in vitro immortalisation of B cells with the B.95.8 Epstein-Barr virus isolate [64]. Cell lines were cultured in RPMI-1640, containing 10% heat-inactivated FBS (Gibco-BRL, Gaithersburg, MD, USA), 2 mmol/l L-glutamine, 100 mg/ml streptomycin, and 100 IU/mL penicillin (Sigma-Aldrich, St. Louis, MO, USA) and maintained at 37°C in 5% CO2. LCLs were generated from the seeding of 2,5×106 PBMC. About 8 million of cells were harvested for each sample after immortalization. Total RNA was isolated and used to synthesize cDNA, as described above. Relative quantitative real-time RT-PCR for E-cadherin expression was performed with 2X SYBR Green Master Mix (Applied Biosystems) using a 7500 Real Time PCR system (Applied Biosystems). Primers (set ii) and amplification were as described above. The Ex10-13 product determined the wild-type E-cadherin transcription level. Normalisation of RT-PCR products was determined using the Pfaffl method with β-actin (ACTB) (FP: 5′-GACCCAGATCATGTTTGAGA-3′; RP: 5′-GACTCCATGCCCAGGAAG-3′) as the endogenous control and BD S189 as the reference sample. All experiments were run in triplicate and the mean values were used to calculate E-cadherin mRNA expression.\n",
      "340\n",
      "The transmembrane proto-oncogene receptor tyrosine kinase (RTK) ROS is an orphan receptor that is aberrantly expressed in neoplasms of the central nervous system. Here, we report the fusion of its carboxy-terminal kinase domain to the amino-terminal portion of a protein called FIG (Fused in Glioblastoma) in a human glioblastoma multiforme (GBM). By characterizing both FIG and ROS genes in normal and in U118MG GBM cells, we determined that an intra-chromosomal homozygous deletion of 240 kilobases on 6q21 is responsible for the formation of the FIG-ROS locus. The FIG-ROS transcript is encoded by 7 FIG exons and 9 ROS-derived exons. We also demonstrate that the FIG-ROS locus encodes for an in-frame fusion protein with a constitutively active kinase activity, suggesting that FIG-ROS may act as an oncogene. This is the first example of a fusion RTK protein that results from an intra-chromosomal deletion, and it represents the first fusion RTK protein isolated from a human astrocytoma. © 2003 Wiley-Liss, Inc. INTRODUCTION  Astrocytoma is the most common type of central nervous system neoplasm, affecting 20,000 Americans each year. Because of the propensity to infiltrate the surrounding stroma, astrocytomas are nearly impossible to resect surgically (Wisoff et al., 1998). In addition, these tumors are largely resistant to radiation and chemotherapy. The most advanced astrocytic neoplasm, glioblastoma multiforme, is among the most aggressive human cancers, with a median survival of less than one year, a statistic that has not changed significantly over the past 20 years. The most promising prospects for the development of new therapeutics are likely to come from studies of the genetics and molecular biology of these tumors.  Normal cells require growth signals for survival and proliferation. Many of these are mediated by diffusible growth factors that transmit their signals into the cell by a group of transmembrane proteins with intrinsic tyrosine kinase activity, the receptor tyrosine kinases (RTKs). Upon binding to their cognate growth factor ligands, RTKs undergo receptor dimerization, autophosphorylation, and ultimately recruitment of adaptor proteins that interact with and activate various downstream effectors. Several of these RTK-based signaling networks have been hijacked by specific types of cancers (for reviews see Danilkovitch-Miagkova and Zbar, 2002; Furstenberger and Senn, 2002; Gupta et al., 2002; Heinrich et al., 2002; Santoro et al., 2002; Shawver et al., 2002; Spencer-Cisek, 2002). The most common methods of RTK activation in cancer cells are genomic rearrangements exemplified by gene amplification, chromosomal translocations, inversions, and deletions. Many of these genomic aberrations have been shown to result in the formation of onco-fusion proteins.  Elevated expression of growth factors and/or their related tyrosine kinase receptors has been found in all grades of astrocytoma (Sara et al., 1986; Wong et al., 1987, 1992; Fleming et al., 1992; Trojan et al., 1992; Smith et al., 2000). In most cases analyzed, this increased expression is the result of a gene amplification. Genomic rearrangements that result in the creation of fusion RTK proteins have not been reported in astrocytomas. In fact, the majority of RTK fusion onco-proteins are found instead in hematologic malignancies.  We recently isolated and characterized a novel, ubiquitously expressed gene that we called FIG (Fused in Glioblastoma) (Charest et al., 2001) (GenBank accession number AY033606). This gene encodes for a 454–amino acid protein that includes a PSD-95, Disc Large, ZO-1 (PDZ) domain, two coiled coil regions, and a leucine zipper. We have shown that FIG associates peripherally with the Golgi apparatus by interacting through its second coiled coil domain with a SNARE protein. Given its modular structure and the nature of its binding counterpart, we postulated that FIG plays a role in Golgi-mediated vesicular transport. Here, we report the analysis of an interstitial deletion of 240 kilobases (kb) on human chromosome 6q21 that fuses this gene to the RTK ROS (GenBank accession number M34353) in GBM cell lines. The resulting FIG-ROS fusion protein is a constitutively activated tyrosine kinase. This is the first example of a genomic event that leads to the formation of an RTK fusion protein in an astrocytoma. MATERIALS AND METHODS  Cell Culture  U118MG, U138MG, SW1088, and COS-1 cell lines (ATCC, Manassas, VA) were maintained in Dulbecco's modified Eagle's medium supplemented with 10% FCS and the antibiotics penicillin and streptomycin at 37°C in a 5% CO2 environment. Antibodies  Production and characterization of the rabbit polyclonal FIG antiserum were previously described (Charest et al., 2001). The anti-SHC antibody was a gift of Dr. Jane McGlade (Hospital for Sick Children, University of Toronto, Toronto, Canada). The anti c-myc epitope 9E10 antibody was obtained from a commercial source (Santa Cruz Biotechnology, Santa Cruz, CA). Isolation of FIG-ROS cDNA  Total RNA isolated from the U118MG cell line was reverse-transcribed by use of random hexamers with Thermoscript Reverse Transcriptase (Gibco, Carlsbad, CA). The fused region of the FIG-ROS cDNA was then amplified by PCR by use of the oligonucleotides Fig-1 AAAACTGTGATCCGAG and hRos-6 CAAGAGACGCAGAGTCAGTTT. The intracellular kinase domain of ROS was amplified by use of hRos-7 CTTCCAACCCAAGAGGAGATT and hRos-5 CAACGCTATTAATCAGACCC oligonucleotides. The full-length FIG-ROS cDNA was reconstituted by inserting the fused FIG-ROS portion into its cognate position within the FIG cDNA and by inserting the ROS intracellular kinase domain into the resulting FIG-ROS construct. The full-length U118MG-derived FIG-ROS was then cloned in pcDNA3.1/Myc-HisA in frame with the carboxy-terminal c-myc epitope (EQKLISEEDL) and 6XHis tags (Invitrogen, Carlsbad, CA). The lysine to methionine mutant (K511M) was generated by site-directed mutagenesis by use of the Quick Change Mutagenesis kit (Invitrogen). All constructs generated by PCR were fully sequenced to ascertain integrity. Genomic and Physical Mapping and Long-Distance PCR  The Genebridge 4 human–hamster radiation hybrid panel was used to map the chromosomal location of both FIG and ROS. A 10 ng sample of DNA from each member of the panel was amplified in duplicate by use of the following oligonucleotide primer pairs: hFig-19 GCAATAGATTTGAGGCCAACC with hFig-20 GGGACAGAAGGAGAAATACAC and hRos-8 GGAGTTATAAGACCACCATGA with hRos-9 CTGAGGAAAGAAGAGAAGGA. The resulting positive-negative PCR patterns for each gene were submitted to the Whitehead Institute/MIT Center for Genome Research RHMAPPER online software (http://www-genome.wi.mit.edu/cgi-bin/contig/rhmapper.pl) to determine the location of FIG and ROS. Human CEPH mega-yeast artificial chromosomes (YACs) (Dausset et al., 1992), P1 artificial chromosomes (PACs), and bacterial artificial chromosomes (BACs) were grown and harvested according to published protocols (Ausubel et al., 2001). A restriction enzyme map was created by pulsed-field electrophoresis followed by Southern blot hybridization with various probes corresponding to FIG and ROS cDNAs. LD-PCR was performed by use of Pfu Turbo polymerase (Stratagene, La Jolla, CA) with the exonic oligonucleotides hFig-34 GCAAAGACACAAGTGGGGAAATC and hRos-17 CTGTCACCCCTTCCTTGGCACTTTT. The intron/exon junctions of both FIG and ROS were determined by amplification of genomic DNA with exon-derived oligonucleotides by use of a long-range PCR amplification scheme. For those junctions where the distances between the primers were too large for amplification, we relied on the public database sequence of the area (http://www.ensembl.org/Homo_sapiens/). Interphase Fluorescence in Situ Hybridization (FISH)  BAC clone sequences were analyzed by RepeatMasker, and PCR primers were designed against genomic BAC sequence to generate 2-kb PCR-product probes devoid of repeat sequences. Probe DNA was labeled with CY3-dCTP, FITC-dUTP, or CY5-dCTP by nick translation. Interphase nuclei were prepared from cultured U118MG and primary lymphoblast cells and fixed in 3:1 methanol:acetic acid by standard methods. Preparations were denatured at 70°C for 2 min in 70% formamide/2× SSC and hybridized overnight at 37°C to denatured probe in Hybrisol VIII (Oncor, Gaithersburg, MD), then washed in 0.1× SSC/0.2 μM DAPI at 45°C for 5 min. Images were obtained with a Zeiss Axioplan II epifluorescence microscope coupled to a cooled CCD camera. Protein Preparations, Immunoprecipitations, in Vitro Kinase Assays, and Western Blot Analysis  Protein preparations were obtained as described (Charest et al., 2001). Protein extracts and immunoprecipitates were separated by SDS–PAGE and electrotransferred to Immobilon P membrane (Millipore, Bedford, MA) according to the manufacturer's directions. Western blot analysis was performed with the indicated primary antibodies at dilutions ranging from 1:500 to 1:1,000 and detected by enhanced chemiluminescence (Renaissance, NEN-Dupont, Stevenage, UK). In vitro kinase assays were performed as described elsewhere (Rodrigues and Park, 1993). RESULTS  A Peripherally Associated Golgi Protein Is Fused to the Kinase Domain of ROS in a Glioblastoma Cell Line  A short nucleotide sequence that corresponds to a portion of FIG had been previously reported as a foreign sequence fused to the region coding for the kinase domain of the RTK ROS in a glioblastoma cell line–derived cDNA library (Sharma et al., 1989). Here, we validate this hybrid FIG-ROS message and characterize the genomic event that leads to its formation. We first confirmed the existence of a FIG-ROS fusion transcript by amplifying and isolating the FIG-ROS hybrid message from the glioblastoma cell line U118MG by RT-PCR by use of oligonucleotide primers derived from FIG and ROS cDNA sequences (see Materials and Methods). A single fragment corresponding to the expected size was amplified and isolated (data not shown). Sequence analysis of the FIG-ROS RT-PCR product predicts an in-frame fusion between the carboxyl-terminal end of FIG at amino acid 411 and the juxtamembrane region of ROS at amino acid position 1881. Accordingly, the fusion protein would retain 90% of the FIG protein, including the two coiled coil domains, the leucine zipper, and the PDZ domain (Fig. 1).  Figure 1.  Figure 1.  Open in figure viewer Structural characteristics of the FIG-ROS hybrid transcript. Schematic representation of FIG and ROS proteins along with the predicted U118MG-derived FIG-ROS fusion hybrid. The arrows denote the position of the oligonucleotides used in the RT-PCR reaction that validated the existence of this fusion hybrid (see Materials and Methods). The rectangle under the PDZ domain represents the piece of FIG cDNA used as a probe on the Northern blot of Figure 2. The sizes of each transcript, the calculated molecular masses of each protein, as well as the number of amino acids of each protein are indicated. FNIII, fibronectin type III-like domain; TM, transmembrane domain. The figure is drawn to scale. Figure 2.  Figure 2.  Open in figure viewer Expression of FIG mRNA and FIG-ROS fusion transcript. Northern blot analysis of human FIG and FIG-ROS mRNA. A membrane containing 20 μg of total RNA from the glioblastoma cell line U118MG and the astrocytoma cell line SW1088 was probed as described under Materials and Methods with a 32P-radiolabeled FIG cDNA fragment corresponding to the PDZ domain. The arrow on the right corresponds to FIG's 4.5-kb message, and the arrow on the left indicates the FIG-ROS hybrid transcript of 4.1 kb. To assess the integrity of the full-length fusion transcript in vivo, we performed Northern blot analysis of total RNA isolated from U118MG cells and a control cell line (human astrocytoma cell line SW1088) by use of the cDNA sequence coding for the PDZ domain of FIG as a probe. Figure 2 demonstrates that, in the SW1088 cells, FIG mRNA is 4.5 kb in length, which is identical to that of other cell types (Charest et al., 2001). However, the size of the U118MG FIG message is 4.1 kb, which correlates exactly with the calculated size of the FIG-ROS hybrid transcript. Genomic Localization and Structure of FIG and ROS  Most oncogenic fusion proteins are the result of major chromosomal aberrations such as translocations or inversions (for reviews, see Faderl and Albitar, 2000; Alcalay et al., 2001). To determine the nature of the rearrangement between FIG and ROS, we first identified their chromosomal location by use of a radiation hybrid mapping panel (see Materials and Methods). The results show that both FIG and ROS are found at the same map location on the long arm of chromosome 6, approximately 2 cR apart from each other (Fig. 3). This result rules out translocation but not inversion nor deletion as the type of rearrangement leading to this fusion. To distinguish between these two possibilities, we physically mapped the location of both FIG and ROS by assembling YAC, PAC, and BAC contigs of a region covering both genes (Fig. 4A–C). A restriction enzyme map of these contigs was created, and the location of both FIG and ROS was determined by Southern blot analysis (data not shown). Our results indicate that FIG is located distally to ROS (Fig. 4).  Figure 3.  Figure 3.  Open in figure viewer FIG and ROS are positioned at adjacent loci on chromosome 6. Schematic representation of a G-banded human chromosome 6 onto which a genetic and radiation hybrid map corresponding to a portion of band q21 is anchored. The relative map positions of both FIG and ROS are indicated on the RH map. An array of four CEPH mega YACs containing either FIG and/or ROS is depicted. Figure 4.  Figure 4.  Open in figure viewer Physical map of both FIG and ROS. Restriction enzyme maps of YAC (A) and PAC/BAC (B) contigs including both FIG and ROS genes. The sizes of each individual YAC, PAC, and BAC are indicated. The stripped box on YACs 759F4 and 850G6 represents non-mapped portions. 32P-labeled probes from FIG and ROS hybridized to different Eag1-digested fragments. C: Composite graphical representation of the physical locus containing both FIG and ROS along with their exonic structure (vertical lines). The breakpoints established in Figure 6 are indicated by arrows. The schematics are drawn to scale. E, Eag1; A, Asc1. Figure 6.  Figure 6.  Open in figure viewer The FIG-ROS hybrid message is properly translated. Western blot analysis of FIG from different astrocytoma cell lines. Ten micrograms of total cellular protein from the indicated cell lines were subjected to Western blot analysis by use of an anti-FIG antibody as described under Materials and Methods. The arrows on the right mark the relative migration distance of FIG and FIG-ROS proteins. The asterisk in the U118MG lane points to a band corresponding to FIG-ROS proteins. The blot was stripped and reprobed with an anti-SHC antibody to control for sample loading. The transcriptional orientation of both genes as well as the distance separating them were established as follows: first, the exonic structure for both genes was elucidated (Table 1A, B). FIG is composed of 8 exons covering a genomic distance of 42 kb, and ROS is composed of 44 exons spanning a region of 137 kb. The gene structures were then integrated with the contig-derived physical map (Fig. 4C). Both genes are transcribed in the same orientation, eliminating inversion as the event leading to the rearrangement seen in U118MG cells. Comparison of the FIG-ROS fusion product sequence to the resulting map indicated that the in-frame fusion occurred between exon 7 of FIG and exon 36 of ROS.  Table 1. Genomic Organization of FIG and ROS A. Splice donor and acceptor sequences for each extron/intron boundary of FIG No.\tExon Size (bp)\tNucleotide and AA sequence\tIntron size (kp) Splice acceptor\t \tSplice donor I\t>482\t5′UTR gcc\tATG\tTCG\tGCG\tGGC\t…\tCAC\tAAG\tCTG\tCAG\tgtgagccgt\t22.937  \t \t \tM\tS\tA\tG\t…\tH\tK\tL\tE\t \t  II\t165\tctttataag\tGCA\tCAG\tTTG\tGTG\t…\tAAG\tGCA\tAAA\tTTG\tgtaagtagt\t3.546  \t \t \tA\tQ\tL\tV\t…\tK\tA\tK\tL\t \t  III\t176\tattacatag\tGAA\tAGA\tGAG\tCTT\t…\tCTG\tGCA\tGGA\tAG\tgtgtgtaaa\t1.540  \t \t \tE\tR\tE\tL\t…\tL\tA\tG\tR\t \t  IV\t165\taatatgaag\tG GTC\tCAA\tCAG\tATA\t…\tCCA\tCCA\tGGC\tCAT\tgtatgtttg\t2.511  \t \t \tV\tQ\tQ\tI\t…\tP\tP\tG\tH\t \t  V\t95\tcccaataag\tGAT\tCAA\tGAT\tTCC\t…\tATT\tTCA\tATT\tACA\tgtaagaagt\t1.123  \t \t \tD\tQ\tD\tS\t…\tI\tS\tI\tT\t \t  VI\t166\ttgttgttag\tGGT\tGGG\tAAA\tGAA\t…\tCTT\tTCT\tCAG\tCAG\tgtaaattcc\t2.536  \t \t \tG\tG\tK\tE\t…\tL\tS\tQ\tQ\t \t  VII\t181\tctgttttag\tAGA\tGGA\tGAG\tATT\t…\tAAA\tGTA\tTTA\tCAA G\tgtaaaaatc\t3.467  \t \t \tR\tG\tE\tI\t…\tK\tV\tL\tQ\t \t  VIII\t3114\tttctttcag\tGA\tTTT\tAAT\tAAG\t…\tAAA\tTCT\tTAT\tTAA\tattga 3′UTR\t   \t \t \tG\tF\tN\tK\t…\tK\tS\tY\tstop\t \t  B. Splice donor and acceptor sequences for each extron/intron boundary of ROS No.\tExon size (bp)\tNucleotide and AA sequence\tIntron size (bp) Splice acceptor\t \tSplice donor I\t>123\t5′UTR gtg\tATG\tAAG\tAAC\tATT\t…\tGTA\tACT\tAAT\tCTG\tgtaagtctc\t7029  \t \t \tM\tK\tN\tI\t…\tV\tT\tN\tL\t \t  II\t45\ttcatttcag\tGGC\tCAG\tCAG\tCTT\t…\tCAG\tAGT\tGAA\tCCG\tgtaagtaat\t2145  \t \t \tG\tQ\tQ\tL\t…\tQ\tS\tE\tP\t \t  III\t60\tctcttgcag\tTGT\tATC\tCAA\tGGA\t…\tGCT\tTTA\tAAG\tTGT\tgtaagtatt\t1713  \t \t \tC\tI\tQ\tG\t…\tA\tL\tK\tC\t \t  IV\t27\ttctatgcag\tAAT\tGAT\tACT\tTAT\t…\tAAC\tGTT\tTGT\tGAG\tgtgagttgg\t4874  \t \t \tN\tD\tT\tY\t…\tT\tV\tC\tE\t \t  V\t61\tcttccacag\tCGG\tGAG\tTCG\tTGT\t…\tGAG\tGAA\tGTA\tCTG G\tgtaagagga\t5155  \t \t \tR\tE\tS\tC\t…\tE\tE\tV\tL\t \t  VI\t149\ttttccccag\tAA\tAAT\tGCA\tGAC\t…\tACT\tTAT\tACT\tAAG\tgtatgaaag\t1004  \t \t \tE\tN\tA\tD\t…\tT\tY\tT\tK\t \t  VII\t139\tttgatctag\tACT\tGTG\tTCC\tAGA\t…\tCAT\tCCT\tCAT\tGGA G\tgtatggtgt\t6024  \t \t \tT\tV\tS\tR\t…\tH\tP\tH\tG\t \t  VIII\t202\tctcttccag\tTT\tCCT\tGAA\tACT\t…\tACT\tATC\tTAC\tAG\tgtaagaatt\t652  \t \t \tV\tP\tE\tT\t…\tT\tI\tY\tR\t \t  IX\t77\tttggaatag\tG TTT\tTCT\tATT\tGCA\t…\tTCA\tTCT\tTCA\tGCA G\tgtatgggtc\t1450  \t \t \tF\tS\tI\tA\t…\tS\tS\tS\tA\t \t  X\t123\tttttttcag\tTT\tCAA\tCAA\tGAG\t…\tCAT\tAAT\tATT\tACA G\tgtcagtatg\t270  \t \t \tV\tQ\tQ\tE\t…\tH\tN\tI\tT\t \t  XI\t185\tctcttctag\tGA\tATA\tTCT\tGTT\t…\tATC\tGAT\tGAA\tCTG\tgttagtccg\t841  \t \t \tG\tI\tS\tV\t…\tM\tD\tE\tL\t \t  XII\t98\ttttttttag\tGTA\tTGT\tGTC\tTGT\t…\tTCA\tTAC\tAAT\tGG\tgtatgtgtt\t3373  \t \t \tV\tC\tV\tC\t…\tS\tY\tN\tG\t \t  XIII\t497\tctgtgtcag\tG TAT\tGTC\tTTT\tTAC\t…\tGCC\tATA\tGGA\tGCC A\tgtgagtgct\t1317  \t \t \tY\tV\tF\tY\t…\tA\tI\tG\tA\t \t  XIV\t255\tcccttgcag\tAT\tGTC\tATC\tCTG\t…\tACC\tCTG\tGTG\tCCA G\tgtgagaaaa\t780  \t \t \tN\tV\tI\tL\t…\tT\tL\tV\tP\t \t  XV\t111\tactaaacag\tCT\tAGT\tGAA\tCCA\t…\tGGA\tAAT\tGTG\tTCA G\tgtaaacccc\t1028  \t \t \tA\tS\tE\tP\t…\tG\tN\tV\tS\t \t  XVI\t179\ttattttgag\tAC\tATG\tGAT\tTGG\t…\tAAG\tACA\tTAT\tGTG\tgtgagttaa\t2174  \t \t \tD\tM\tD\tW\t…\tK\tT\tY\tV\t \t  XVII\t192\tgttttttag\tATA\tCAA\tGGG\tCAG\t…\tTCT\tGGG\tAAA\tAAG\tgtgacaaat\t4157  \t \t \tI\tQ\tG\tQ\t…\tS\tG\tK\tK\t \t  XVIII\t101\ttctacttag\tGTA\tATT\tGCT\tCTA\t…\tCGG\tGGA\tCAG\tAG\tgtaagtaga\t12766 XIX\t218\tatgttttag\tC AGC\tGCT\tGGG\tGAT\t…\tAAG\tCCC\tCTG\tCCA G\tgtacagtat\t335  \t \t \tS\tA\tG\tD\t…\tK\tP\tL\tP\t \t  XX\t161\tatcctgtag\tGG\tAAC\tTTT\tTCC\t…\tGCT\tCAT\tTCT\tAAG\tgtatggtgt\t377  \t \t \tG\tN\tF\tS\t…\tA\tH\tS\tK\t \t  XXI\t145\tatattttag\tTTC\tTTG\tGCT\tAGT\t…\tGCA\tCCT\tGAA\tACA G\tgtacagggg\t2195  \t \t \tF\tL\tA\tS\t…\tA\tP\tE\tT\t \t  XXII\t262\tacttttcag\tTT\tCCA\tTCA\tGCA\t…\tATT\tGCC\tTTC\tCAG\tgtatgagag\t2198  \t \t \tV\tP\tS\tA\t…\tI\tA\tF\tQ\t \t  XXIII\t64\ttttctttag\tGTT\tAGG\tGCC\tTTT\t…\tTCT\tACA\tACA\tTCA G\tgtatttgtt\t330  \t \t \tV\tR\tA\tF\t…\tS\tT\tT\tS\t \t  XXIV\t203\ttgctttcag\tAA\tATC\tAAC\tCCA\t…\tTCT\tAGC\tTCT\tGAG\tgtatttcac\t1800  \t \t \tE\tI\tN\tP\t…\tS\tS\tS\tE\t \t  XXV\t207\tgttgttaag\tCTT\tTTC\tCAA\tGAT\t…\tCCT\tCTT\tTTA\tAG\tgtatgtaaa\t889  \t \t \tL\tF\tQ\tD\t…\tP\tL\tL\tS\t \t  XXVI\t287\tcctctatag\tT CGC\tTTG\tTAT\tTGG\t…\tCCT\tGCT\tATG\tCTC G\tgtaagatgc\t3459  \t \t \tR\tL\tY\tW\t…\tP\tA\tM\tL\t \t  XXVII\t179\tttttaacag\tCA\tGGA\tAAA\tACC\t…\tCAG\tCCT\tTTT\tCCA G\tgtaataaag\t8728  \t \t \tA\tG\tK\tT\t…\tQ\tP\tF\tP\t \t  XXVIII\t202\tttattttag\tAT\tAAA\tGCG\tTTT\t…\tTAT\tAGA\tATT\tCTG\tgtaagattg\t1516  \t \t \tD\tK\tA\tF\t…\tY\tR\tI\tL\t \t  XXIX\t144\tcaattacag\tGAA\tTTT\tCAG\tGAC\t…\tACT\tAAA\tAAT\tGGA G\tgtaagttat\t768  \t \t \tE\tF\tQ\tD\t…\tT\tK\tN\tG\t \t  XXX\t232\tttatttcag\tTA\tCCA\tGAA\tGCA\t…\tTAT\tGTG\tTTA\tAAG\tgtacagtga\t89  \t \t \tV\tP\tE\tA\t…\tY\tV\tL\tK\t \t  XXXI\t176\ttaccaaaag\tGTT\tCTT\tGCC\tTGC\t…\tGAG\tCTA\tCAG\tAAG\tgtagacttt\t3795  \t \t \tV\tL\tA\tC\t…\tE\tL\tQ\tK\t \t  XXXII\t168\ttttttctag\tTCC\tAAA\tTAC\tAAT\t…\tTTT\tAAG\tACA\tAAA G\tgtgagtact\t7726  \t \t \tW\tK\tY\tN\t…\tF\tK\tT\tK\t \t  XXXIII\t117\ttccctaaag\tCT\tGGA\tGTC\tCCA\t…\tCTT\tGAG\tATA\tAG\tgtatgtgtg\t2915  \t \t \tA\tG\tV\tP\t…\tL\tE\tI\tR\t \t  XXXIV\t190\ttttttgcag\tA AAG\tAGC\tACT\tTCA\t…\tATA\tTTA\tGTT\tGGA G\tgtatgttac\t1809  \t \t \tK\tS\tT\tS\t…\tI\tL\tV\tG\t \t  XXXV\t83\ttattattag\tAT\tGAT\tTTT\tTGG\t…\tCCA\tCTG\tACC\tTTT G\tgtaagtata\t2938  \t \t \tD\tD\tF\tW\t…\tP\tL\tT\tF\t \t  XXXVIa\t135\tcctttatag\tTC\tTGG\tCAT\tAGA\t…\tTAT\tGCA\tATA\tCA\tgtatgtagc\t1229  \t \t \tV\tW\tH\tR\t…\tY\tA\tI\tH\t \t  XXXVIb\t177\t \t \t \t \t \t…\tCTA\tGGC\tAAA\tGG\tgtgggcaac\t1187  \t \t \t \t \t \t \t \tL\tG\tK\tG\t \t  XXXVII\t162\tattttacag\tTACT\tCTT\tCCA\tACC\t…\tGTA\tGCA\tGTG\tAAG\tgtaatgtga\t1616  \t \t \tT\tL\tP\tT\t…\tV\tA\tV\tK\t \t  XXXVIII\t64\tgttgtatag\tACT\tTTG\tAAG\tAAG\t…\tCAT\tCTG\tATG\tAG\tgtagctctg\t916  \t \t \tT\tL\tK\tK\t…\tR\tL\tM\tS\t \t  XXXIX\t129\tacctttcag\tCAAA\tTTT\tAAT\tCAT\t…\tCGG\tATG\tGCA\tACG\tgtaggcagt\t6026  \t \t \tK\tF\tN\tH\t…\tR\tM\tA\tT\t \t  XL\t97\tttcttacag\tTTT\tTAT\tGGT\tCCT\t…\tTTC\tATT\tCAC\tAG\tgtacaattc\t739  \t \t \tF\tY\tG\tP\t…\tF\tI\tH\tR\t \t  XLI\t200\tctgccacag\tG GAT\tCTG\tGCA\tGCT\t…\tTCT\tGAT\tGTA\tTG\tgtaagttta\t1153  \t \t \tD\tL\tA\tA\t…\tS\tD\tV\tW\t \t  XLII\t134\ttattcttag\tG TCT\tTTT\tGGA\tATT\t…\tCCT\tGAT\tGAT\tCT\tgtaagttaa\t7657  \t \t \tS\tF\tG\tI\t…\tP\tD\tD\tl\t \t  XLIII\t163\ttatatttag\tG TGG\tAAT\tTTA\tATG\t…\tGAA\tAGC\tTTT\tGAA G\tgtaagtttg\t12172  \t \t \tW\tN\tL\tM\t…\tE\tS\tF\tE\t \t  XLIV\t>420\tttttcacag\tGT\tGAA\tGAT\tGGC\t…\tGGG\tTCT\tGAT\tTAA\ttagcgttgt\t3′UTR  \t \t \tG\tE\tD\tG\t…\tG\tS\tD\tstop\t \t  The physical and transcriptional orientation of both FIG and ROS suggests that a small genomic deletion or an aberrant splicing event underlies the formation of the fusion gene. To test this, we amplified U118MG genomic DNA by long-distance (LD)-PCR, by use of oligonucleotides derived from exons 7 of FIG and 36 of ROS to isolate the fusion breakpoint. A single fragment of 4.6 kb was amplified, cloned, and sequenced. Comparison of its sequence with that of the genomic sequence of the region demonstrates that a deletion occurred between FIG intron 7 and ROS intron 35, leading to a loss of 240,802 bp (Fig. 5).  Figure 5.  Figure 5.  Open in figure viewer Isolation of the FIG-ROS breakpoint in the U118MG cell line. Sequence analysis of the fusion breakpoint at the genomic level with the resulting in-frame fusion protein. Genomic DNA from U118MG cells was amplified by LD-PCR by use of oligonucleotide primers derived from exons 7 of FIG and 36 of ROS as described under Materials and Methods. A 4.6-kb DNA fragment was isolated, sequenced, and compared to the publicly available genomic DNA sequence. The location marked by an arrow represents the fusion point. The FIG-ROS Fusion Transcript Is Translationally Competent  Because not all modified transcripts are properly processed (Delaney et al., 1993; Ivell, 1994), the translational competency of the FIG-ROS message was investigated in U118MG and other glioma-derived cell lines by Western blot analysis (Fig. 6). We found that unmodified FIG is present in all cell lines as a 59-kDa protein except in the U118MG and U138MG lines where it is present as a 109-kDa species, which corresponds to the calculated molecular mass of the FIG-ROS hybrid protein. Interestingly, in the cell lines where FIG-ROS is present, we cannot detect normal FIG proteins (Fig. 6, U118MG and U138MG lanes), suggesting that the non-rearranged allele is not transcriptionally active or the deletion is homozygous in nature. The del(6)(q21q21) in the U118MG Cell Line Is a Homozygous Deletion  To resolve this issue, we performed interphase FISH by use of probes corresponding to regions flanking each side of the breakpoints and the intervening deleted portion (Fig. 7A). Co-cultures of primary lymphoblasts (serving as normal control) and U118MG cells were prepared and hybridized. U118MG and primary lymphoblast cells are readily distinguishable from one another on the basis of the size and shape of their nuclei. Figure 7B demonstrates that, in U118MG cells, two signals representing hybridization by only the flanking probes (green and blue, left inserts) are present, whereas in the lymphoblast cells, the two signals contain all three colors (green, red, and blue, right inserts). This clearly indicates that, in U118MG cells, both copies of chromosome 6, within that region, are missing the intervening signal (red), suggesting that the deletion is homozygous. To substantiate this claim, we performed a chromosome 6–wide SSLP analysis on genomic DNA from both the U118MG and U138MG cell lines and four additional astrocytoma cell lines and two CEPH family individuals as controls. The results indicate that both the U118MG and U138MG cell lines are homozygous at all loci tested on chromosome 6 (data not shown), suggesting that chromosome 6 duplicated after the microdeletion occurred.  Figure 7.  Figure 7.  Open in figure viewer Interphase FISH analysis of U118MG and primary lymphoblast cells. A: Schematic representation of the genomic region containing FIG and ROS. Probes derived from FIG (green), the deleted region in the U118MG cells (red), and from ROS (blue) are indicated by colored bars. B: Pseudo-color rendition of a triple hybridization performed on co-cultures of U118MG and primary lymphoblasts as described under Materials and Methods. The hybridization shown is representative of several experiments. The FIG-ROS Fusion Protein Displays Tyrosine Kinase Activity  Having demonstrated that the del (6)(q21q21) event creates a fusion protein, we sought to determine whether FIG-ROS is endowed with intrinsic tyrosine kinase activity. We first created an inactive kinase domain variant of FIG-ROS by mutating the ATP binding site lysine residue to methionine (K511M). Similar single-point mutations in various tyrosine kinases have all been shown completely to abrogate kinase activity. Both the unmodified, wild type (WT) and kinase inactive K511M versions of FIG-ROS were then expressed transiently in mammalian cells and immunoprecipitated for an in vitro kinase assay (Fig. 8). The results demonstrate that p109Fig-Ros WT was capable of phosphorylating itself as well as an exogenous substrate (myelin basic protein), whereas the kinase inactive version was not.  Figure 8.  Figure 8.  Open in figure viewer The FIG-ROS fusion protein displays intrinsic tyrosine kinase activity. In vitro kinase assays of unmodified (WT) and kinase inactive (K511M) variants of FIG-ROS proteins. FIG-ROS proteins were immunoprecipitated from transiently transfected COS-1 cells and subjected to an in vitro kinase assay as described under Materials and Methods. FIG-ROS autophosphorylation (upper panel) and exogenous substrate myelin basic protein phosphorylation (middle panel) are seen by autoradiography. The immunoprecipitates were Western blotted against FIG-ROS (lower panel) to control for the presence of identical amounts of FIG-ROS. The position of molecular size markers is indicated on the left. FIG-ROS proteins used in this experiment are tagged with the c-myc epitope (EQKLISEEDL), which is recognized by the monoclonal 9E10 antibody. DISCUSSION  Aberrant cDNA hybrid formation is a recurrent phenomenon associated with cDNA library construction. The original report of the FIG-ROS hybrid message did not address this issue (Sharma et al., 1989). Our first goal was to validate the existence of this fusion transcript. By performing RT-PCR and Northern and Western blot hybridizations, we unequivocally demonstrated that indeed a fusion transcript between FIG and ROS exists and is expressed in these cells.  Genomic aberrations that lead to oncogenic activations of RTKs are a common phenomenon in cancer. There are 58 known human RTKs, of which 7 have been reported in chimeric fusion transcripts (Blume-Jensen and Hunter, 2001). In all cases, the unrelated sequences that are fused 5′ to the sequence coding for the tyrosine kinase domain contain homodimerization domains that are responsible for the observed constitutive kinase activation. Nearly all examples of RTK-derived fusion proteins reported in the literature are the result of chromosomal translocations, with a minority represented by large chromosomal inversions. In some cases, the reciprocal fusions are also expressed. However, the role of reciprocal fusion proteins in the pathogenesis of cancer is not well understood (Rego and Pandolfi, 2002). To our knowledge, FIG-ROS represents the first example of an RTK fusion protein that does not arise as a result of a translocation or inversion but rather from a relatively small intra-chromosomal deletion.  The majority of reported genomic rearrangements leading to fusion of RTK genes are in hematologic malignancies, perhaps reflecting the ease of cytogenetic assessment of these cancers. However, recent advances have allowed for an increase in solid-tumor cytogenetic characterization (Pandis et al., 1994; Schwab, 1999). As such, the Cancer Genome Anatomy Project site (http://www.ncbi.nlm.nih.gov/ncicgap) reports more than 500 cases of chromosomal aberrations involving 6q21. The majority are interstitial 6q21 deletions and multi cytogenetic band-spanning intra-chromosomal deletions. More than 75% of them are of hematologic origin. In astrocytomas, there are seven reported cases of 6q21 deletions (Rey et al., 1987; Shapiro et al., 1993; Yamada et al., 1994; Sainati et al., 1996). These cytogenetic studies do not provide the resolution necessary for assessing the status of both FIG and ROS. However, they point to an important site of recurrent rearrangement in different types of cancers for that chromosome area.  Molecular genetic profiling of astrocytomas reveals that activation of RTKs is often a necessary step in the development of this cancer. Interestingly, overexpression of full-length RTKs as a result of gene amplification is the only mechanism of activation reported thus far (for a review, see Dai and Holland, 2001). We believe that FIG-ROS is the first example of a fusion RTK characterized in an astrocytoma.  To ascertain the frequency of the FIG-ROS fusion event, we screened 10 different cell lines derived from all grades of astrocytomas for the presence of FIG-ROS (Fig. 6). Two GBM cell lines (U118MG and U138MG) show an expression of the FIG-ROS locus. In fact, LD-PCR performed on genomic DNA from the U138MG cell line by use of FIG exon 7 and ROS exon 36 oligonucleotide primers revealed a breakpoint identical to that seen in the U118MG (data not shown). These two cell lines were established concurrently from the same patient (Ponten and Macintyre, 1968) (for additional information, consult the ATCC web site at www.atcc.org), strongly suggesting that the deletion was present in the primary tumor before the establishment of these two cell lines in culture and is less likely to represent a culture artifact.  Figure 6 demonstrates that unmodified FIG is not expressed in the cell lines that harbor the FIG-ROS locus. This would arise as the consequence of one of two events: the non-fused allele of FIG either is transcriptionally silenced or is deleted altogether. Examples for each mechanism can be found in the literature (Arlinghaus, 1998; Hannemann et al., 1998). Our three-colored interphase FISH experiments on U118MG cells allowed for an investigation of the status of FIG and the intervening deleted sequences on both chromosomes. Figure 7 demonstrates that, whereas the FIG genomic DNA is present on both alleles, the intergenic region is not. This, however, does not preclude the possibility of two independent deletion events. Our original RT-PCR reaction by use of FIG- and ROS-specific oligonucleotides produced a single fragment (data not shown), and the LD-PCR amplification of the fusion breakpoint in U118MG cells also yielded a single band, thus suggesting that both FIG-ROS alleles have the same breakpoint. Taken together, these results strongly suggest that the original deletion event was followed by gene duplication, perhaps involving the entire chromosome 6, and is not as likely to result from two independent events.  Very little is known about the genomic structural requirements necessary for eliciting chromosomal translocations. It has been postulated that some are mediated by repetitive elements such as Alu sequences (Kolomietz et al., 2002). However, the large number of such repetitive sequences and their widespread distribution throughout the genome suggest that the overall three-dimensional state of the chromatin in and around the breakpoints must play a role (Chakraborty et al., 2001). In fact, illegitimate recombination between RET and H4, which gives rise to PTC1, has been shown to occur in papillary thyroid cells and not in other cell types because of a thyroid cell–specific chromatin structure (Nikiforova et al., 2000). The most extensive study of breakpoint sequences has been performed on the RTK RET. Alignments of several fusion breakpoint sequences demonstrate that nearly all of the fusion events are located within a small A:T-rich sequence hot spot (Bongarzone et al., 1997). The sequence surrounding the FIG-ROS breakpoint does not contain repetitive sequences, nor is it particularly A:T- or G:C-rich. Therefore, one cannot address the mechanism that led to this fusion event, and additional rearrangements involving either FIG or ROS would help ascertain whether a fusion hot spot exists at either of these loci.  We have demonstrated that FIG-ROS displays a constitutive tyrosine kinase activity. For all other examples of fusion RTKs, it has been shown that the 5′ fusion partners serve as activation moieties because of the presence of dimerization domains within their sequences. It is expected that FIG sequences within FIG-ROS are also responsible for the observed activation. We previously showed that FIG peripherally associates with the Golgi apparatus through its second coiled coil domain (Charest et al., 2001). This portion is retained within FIG-ROS, which suggests that it may also target the Golgi apparatus. It is possible that targeting of FIG-ROS to the Golgi apparatus is sufficient to incur an activation of its tyrosine kinase domain.  Small interstitial deletions similar to that leading to the formation of the FIG-ROS locus might be more frequent than originally thought. Novel technologies such as arrayed comparative genome hybridization (Buchholz et al., 2001; Balsara and Testa, 2002) will, without a doubt, reveal the presence of similar deletions specific to different types of cancers. Knowing the molecular consequences of FIG-ROS activation might lead to a better understanding of astrocytoma formation and result in novel therapeutic leads.  ABSTRACT The human ROSI gene, which possibly encodes a growth factor receptor, was found to be expressed in human tumor cell lines. In a survey of 45 different human cell lines, we found ROSI to be expressed in glioblastoma-derived cell lines at high levels and not to be expressed at all, or expressed at very low levels, in the remaining cell lines. The ROSI gene was present in normal copy numbers in all cell lines that expressed the gene. However, in one particular glioblastoma line, we detected a potentially activating mutation at the ROSI locus. We have reported the characterization of an activated ROSJ gene which we previously called mcf3 (1). This gene was identified using an assay for human protooncogenes based on the tumorigenicity of mouse NIH 3T3 cells cotransfected with human tumor DNA (2). mcf3 encodes a protein-tyrosine kinase (ATP: protein-tyrosine O-phosphotransferase, EC 2.7.1.112) with a potential transmembrane domain. It arose from the human ROSJ gene by a rearrangement introduced during gene transfer, which deleted the putative extracellular domain of ROSI, leaving potential transmembrane and intracellular protein-tyrosine kinase domains intact (1). The human ROSI is the closest homologue of the v-ros oncogene, which is the transforming gene of the avian UR2 retrovirus (3). Structurally mcf3 and v-ros are similar to the v-erbB, v-fms, neu, and trk oncogenes (4-7). The cellular homologue of two genes in this group, v-erbB and v-fms, are known to encode the receptors for the epidermal growth factor and the macrophage colony-stimulating factor, respectively (8, 9). The cellular rosi gene may therefore encode a hormone or growth factor receptor as well. As a first step toward characterizing the normal ROSJ gene, we have screened for its expression in a variety of human cell lines established from normal and tumorigenic cells. ROSI is expressed in a high proportion of human glioblastoma cell lines and at very low levels, or not at all in the remainder of human cell lines. We did not find detectable levels of ROSI in either human brain or in a cell line established from a normal human glial cell. In most glioblastoma cell lines, the ROSI-encoded transcript was found to be 8.3 kilobases (kb) in size. Upon analysis of the ROSI locus by Southern blotting, an aberrant configuration due to a rearrangement in sequences encoding the putative extracellular domain, was seen in one particular glioblastoma cell line. MATERIALS AND METHODS ROSI Expression Studies. Total RNA from various human tumor cell lines and tissues was prepared by the guanidinium/ cesium chloride method (10). Human placenta was obtained from elective cesarean sections and superficially normal brain tissue (temporal lobe) from a patient undergoing surgery for epilepsy. The quantity of RNA was determined by measurements of the optical density, and the quality and quantity of the RNA preparations were verified on glyoxal gels (11). Twenty micrograms of total RNA was hybridized overnight at 520C to =1 fmol of 32P-labeled RNA transcript produced by Sp6 RNA polymerase (specific activity, 8 X 1 dpm/pmol) as described (12). The hybridization mixture was diluted 10-fold with buffer containing 50 mM sodium acetate (pH 4.5)/2 mM EDTA/100 mM NaCl to a final volume of 300 pl and treated with 10 units of RNase T2 for 1 hr at 30'C. The nucleic acids were precipitated with ethanol and analyzed on a 5% acrylamide gel containing Tris/borate/EDTA buffer and 8 M urea, which was autoradiographed. A complete description of the cell lines used for the expression studies, with the exception of cell lines CJ, DI, El, SN, and HF, can be obtained from Jim Loveless, the Human Tumor Cell Line Bank, Human Tumor Cell Laboratory, Memorial Sloan-Kettering Cancer Institute. Cell lines CJ, DI, El, and SN are human glioblastoma lines and primary glial cells, respectively, established by J. Shapiro at Memorial Sloan-Kettering Cancer Institute. HF cells are primary human foreskin fibroblasts obtained from J. Feramisco, Cold Spring Harbor Laboratory. Southern and RNA Blot Analysis. Southern blot analysis was done as described (13). RNA for blotting was isolated by the guanidinium/cesium chloride method and was subsequently purified on oligo(dT) columns (11). RNA blotting was done essentially as described (14). For hybridization, the same ROS1 probe synthesized by Sp6 RNA polymerase and described was used. The RNA blot was hybridized for 36 hr at 62°C in 50%6 formamide/5 x standard saline citrate (SSC) (lx SSC is 0.15 M sodium chloride/0.015 M sodium citrate)/ 10 mM EDTA/10 ,ug of carrier DNA per ml/5 x Denhardt's solution (lx Denhardt's solution is 0.02% polyvinylpyrrolidone/0.02% Ficoll/0.02% bovine serum albumin), and washed in 0.1 x SSC/0.1% NaDodSO4 once for 2 min at room temperature and four times for 30 min at 64°C. RESULTS We previously reported the isolation of a cDNA corresponding to the transcript of an activated ROS1 gene, mcf3 (1). A portion of this cDNA encoding amino acids 140-295 was subcloned into pSp65, a vector containing a Sp6 RNA polymerase promoter (12). A 32P-labeled Sp6 RNA polymerase transcript of the overall length of 730 nucleotides (nt), containing 460 nt of mcJ3 sequence complementary to ROSlencoded mRNA and 270 nt of sequence derived from the vector was synthesized and gel purified. This transcript was hybridized in solution to total RNA prepared from a variety of human tumor cell lines and was then digested with RNase (12). The RNA hybrids protected from nuclease digestion were subsequently analyzed on denaturing acrylamide gels. Fig. 1 shows, as an example, the analysis of RNA from six 9270 The publication costs of this article were defrayed in part by page charge payment. This article must therefore be hereby marked \"advertisement\" in accordance with 18 U.S.C. §1734 solely to indicate this fact. Proc. Natl. Acad. Sci. USA 84 (1987) 9271 2 3 4 5 6 7 8 9 10IM1 -622 a o S -527 - 404 mRNA per cell. mcf3 transcripts were still detected on 0.025 ,4g of the total RNA from MCF-7-3-7 cells after autoradiograms were exposed for 1 week, which indicated that the detection limit of this assay was 0.25 pg of mcf3 mRNA. In all experiments, RNA from MCF-7-3-7 was used in parallel to estimate the levels ofROSI transcripts in tested RNAs. Table 1 summarizes the result of these experiments and includes name and tissue of origin of tested cell lines together with the approximate number ofROSI transcripts found in these cells. Table 1. ROSI transcripts in tested human cell lines and tissues ROSI transcripts Human cell line per cell,* or tissue Origin =no. . -309 FIG. 1. Expression of ROSI in glioblastoma cells. Total RNA from human glioblastoma and mouse MCF-7-3-7 cells was hybridized to the 32P-labeled RNA transcript synthesized by Sp6 RNA polymerase as described. After digestion of the single-stranded RNA with RNase T2, the nucleic acids were analyzed on an acrylamide gel under denaturing conditions. The figure was photographed from an autoradiogram exposed for 36 hr. Numbers at right, size of DNA marker fragments in nt. Bar at left (P), fragment protected from RNase T2 digestion by mcf3 orROSJ-encoded RNA. Lanes: 1, 0.033 Mug of RNA from MCF-7-3-7 and 20 Mig of yeast tRNA; 2, 0.1 Mg of RNA from MCF-7-3-7 and 20 Mig of yeast tRNA; 3, 0.33 ,g of RNA from MCF-7-3-7 and 20 Mug of yeast tRNA; 4, 1 ,ug of RNA from MCF-7-3-7 and 20 ,ug of yeast tRNA; 5, 10 Mg of RNA from human U-343 MG cells and 10 Mug of yeast tRNA; 6, 20 Mig of RNA from human SW-1088 cells; 7, 20 Mg of RNA from human U-118 MG cells; 8, 20 Mg of RNA from human U-251 MG cells; 9, 20 Mg of RNA from human U-105 MG cells; 10, 20 M&g of RNA from human U-138 MG cells; M, pBR322 DNA cut with Msp I; and I, labeled-RNA transcript before RNase T2 treatment. different glioblastoma cell lines by this technique. The original transcript of 760-nt length used for hybridization is shown in lane I. RNA from MCF-7-3-7, a mouse NIH 3T3 cell line transformed by mcf3 (1), protects a fragment of 460 nt from digestion by RNase because only 460 nt of the labeled probe are complementary to mcf3-encoded RNA (Fig. 1, lane 1-4). RNA from glioblastoma cell lines protected a fragment identical in length (Fig. 1, lanes 5-10). Thus ROSJ transcripts are presented in the tested glioblastoma lines. As can be seen in Fig. 1, the protected probe resolved into a doublet on denaturing acrylamide gels. This was reproducibly observed with RNAs from all cell lines that protected this probe. The appearance of this doublet might either reflect a heterogeneity in the mcf3- and ROSI-encoded mRNA or might be an artifact due to strong secondary structures within the protected fragment. The intensity of the band on the autoradiogram corresponding to the protected fragment can be used to quantify the amount of ROSJ transcripts in the tested RNA. For standardization, we used RNA from cell line MCF-7-3-7 and determined the counts present in the protected fragment after analysis by scintillation counting. Five micrograms of total RNA from MCF-7-3-7 cells protected 5000 dpm of the labeled probe (specific activity, 8.107 dpm/pmol) or 0.06 fmol; 0.06 fmol corresponds to 50 pg of the 2.5-kb mcf3-encoded mRNA. Assuming that hybridization went essentially to completion, 50 pg of mcf3 mRNA was present in 5 ,ug of the total RNA. Assuming that 2% of total RNA is mRNA, this amount of mcf3 mRNA represents 0.05% of the mRNA; and assuming 400,000 mRNA molecules per cell, this amount would correspond to =200 molecules of mcf3-encoded U-343 MG U-118 MG El U-105 MG U-178 MG U-251 MG U-138 MG SW-1088 DI CJ A-172 A-382 SN Wiltu-1 SK-NEP-l Caki-l RD-2 575A TCCSUP VM-CUB-2 486P MCF7 734B SK-Br-3 SK-N-MC SK-N-SH SK-HEP-l LS-174T HT-29 SW-480 8387 SK-MEL-3 RPMI-4445 C4II RPMI-2650 Fadu RPMI-8402 SK-L3 U-698 M NB-58B Calu-3 SK-MES-1 SW-982 HF A431 Brain, temporal lobe Placenta ND, not detected. Glioblastoma multiforme Glioblastoma multiforme Glioblastoma multiforme Astocytoma, grade III Astocytoma, grade II Astocytoma, grade I Glioblastoma, not graded Glioblastoma, not graded Glioblastoma multiforme Glioblastoma, astocytoma, medium grade Glioblastoma, not graded Glioblastoma, not graded Normal glial cells Kidney, Wilm tumor Kidney, Wilm tumor Kidney, carcinoma Rhabdomyosarcoma Bladder, carcinoma Bladder, carcinoma Bladder, carcinoma Bladder, carcinoma Breast, carcinoma Breast, adenocarcinoma Breast, carcinoma Neuroblastoma Neuroblastoma Liver, adenocarcinoma Colon, adenocarcinoma Colon, adenocarcinoma Colon, adenocarcinoma Bone, sarcoma Melanoma Melanoma Cervix, carcinoma Nasal, squamous cell carcinoma Pharynx, squamous cell carcinoma Leukemia, lymphoplastic Leukemia, myelomonocytic Lymphoma Normal lymphocyte Lung, adenocarcinoma Lung, squamous cell carcinoma Fibrosarcoma Normal fibroblasts Epidermoid carcinoma 60 10 10 3\n",
      "341\n",
      "Acute myeloid leukemia (AML) with t(8;21) (q22;q22) is considered to have favorable risk; however, nearly half of t(8;21) patients are not cured, and recent studies have highlighted remarkable genetic heterogeneity in this subset of AML. Here we identify somatic mutations in additional sex combs-like 2 (ASXL2) in 22.7% (25/110) of patients with t(8;21), but not in patients with inv(16)/t(16;16) (0/60) or RUNX1-mutated AML (0/26). ASXL2 mutations were similarly frequent in adults and children t(8;21) and were mutually exclusive with ASXL1 mutations. Although overall survival was similar between ASXL1 and ASXL2 mutant t(8;21) AML patients and their wild-type counterparts, patients with ASXL1 or ASXL2 mutations had a cumulative incidence of relapse of 54.6% and 36.0%, respectively, compared with 25% in ASXL1/2 wild-type counterparts (P = .226). These results identify a high-frequency mutation in t(8;21) AML and identify the need for future studies to investigate the clinical and biological relevance of ASXL2 mutations in this unique subset of AML.  Go to: Introduction Acute myeloid leukemia (AML) with t(8;21) (q22;q22) is recognized by the World Health Organization1 as a unique subtype of AML within the category of “AML with recurrent genetic abnormalities.” Compared with other cytogenetic subsets of AML, patients with t(8;21) are considered a favorable risk group according to their high remission and survival rates2. At the same time, nearly half of t(8;21) AML patients are not cured,3,4 and there is a need for markers to identify patients unlikely to respond to current therapies and develop novel therapeutic approaches based on better understanding of the pathophysiology of this subset of AML.  The t(8;21) results in fusion of RUNX1 with RUNX1T1, and considerable experimental evidence reveals that full-length RUNX1-RUNX1T1 is not sufficient to induce leukemic transformation on its own.5 It is therefore posited that additional genetic alterations cooperate with RUNX1-RUNX1T1 translocations to induce overt leukemia. Identification of these additional genetic abnormalities in patients with core-binding factor (CBF) translocations has been helpful in predicting outcome and hopefully serving as novel therapeutic targets for CBF AML. Indeed, more than 50% of t(8;21) AML patients have been shown to have a mutation in KIT, FLT3, N-RAS, or K-RAS,6-10 all of which have been pursued as potential therapeutic targets. Moreover, work by Krauth et al identified that mutations in the polycomb-associated gene ASXL1 occur in 11.5% of t(8;21) adult AML patients and are associated with adverse event-free survival.11 Intriguingly, Huether et al recently identified mutations in additional sex combs-like 2 (ASXL2) in several pediatric leukemia patients.12 Here we identify a high frequency of ASXL2 mutations among CBF AML patients specifically bearing RUNX1-RUNX1T1 translocations and use data from uniformly treated CBF AML adult and pediatric patient populations to identify clinical correlates and mutational cooccurrence of ASXL2 mutations in this unique subset of AML.  Go to: Methods Patients and treatments  One hundred ten t(8;21) and 60 inv(16)/t(16;16) AML patients were included, including 75 t(8;21) and 32 inv(16)/t(16;16) adults from the CBF-2006 trial13 (A Phase 3 Trial of Systematic Versus Response-adapted Timed-Sequential Induction in Patients with Core Binding Factor [CBF] Acute Myeloid Leukemia [AML]; EudraCT 2006 005163-26; ClinicalTrials.gov NCT00428558), as well as 35 t(8;21) and 28 inv(16)/t(16;16) children from the ELAM02 trial (Treating Patients with Childhood Acute Myeloid Leukemia with Interleukin-2; ClinicalTrials.gov NCT00149162). Twenty-six de novo adult AML patients with RUNX1 mutations were also included (from the ALFA-0701 trial14 [A Randomized Study of Gemtuzumab Ozogamicin (GO) with Daunorubicine and Cytarabine in Untreated AML Aged of 50-70 Years Old]). Studies were approved by the Ethics Committee of Nimes University Hospital and by the Institutional Review Board of the French Regulatory Agency and were conducted in accordance with the Declaration of Helsinki protocol. Outcome data were updated as of December 2013, for a median follow-up of 44.8 months.  Mutational and minimal residual disease analysis  Whole-exome sequencing of DNA was performed from pretreatment as well as paired remission bone marrow mononuclear cells from 3 t(8;21) AML patients (supplemental Methods, available on the Blood Web site). All coding exons of ASXL2 were sequenced by next-generation sequencing for all patients at a median depth of 1153× (supplemental Table 1; supplemental Figure 1; supplemental Methods). All additional molecular and cytogenetic analyses are described in the supplemental Methods.  Statistical methods  Details of statistical analysis are located in supplemental Methods.  Go to: Results and discussion Discovery and recurrence of ASXL2 mutations  To identify novel mutations cooperating with RUNX1-RUNX1T1 fusions, we performed whole-exome sequencing on 3 diagnostic t(8;21) AML samples paired with a remission sample for each patient. This analysis revealed a somatic ASXL2 p.R741EfsX15 mutation restricted to the diagnosis sample of 1 patient with a variant allele frequency of 39.5% (supplemental Figure 2A) and a somatic ASXL1 p.L823X mutation in a second patient (variant allele frequency, 39%; supplemental Table 2). After validating the mutation in ASXL2 by Sanger sequencing (supplemental Figure 2B), we next performed targeted sequencing of ASXL2 across 110 pediatric and adult t(8;21) patients. Overall, 22.7% of t(8;21) patients (25/110) harbored an ASXL2 mutation (Figure 1A). The majority of ASXL2 mutations (85%) were out-of-frame frameshift mutations (supplemental Table 3). The somatic nature of these mutations in ASXL2 was verified by sequencing DNA from time of presentation and at remission in 6 patients total. Similar to previously described mutations in ASXL1,15 ASXL2 mutations were exclusively heterozygous, with the exception of a single patient with 2 different truncating ASXL2 mutations (supplemental Table 3), and were enriched in the 3′ region of the gene, occurring exclusively in exons 11 and 12. Interestingly, ASXL1 mutations (supplemental Table 3) were mutually exclusive with ASXL2 mutations (Figure 1B). Analysis of mutational cooccurrences revealed a significant cooccurrence of ASXL2 and FLT3-ITD mutations (P = .046), a feature not shared by ASXL1 mutant t(8;21) patients.  Figure 1 Figure 1 ASXL2 mutations are frequent in AML patients bearing the t(8;21) translocation and are associated with a trend for increased risk for relapse. (A) Gene diagram depicting ASXL2 mutations in adult and pediatric patients with t(8;21) AML. (B) Pattern of ... No ASXL1 or ASXL2 mutations were found in 60 inv(16)/t(16;16) AML patients (supplemental Figure 3). Likewise, although ASXL1 mutations were seen in 30.8% of RUNX1-mutated de novo AML patients, 0/26 RUNX1-mutant AML samples had an ASXL2 mutation (supplemental Figure 3).  Implications of ASXL2 mutations  Unlike mutations in ASXL1, which have been associated with advanced age in AML,16 ASXL2 mutations were similarly frequent among pediatric and adult t(8;21) AML patients (9/35 vs 16/75, respectively; P = .6; supplemental Figure 2C). Patients with ASXL2 or ASXL1 mutations had a significantly higher median white blood cell count at presentation than their wild-type counterparts (20.3 vs 15.8 vs 10.7 × 109 cells/μL, respectively; P = .029) (Table 1). Data for adult and pediatric t(8;21) AML patients are reported separately in supplemental Table 4.  Table 1 Table 1 Characteristics and outcome of t(8;21) AML patients according to ASXL2 and ASXL1 mutational status Although overall survival was similar between ASXL1 or ASXL2 mutant t(8;21) AML patients and wild-type counterparts (supplemental Figure 4A-C), patients with ASXL1 or ASXL2 mutations had a 3-year cumulative incidence of relapse of 54.6% and 36.0%, respectively, compared with 25% in ASXL1/2 wild-type counterparts (P = .226; Figure 1C and supplemental Figure 4D). Previous analysis of the CBF-2006 trial identified achievement of more than a 3 log-fold reduction in the RUNX1-RUNX1T1 transcript before initiation of second consolidation (minimal residual disease 2 [MRD2]) as the sole factor significantly influencing outcome in multivariate analysis of CBF patients.13 Among adults achieving more than 3 log-fold reduction at MRD2, the 3-year cumulative incidence of relapse was 44.4%; it was 35.7% for ASXL1 and ASXL2 mutated patients, respectively, compared with 15.2% in ASXL1/2 wild-type patients (P = .128; Figure 1D and supplemental Figure 4E).  Huether et al first noted ASXL2 mutations in 7.2% of pediatric CBF AML patients.12 Here we extend those findings and identify that ASXL2 mutations are specifically recurrent in adult as well as pediatric CBF AML patients bearing t(8;21), occurring in nearly 25% of such patients compared with 0% of inv(16)/t(16;16) patients (P = .00005). The fact that ASXL2 mutations were not identified in prior whole-exome/genome sequencing studies in AML17-21 suggests that genomic studies focused on specific clinical and genetic subsets of AML may identify additional novel mutations enriched within defined subsets of AML patients. Moreover, the discovery that t(8;21) AML patients who achieve a more than 3 log-fold reduction at MRD2 but harbor ASXL1 or ASXL2 mutations have a potentially higher risk for relapse suggests the need for further studies addressing the prognostic relevance of ASXL2 mutations in t(8;21).13 The fact that ASXL1 and ASXL2 mutations were mutually exclusive with one another, combined with the multiple unique domains in common between ASXL1 and ASXL2,22 suggests a shared mechanism of transformation for mutations in these 2 genes. Indeed, 32.7% of t(8;21) patients harbor an ASXL1 or ASLX2 mutation, making ASXL gene family alterations among the most common genetic alterations in t(8;21) AML patients. Understanding the functional basis for this frequent cooccurrence of RUNX1-RUNX1T1 translocation and mutations in ASXL1 or ASXL2 may be a critical step in furthering our knowledge of CBF AML pathogenesis and therapy.\n",
      "342\n",
      "Mutations in the E-cadherin (CDH1) gene are a well documented cause of hereditary diffuse gastric cancer (HDGC). Development of evidence based guidelines for CDH1 screening for HDGC have been complicated by its rarity, variable penetrance, and lack of founder mutations. Methods: Forty three new gastric cancer (GC) families were ascertained from multiple sources. In 42 of these families at least one gastric cancer was pathologically confirmed to be a diffuse gastric cancer (DGC); the other family had intestinal type gastric cancers. Screening of the entire coding region of the CDH1 gene and all intron/exon boundaries was performed by bi-directional sequencing. Results: Novel mutations were found in 13 of the 42 DGC families (31% overall). Twelve of these mutations occur among the 25 families with multiple cases of gastric cancer and with pathologic confirmation of diffuse gastric cancer phenotype in at least one individual under the age of 50 years. The mutations found include small insertions and deletions, splice site mutations, and three non-conservative amino acid substitutions (A298T, W409R, and R732Q). All three missense mutations conferred loss of E-cadherin function in in vitro assays. Multiple cases of breast cancers including pathologically confirmed lobular breast cancers were observed both in mutation positive and negative families. Conclusion: Germline truncating CDH1 mutations are found in 48% of families with multiple cases of gastric cancer and at least one documented case of DGC in an individual under 50 years of age. We recommend that these criteria be used for selecting families for CDH1 mutational analysis. Gastric cancer is one of the leading causes of cancer death worldwide.1 There are two major histological variants: (1) diffuse gastric cancer (DGC) and (2) intestinal type gastric cancer (IGC).2 In the diffuse form, a malignant infiltrate containing cells that have a large globule of intracellular mucin and an eccentric nucleus permeates the wall of the stomach, often spreading under histologically normal mucosa. In the more common intestinal type, the malignant cells exist in glandular structures with variable levels of differentiation. Approximately 10% of gastric cancers have a mixed phenotype; the genetic basis for the phenotypic heterogeneity in these cases is believed to be somatic Ecadherin loss in the diffuse gastric cancer component of such tumours.3 The relative frequency of diffuse and intestinal cancers varies with the population surveyed and the classification criteria used.4 A small proportion of diffuse gastric cancers, perhaps as few as 1%, occur in families with autosomal dominant gastric cancer susceptibility.5 Although only a minority of gastric cancers are hereditary, the identification of such cases offers the opportunity to provide genetic counselling, genetic testing for gastric cancer susceptibility, and gastric cancer risk reduction strategies to as yet unaffected family members at risk for the development of gastric cancer. Gastric cancer is usually lethal if diagnosed at a late stage (,20% 5 year survival) but is curable if resected before invasion through the gastric wall (.90% 5 year survival1 ). Unfortunately, most gastric cancers, and DGCs in particular, become symptomatic only when they are incurable. Effective treatment of familial gastric cancer, therefore, depends on the presymptomatic identification of individuals at risk for this cancer. The majority of families with autosomal dominant gastric cancer susceptibility have diffuse type gastric cancer. Hereditary diffuse gastric cancer (HDGC) has also been noted to occur in families in combination with breast cancer, particularly lobular breast cancer (LBC).6 In 1999 the International Gastric Cancer Linkage Consortium (IGCLC) defined hereditary diffuse gastric cancer by clinical criteria that include: (1) any family with two or more documented cases of diffuse gastric cancer in first or second degree relatives with one case under the age of 50, or (2) three documented diffuse gastric cancers in first or second degree relatives at any age.7 Germline mutations in the E-cadherin gene (epithelial cadherin, CDH1, OMIM# 192090) have been demonstrated to underlie disease in some but not all HDGC families, but not in hereditary intestinal type gastric cancer families.7 CDH1 mutations were first described in three Maori families with autosomal dominant diffuse gastric cancer.8 Since then, germline loss of function mutations have been demonstrated Abbreviations: FAP, familial adenomatous polyposis; FFPE, formalinfixed paraffin embedded; HDGC, hereditary diffuse gastric cancer; HNPCC, hereditary non-polyposis colon cancer; IGC, intestinal type gastric cancer; LCIS, lobular breast carcinoma in situ; PJS, Peutz-Jeghers syndrome 508 www.jmedgenet.com to underlie approximately 30% of HDGC in families of various ethnic backgrounds.9 10 E-cadherin is a member of the cadherin family of adhesion molecules, which are transmembrane glycoproteins mediating calcium-dependent cell–cell adhesion.11 E-cadherin is critical for establishing and maintaining polarised and differentiated epithelia during development. It also plays important roles in signal transduction, differentiation, gene expression, cell motility, and inflammation. Somatic loss of E-cadherin expression is considered to be a defining feature of both DGC and lobular breast cancer.12 13 We ascertained 43 apparent cases of hereditary gastric cancer and screened them for germline E-cadherin mutations to determine the optimum criteria for clinical testing and to identify other cancer risks associated with germline Ecadherin mutations. METHODS Families and patients A total of 43 families were identified: 18 from Canada, five from the United States and 20 from the United Kingdom. Ethnicity was known in 20/43 families. Of these, 15 were of Northern European descent. The others were of Spanish, Haida, French-Canadian, and Italian descent. The criteria used for ascertainment and testing are described in table 1. Criteria 1 and 2 are the IGCLC criteria for HDGC families. Criteria 1A and 2A are similar to 1 and 2 but have been relaxed to account for the difficulty in obtaining pathologic confirmation on referral cases. The age of onset of GC in these families ranged from 9 to 65 years of age. Other cancers seen in these families include lobular breast, colon, prostate, ovarian, head and neck, and oesophageal cancers. Examples of two of the HDGC families are shown in figs 1 and 3A. Constitutional genomic DNA was extracted by standard methods from a blood sample from one affected individual of each family, usually the proband. In two families, blood samples from an as yet unaffected family member were used (table 2); in one family, an obligate carrier was screened because DNA of an affected family member was unavailable and, in another family, an at risk first degree relative was screened. In both cases, a mutation was found. The investigations were performed after approval by the Clinical Research Ethics Board of the University of British Columbia. Informed consent was obtained from each subject, or next of kin in the case of a deceased affected individual. DNA extraction Patients’ peripheral blood samples were collected in EDTA tubes. The buffy coat layers of whole blood were treated with Puregene RBC lysis solution (Gentra Systems, MN, USA) and cell wash. A portion of white cells was used for DNA extraction and remaining cells were stored in freezer media. Puregene DNA Purification Kit (Gentra Systems) was used and the manufacture’s manual was followed for genomic DNA isolation. Paraffin embedded sections from normal tissue were deparaffinised with xylene for 5 min at room temperature and centrifuged at 14 000 rpm for 3 min. After removal of the xylene, the samples were treated with 100%, 70% and 50% ethanol series for rehydration. Pellets were dried and incubated with Proteinase K (10 mg/ml Proteinase K, 1 M KCl, 1 M Tris pH 8.0, 1 M MgCl2) at 56˚C overnight and then boiled to inactivate the enzyme and centrifuged at 14 000 rpm. The supernatant was purified by phenol/chloroform extraction. DNA was precipitated with 100% ethanol, dried at room temperature and resuspended with ddH2O. Variant detection sequencing Primers were designed using Primer 3 (http://www-genome. wi.mit.edu/cgi-bin/primer/primer3_www.cgi). Forward and reverse primers incorporated the -21M13F (TGTAA AACGACGGCCAGT) or M13R (CAGGAAACAGCTATGAC) extensions, respectively, at their 59 ends. The sequences of all primers used in this study, and their annealing temperatures, are shown in table 3. PCR reactions were carried out in a volume of 20 ml containing 10 ng genomic DNA template, 1 mM MgSO4, 0.5 mM of each PCR primer, 2 mM dNTPs, 16Pfx Amplification Buffer and 0.25 U Platinum Pfx DNA polymerase (Invitrogen, ON, Canada). Thirty cycles of 30 s at 94˚C, 30 s at a primer pair specific annealing temperature of 50–65˚C and 1 min at 68˚C were performed in programmable thermocyclers (MJ Research PTC-225 or Tetrads, Waltham, MA). A 3 ml aliquot of each PCR reaction was run on a 2% agarose gel to confirm the size, quantity, and purity of each PCR product. The remaining 17 ml of PCR product was purified using AmPure magnetic beads (Agencourt Bioscience, Beverly, MA) and eluted in a volume of 30 ml of TE (Tris-EDTA pH 8.0) according to the manufacturer’s instructions. A 2 ml aliquot of purified PCR product was then cycle sequenced using Big Dye Terminator Mix V.3 at 0.256 chemistry in a total volume of 4 ml (Applied Biosystems, Foster City, CA). Both forward (221M13F primer) and Table 1 CDH1 mutation status and study criteria Criteria no Definition Families (n) Truncating mutations (n) Splice site mutations (n) Missense mutations (n) Total number of mutations 1 Two or more documented cases of DGC in first degree relatives, with at least one diagnosed before age 50 15 5 1 2 8 (53%) 1A Two or more cases of GC, with at least one DGC diagnosed before age 50 10 3 1 0 4 (40%) 2 Three or more documented cases of DGC in first degree relatives, diagnosed at any age 10 0 0 0 2A Three or more cases of GC, diagnosed at any age, with at least one documented case of DGC 30 0 0 0 3 Isolated individual diagnosed with DGC at less than 45 years of age 90 0 0 0 4 Isolated individual diagnosed with both DGC and lobular breast cancer (no other criteria met) 00 0 0 0 5 One family member diagnosed with DGC and another with lobular breast cancer (no other criteria met) 30 0 0 0 6 One family member diagnosed with DGC and another with colon cancer (no other criteria met) 1* 0 0 1 1 (100%) 7 Intestinal gastric cancer 1 0 0 0 0 *Family member with signet ring carcinoma of the colon. E-cadherin mutations in HDGC 509 www.jmedgenet.com reverse (M13R primer) directions were sequenced. Cycle sequencing reactions consisted of 30 cycles of 10 s at 96˚C, 5 s at 52˚C (221M13F forward primer) or 43˚C (M13R reverse primer), and 3 min at 60˚C. Reaction products were precipitated with isopropyl alcohol, washed with 70% ethanol, and resuspended in 8 ml of double distilled water before loading on ABI 3700 capillary sequencers. Sequence reads were base-called using Phred and sequence reads assembled with reference sequences using Phrap.14 15 Contigs of sequence traces corresponding to each exon were examined using PolyPhred16 for detection of heterozygotes and visualised in Consed17 to facilitate verification of sequence variants by examination of individual traces. Single strand conformation polymorphism (SSCP) analysis of CDH1 for formalin-fixed paraffin embedded (FFPE) DNA samples In four families (F10, F19, F20, and F21), only DNA extracted from paraffin tissue blocks was available. In these cases, the CDH1 gene was screened by SSCP. FFPE germline DNA from six samples were screened for CDH1 mutations by PCR of all exons and intron– exon boundaries followed by SSCP analysis on 8% polyacrylamide, 10% glycerol, and 0.56TBE gels. Primer sequences were based on those reported previously,13 except for exons 1–5, which were amplified using new primers (EX1F: 59-TACGGGGGGCGGTGCCT-39; EX1R: 59-CTGGGG CGCGGAGCTTG-39; EX2F: 59-TACCCCGGTTCCATCTACC TTT-39; EX2R: 59-GCAATTTCTCGGCCCCTTTCC-39; EX3AF: 59-GTCTTTAATCTGTCCAATTTCC-39; EX3AR: 59-GCGTAGA CCAAGAAATGGAT-39; EX3BF: 59-TACAGTCAAAAGGCC TCTACG-39; EX3BR: 59-AAACAACAGCGAACTTCTCAG-39; EX4R: 59-CCAGAGAAACAGAGAAC-39; EX5F: 59-CTAATT CTTTTTCTTTCATTTTG-39; EX5R: 59-TGGGTGGATGTTACCC CG-39). PCR was performed in a 20 ml reaction volume with a buffer of 20 mM Tris-HCl pH 8.4, 50 mM KCl, 1 mM dNTPs, 1–3.5 mM MgCl2 (1.0 mM for exon 1; 1.5 mM for exons 2, 3A, 3B, 7, 8, 12, 14; 3.0 mM for exons 5, 6, 9, 11, 15; 3.5 mM for exons 10, 16), 0.45 mM each primer, 0.05 U Taq polymerase (Platinum Taq, Invitrogen Life Technologies, USA) and 2.0 mCi a- 33P. Furthermore, 5% DMSO was added to exon 1 PCR. The following cycling conditions were used: 30 s at 94˚C, 30 s at optimised annealing temperatures ranging from 51˚C to 68˚C (51˚C for exon 8, 56˚C exon 9, 57˚C exon 5, 58˚C exon 4, 60˚C exon 3A, 62˚C exons 3B, 6, 7, 10, 64˚C exons 11, 12, 14, 16, 65˚C exon 15, 66˚C exon 13, and 68˚C exons 1, 2) and 45 s at 72˚C. Reaction products were diluted 1:1 with denaturing buffer (formamide with 0.025% xylene cyanol and 0.025% bromophenol blue) and heated to 94˚C for 4 min before loading. Two sets of gel run conditions, 18 h at 4˚C and 22 h at room temperature, were used except for exon 1 where only the room temperature run was done. Products were detected by autoradiography. For exons containing any possible band shifts the PCR was repeated. Samples were run on a 2% agarose gel and purified using a MinElute gel extraction kit (Qiagen, Valencia, CA) and eluted into a volume of 10 ml according to the manufacturer’s instructions. A 2 ml sample of purified product was then sequenced using a Big Dye Terminator V.3 cycle sequencing kit (Applied Biosystems, Foster City, CA) in a final volume of 10 ml. All samples were sequenced in both directions. Cycle sequencing conditions were: initial denaturation of 96˚C for 3 min followed by 25 cycles of 96˚C for 30 s, 58˚C for 30 s, and 60˚C for 2 min. Reaction products were precipitated with ethanol/sodium acetate, washed with 70% ethanol, and resuspended in 15 ml of template suppression reagent (Applied Biosystems, Foster City, CA) before loading on an ABI capillary sequencer. RT-PCR assay for IVS11+5 splice site mutation RNA was extracted from a skin biopsy using Trizol (Invitrogen, ON, Canada). The RNA was DNAseI treated prior to being reverse transcribed. DNAseI treatment was carried out in a final volume of 10 ml containing 1 mg RNA 16 DNAseI buffer and 1 U DNAseI. Samples were incubated at room temperature for 15 min, 0.5 ml of 25 mM EDTA was added, and the samples were heat inactivated at 65˚C for 10 min. The reverse transcription was performed in a volume of 25 ml containing 16MMLV buffer, 8 mM DTT, 400 mM dNTP, random hexamers, and 400 U MMLV. The samples were incubated at room temperature for 10 min and 37˚C for 1 h followed by heat inactivation of the reverse transcriptase at 65˚C for 10 min. A 1 ml aliquot of reverse transcribed product was amplified in a final volume of 25 ml containing Gastric cancer  87  61 45 Gastric 45  90 84 Breast 75 d. ?cause d. heart prob 2  79 Gastric 79 Gastric 75 Head and neck 75 Gastric 79 Gastric 42 Gastric 49 2  Gastric 3 3 4  Gastric 55 Gastric 57 3  49 d. stroke  Uterine 4 4 Breast cancer Gyn-GU cancer Unknown cancer Figure 1 Family F4, showing autosomal dominant inheritance of gastric cancer with variable age of onset; the family has a mutation in the CDH1 gene (see table 2 and fig 2). 510 Brooks-Wilson, Kaurah, Suriano, et al www.jmedgenet.com 16PCR buffer, 1.5 mM MgCl2, 200 mM dNTP, 0.5 mM forward and reverse primer and 1 U Taq polymerase (Invitrogen, ON, Canada). Cycling conditions were as follows: initial denaturation at 94˚C for 1 min, followed by 35 cycles of 94˚C for 30 s, 60˚C for 30 s, and 72˚C for 1 min. Samples were run on a 2% agarose gel for visualisation. For the patient sample both bands were excised from the gel and purified using a QIAquick gel extraction kit and eluted into a 30 ml volume. A 1.5 ml aliquot of purified product was then sequenced using a Big Dye Terminator V.3 cycle sequencing kit (Applied Biosystems, Foster City, CA) in a final volume of 10 ml. Cycle sequencing conditions were: initial denaturation of 96˚C for 3 min followed by 25 cycles of 96˚C for 30 s, 55˚C for 30 s, and 60˚C for 2 min. Reaction products were precipitated with ethanol/sodium acetate, washed with 70% ethanol, and resuspended in 15 ml of template suppression reagent (Applied Biosystems, ON, Canada) before loading on an ABI capillary sequencer. Functional assessment of missense variants Construction of the plasmids encoding wild type and mutant E-cadherins CHO-K1 (Chinese hamster ovary) cells stably expressing the E-cadherin cDNA mutants were established by electroporation as previously described.18 19 Briefly, mutant plasmids were obtained by nested PCR using specific primers using wild type E-cadherin cloned in pcDNA3 as DNA template (G2195A: For 59-CTGTTTCTTCAGAGGAGAGCG-39, Rev 59- CGCTCTCCTCTGAAGAAACAG-39; T1225C: For 59-AATACCC CAGCGCGGGAGG-39, Rev 59-CCTCCCGCGCTGGGGTATT-39; G892A: For 59-ACAATACCGCCATCGCTTAC-39, Rev 59- GATGGCGGTATTGTAGGTGT-39). After transfection, cells were grown at 37˚C under 5% CO2 in humidified air, in a-MEM (+) medium (Gibco BRL) supplemented with 5% foetal bovine serum, 2 mM L-glutamine, 1% penicillin/ streptomycin, and 1 mg/ml geneticin. Single cell clones were selected and analysed for E-cadherin expression by Western blotting, using the human E-cadherin monoclonal antibody HECD1 (R&D System, 1/3500 dilution) for protein staining. At least two independent clones for each cell line were used in each experiment, to exclude clonal dependence of the results. Cell aggregation assay As previously reported,18 19 the ability of cells to aggregate was characterised in a fast aggregation assay. Briefly, single cell suspensions of transfected cells were incubated in an isotonic buffer containing 1.25 mM Ca2+ . Particle diameters were measured at the start (T0) and after 30 min (T30), using an LS200 particle size counter (Coulter Electronics). Results were plotted against the percentage of volume distribution. The E-cadherin-blocking antibody MB2 (1/20 dilution) was used as a control for inhibition of aggregation. Collagen invasion assay The invasion assay was performed as previously described.18 19 Briefly, collagen gels with a minimum thickness of 250 mm, were prepared in a 6 well plate (Nunc International, Rochester, NY), from a collagen type I solution (Upstate Biotechnology, Lake Placid, NY), and polymerised overnight at 37˚C. Mutant E-cadherin expressing cells (16105 ) were incubated on top of the collagen gels, for 24 h at 37˚C and invasion indices (%) expressed as ratios between the number of invasive cells inside the gel and the total number of cells, counted in at least 12 microscopic fields. RESULTS Variant detection sequencing of the CDH1 gene Ten loss-of-function mutations including two insertions, five deletions, two splice site substitutions, and one complex deletion/insertion involving a splice site were seen in 10 different families (table 2). Three missense mutations were also identified (table 2). Their functional consequences are described below. All mutation carriers found were heterozygous for a specific mutation, consistent with the autosomal dominant inheritance of HDGC in these families. No two families had the same mutation. All mutations seen were novel (not seen before). Of the three mutations involving Figure 2 CDH1 mutations found to date. Truncating and splice site mutations are found above the schema, missense mutations below it. Mutations found in the present study are indicated in red. Those marked with an asterisk have also been reported as somatic mutations in sporadic DGC. Polymorphisms are not shown. E-cadherin mutations in HDGC 511 www.jmedgenet.com Figure 3 (A) Family F11. (B) Sequencing of genomic DNA. Arrow indicates location of G to A substitution. (C) Results of RT-PCR. Lane 2 shows both the normal band (upper band) and the band with deletion of exon 11. Lanes 3 and 4 show the RT-PCR from normal control samples. Sequencing chromatogram confirms complete loss of exon 11. 512 Brooks-Wilson, Kaurah, Suriano, et al www.jmedgenet.com splice sites, the one in family F1 was a G to A substitution of the highly conserved (+1) position of the splice donor site of IVS5, a substitution almost certain to have functional consequences for the splicing of that intron, and likely to lead to in-frame deletion of exon 5 from the transcript. Consistent with this predicted effect being deleterious for the function of E-cadherin, an exon 5 deletion has previously been described as a somatic CDH1 mutation in invasive lobular breast cancer.20 The second splice site mutation was a G to A substitution at the (+5) position of IVS11 in family F11 (fig 2). This mutation leads to the deletion of exon 11 from the CDH1 transcript as demonstrated by RT-PCR (fig 3). The third splice site variant involved a deletion of the exon 8/ intron 8 splice boundary and would be expected to have an adverse functional effect. All the mutation positive families were of Northern European descent except for family F10 (Italian) and family F42 (Spanish). All the mutations in this study have been deposited into the database, dbSNP (http:// www.ncbi.nlm.nih.gov/SNP/). A total of 29 DGC families were identified that did not have CDH1 mutations within their protein-coding exons. Germline deletion of the entire CDH1 gene can be excluded for 23/29 of these families by the presence of heterozygosity for polymorphisms within CDH1 exons or in flanking intronic sequences. Deletion of single whole exons of CDH1 cannot be ruled out by this screening method; however, this mechanism is unlikely to account for all of the 29 families studied that did not show CDH1 mutations. Functional assays of missense mutations Sorting Intolerant from Tolerant (SIFT)21 uses evolutionary conservation to predict the impact of amino acid substitutions on protein function. SIFT analysis predicted that all three missense mutations identified in DGC families affect conserved residues and would be expected to have deleterious effects on protein function. Residue W409 and R732 are conserved in human, mouse, rat, Xenopus, and zebrafish, implying an important function. W409 is found in the third Table 2 Details of the gastric cancer families in the study and mutations detected Family no Cancer type, age Study criteria met Other family members with gastric cancers, n (ages) Family members with breast cancer, n (confirmed lobular breast cancer) CDH1 mutation: exon, nucleotide (amino acid) Type of mutation F1 DGC, 40 1A 3 (36, 48, 50) 1 IVS5(+1) G.A Splice site F2 DGC, 33 1 2  (35, 42) 0 Exon 12, 1779insC Insertion F3 DGC, 9 1A 1 (40) 0 Negative F4 Unaffected 1A 8 (42, 45, 49, 57, 75, 79, 79, unknown) 1 Exon 15, 2310delC Deletion F5 Colon, 74 1 4 (25 , 42 , 50, unknown) 3 Negative F6 DGC, 45 5 0 3 (1) Negative F7 DGC, 51 6* 0 0 Exon 9, T1226C (W409R) Missense F8 DGC, 62 2A 3 (58, ,60, 63) 0 Negative F9 DGC, 61 1 4 (17, 32 , 46 , 47 ) 4 Exon 9, 1212delC Deletion F10 DGC, 32 1A 6 (40, 42, 45, 50, 55, 56) 2 Exon 3, 382delC Deletion F11 DGC 48 1 2 (44 , 44 ) 5 (1) IVS11(+5) G.A Splice site F12 DGC, 31 3 0 0 Negative F13 Unaffected (colonic polyp) 1 2 (36 , 70 ) 4 (2) Exon 14, G2195A (R732Q) Missense F14 DGC, 65 2A 2 (44, 70) 0 Negative F15 Bilateral lobular breast cancer, 52 5 1 (40 ) 1 Negative F16 DGC, 24 1 1  (47) 0 Exon 13, 2061del (TG) Deletion F17 Prostate, 72 1A 5 (30, 37 , 39`,50, 70) 0 Negative F18 DGC, 32 1A 1 (40) 0 Exon 10, 1476del (AG) Deletion F19 DGC, 74 3 0 3 (2) Negative by SSCP F20 DGC, 75 2A 2 (55;72) 0 Negative by SSCP F21 DGC, 58 2A 1 (59) 0 Negative F22 GC, 61 7 6 (54`; 59; 74; 3 in 70s) 2 Negative F23 Unaffected 1A 2(42 ; 60s) 2 Negative F24 Oesophageal, 62 1A 3 (,50, 63, 65) 0 Negative F25 DGC, 34 1 2 (50, 58 ) 0 Exon 8, 1064insT Insertion F26 DGC, 36 1 2 (32 , 33) 0 Exon 7, G892A (A298T) Missense F27 DGC, ,50 3 0 0 Negative F28 DGC, 43 1A 1 (60) 0 Negative F29 DGC, ,50 3 0 0 Negative F30 DGC, 55 1 3 (44 , 52 , unknown) 2 Negative F31 DGC, ,50 3 0 0 Negative F32 DGC, 45 3 0 0 Negative F33 DGC, 48 3 0 0 Negative F34 DGC, 33 1 2 (35 , unknown) 0 Negative F35 Oesophageal, 60 1 3 (34 , 44 , 68) 0 Negative F36 GC, 46 1A 1 (unknown) 0 Negative F37 DGC, 36 3 0 0 Negative F38 DGC, 39 1 3  (33, 67, 69) 0 Negative F39 DGC, 48 3 0 0 Negative F40 DGC, 38 1 1 (54) 0 Negative F41 DGC, 27 1 3 (30 ; 32 ; unknown) 0 Exon 8, 1134 del8, ins5 Deletion and insertion F42 DGC, 41 1 2 (44 , 57) 0 Negative F43 DGC, ,50 3 0 0 Negative  DGC, confirmed by pathology report; `IGC, confirmed by pathology report; Mixed histology, confirmed by pathology report. E-cadherin mutations in HDGC 513 www.jmedgenet.com calcium binding domain of the E-cadherin protein; R732 is located at the intracellular border of the cytoplasmic domain of the molecule. Residue A298 is in the second calcium binding domain of the molecule and is conserved in all these species but Xenopus, which contains a conservative substitution of glycine for the alanine residue. To confirm the theoretical prediction of a deleterious effect on protein function for the three identified E-cadherin germline missense mutations, functional assays including cell–cell adhesion and collagen invasion have been performed in vitro. CHO E-cadherin negative cells with epithelial origin have been used as cell model system and transfected to stably express the different E-cadherin germline mutations. To exclude clonal dependence of the results, at least two independent clones for each mutation have been used for the functional analysis; results have been discussed in comparison to what was observed for cells expressing the wild type protein. For the study, only clones expressing an equal amount of protein upon transfection were selected using quantitative Western blot analysis. All three mutants resulted in the expected 120 kDa band. Results for the functional characterisation are reported in table 4. All mutations dramatically affect the ability of E-cadherin to mediate cell–cell adhesion and suppress cell invasion, supporting their pathogenic role in gastric cancer. Family ascertainment criteria and CDH1 screening To determine the optimal inclusion criteria for families for CDH1 diagnostic screening, we assessed six different criteria for ascertainment of families and determined which ones identify families likely to benefit from CDH1 screening. Eight of the 15 (53%) families who meet criterion 1 (two or more documented cases of DGC in first degree relatives, with at least one diagnosed before age 50) had a CDH1 mutation; four of the 10 (40%) families who met our revised criterion 1A (two or more cases of GC, with at least one documented DGC diagnosed before the age of 50) and one family that met criterion 6 (one family member diagnosed with DGC and another with colon cancer) carried CDH1 mutations (table 1). The single family that met criterion 6 had a family member with a signet ring carcinoma of the colon, a cancer known to be associated with somatic loss of E-cadherin expression.22 The optimal criteria for CDH1 diagnostic screening, as determined in the present study, are fulfilment of criterion 1, or the less stringent criterion 1A. Insufficient families met criterion 2, so assessment of this criterion as a means of identifying families for CDH1 diagnostic screening could not be addressed using this sample set. No mutations were identified in the family (F22) that, upon pathologic review, was demonstrated to have intestinal type gastric cancer. Twelve of the 42 DGC families included women with breast cancers. Within the 25 families who met criteria 1 or 1A there were 17 women with breast cancer; of these three had confirmed lobular breast cancer and one ductal adenocarcinoma: the pathologic subtype of the other breast cancers is unknown. Within these 25 families there were 128 women who were either sisters or mothers of individuals with gastric cancer or sisters or mothers of obligate carriers for gastric cancer susceptibility. In the 12 criteria 1 or 1A families in whom a CDH1 mutation was identified, 12 breast cancers occurred amongst 75 at risk females only half of whom would be expected to carry mutations. These 12 breast cancers occurred in five CDH1 mutation positive families with five cases concentrated in family F11 (fig 2). Although insufficient genotyping and pathological data for a formal penetrance study are available, our results appear concordant with the previous calculated breast cancer risk of 39% in germline CDH1 mutation carriers. The five CDH1 mutations from the families with HDGC and breast cancers were distributed across the gene suggesting that no genotype– phenotype correlation is present. DISCUSSION A role for E-cadherin in tumour development is now well established,23 since many human carcinomas such as skin, head and neck, lung, breast, thyroid, gastric, colon, and ovarian exhibit reduced E-cadherin expression relative to their normal cellular counterparts. Somatic loss of E-cadherin expression is seen in most diffuse gastric cancers and in lobular breast cancers, although expression is usually maintained in intestinal gastric cancers and ductal breast cancers.10 24 25 In sporadic diffuse gastric cancers and lobular breast cancers, E-cadherin loss is associated with somatic point mutations of the E-cadherin gene, loss of heterozygosity, and/ or promoter hypermethylation.10 12 22 26 The sequence of these events is unknown. In most carcinomas loss of E-cadherin is usually a late event associated with invasion and metastasis. In hereditary diffuse gastric cancers, however, E-cadherin loss is likely to be an early, and potentially an initiating event. The genetic attributable risk for gastric cancer development is similar to that for breast and colorectal cancers.27 As with these other cancers, there are families demonstrating autosomal dominant inheritance and also those that show clustering of gastric cancers for which the etiology is likely multifactorial.27 Family history is a positive risk factor for Table 3 Primer sequences used in the study and their annealing temperatures Exon Forward 59–39 Reverse 59–39 Tm (˚C) 1 M13F GTGAACCCTCAGCCAATCAG M13R TGACGACGGGAGAGGAAG 63 2 M13F TGTTGGTTTCGGTGAGCAG M13R GGTGTGGGAGTGCAATTTCT 61 3 M13F CGCTCTTTGGAGAAGGAATG M13R AACGGTACCAAGGCTGAGAA 58 4 M13F GCTGTCTGGCTAGGTTGGAC M13R TTTTCCCTTTCTCTCCTTGG 58 5 M13F GAAAGGGAAAAGACCCAGTG M13R GGATCCAGCATGGGTTGAC 58 6 M13F GCCCCTTCTCCCATGTTT M13R CTTTGGGCTTGGACAACACT 56 7 M13F GGGCAGAATTGGATTAAGCA M13R TGTCCACGGGATTGAGCTA 57 8 M13F CTGGGCTAGGCCAAAGGT M13R CCATGAGCAGTGGTGACACTT 57 9 M13F AATCCTTTAGCCCCCTGAGA M13R AGGGGACAAGGGTATGAACA 61 10 M13F CCAAAAGCAACAGTTAAGGA M13R CAAATGACAAAATGCCATGA 56 11 M13F AGCGCTTAAGCCGTTTTCA M13R GAGGGGCAAGGAACTGAACT 60 12 M13F AAGGCAATGGGGATTCATTA M13R ATTGAAAGGTGGGGATCTGG 59 13 M13F CAATTTTATTCTGGAATGAGCTTTT M13R CAGGAAATAAACCTCCTCCATTT 55 14 M13F GCTGCTTCTGGCCTTCTTTA M13R GCTGTTTCAAATGCCTACCTCT 55 15 M13F TGAACATAGCCCTGTGTGTATG M13R TTTTTGACACAACTCCTCCTG 58 16 M13F AGACTTCTTGCCCCAGATGA M13R AACCACCAGCAACGTGATTT 63 Forward primers have 221M13F tail attached for sequencing: TGTAAAACGACGGCCAGT; reverse primers have M13R tail attached for sequencing: CAGGAAACAGCTATGAC. 514 Brooks-Wilson, Kaurah, Suriano, et al www.jmedgenet.com both diffuse and intestinal gastric cancers.7 The heritable factors that account for the increased risk in families of index cases include both highly penetrant mutations as well as polymorphisms that may modify host–environment interactions.27 28 Autosomal dominant gastric cancer is both clinically and genetically heterogeneous. Most families show only diffuse cancers.7 Gastric carcinomas are seen in families with germline p53 mutations,29 and hereditary non-polyposis colon cancer (HNPCC).29 However, 85% of HNPCC associated gastric cancers are of the intestinal type.30 31 Gastric cancer is also seen in both Peutz-Jeghers syndrome (PJS) and familial adenomatous polyposis (FAP). In PJS, hamartomatous polyps in the stomach occur in approximately 24% of patients.32 33 However, the risk of gastric cancer is small. Although gastric adenomatous polyps occur in about 10% of individuals with FAP, the risk of gastric cancer is small.33–35 In 1998, Parry Guilford and colleagues described germline truncating E-cadherin mutations in three Maori families with autosomal dominant diffuse gastric cancer.8 Soon afterward, we reported similar mutations in three families of Northern European origin.9 E-cadherin acts as a tumour suppressor gene in these families, with the loss of expression of the wild type E-cadherin allele usually occurring through promoter hypermethylation.26 To date, including this study, 27 truncating E-cadherin mutations have been reported, only one of which has been found twice.9 10 36–42 These mutations are present in approximately one third of families with two or more first degree relatives having confirmed cases of diffuse gastric cancer.7 A penetrance analysis of E-cadherin mutations in 11 families showed an estimated cumulative risk of gastric cancer by age 80 years of 67% for men (95% confidence interval (95% CI) 39 to 99) and 83% for women (95% CI 58 to 99).43 We have screened a series of 42 DGC families for germline mutations in the CDH1 gene. Thirteen families (31% overall) had either overt loss of function mutations or missense mutations likely to have functional consequences for the E-cadherin protein. Functional assays have demonstrated that all missense mutations do not retain the wild type E-cadherin functions of cell adhesion and suppression of invasion. All of the CDH1 mutations observed in these families are novel, underlining the absence of founder effect previously noted for CDH1 mutations underlying this familial cancer.9 10 When a detailed comparison of CDH1 mutation status v family ascertainment criteria is performed it is apparent, however, that families that have CDH1 mutations are those with a strong family history of early onset diffuse gastric cancer. Some 48% of such families had germline CDH1 mutations in this study. The nature of the families in this sample set makes it difficult to determine whether a strong family history alone (in the absence of early onset disease; criterion 2) also correlates with the presence of CDH1 mutations. The only family (F7) with an E-cadherin mutation (W409R) that does not meet criteria 1or 1A, included an individual with a histologically defined signet ring cell cancer of the colon (SRCC). We have previously observed an SRCC in a 35 year old male with another CDH1 missense mutation (Thr340Ala).10 Signet ring carcinomas of the colon are extremely rare, representing nine of 3000 (0.003%) of colon cancers in a series of colon cancers from Singapore,44 27 of 2589 (0.01%) cases from Canada,45 and 24 of 323 (7%) colon cancer from the USA.46 The higher proportion of signet ring carcinomas in the latter study reflect that this study cohort was derived from a clinical trial versus being population based. Although up to 50% of SRCCs have microsatellite instability, a feature associated with HNPCC, little is known about the pathogenesis of microsatellite stable cases other than loss of E-cadherin expression.22 46 The occurrence of this rare subtype of colon cancer in two CDH1 mutation positive families is insufficient to prove an association. However, given the known role of loss of somatic E-cadherin expression in SRCCs and the reported presence of intramucosal signet ring carcinoma, a lesion previously described in the prophylactic gastrectomy specimens of germline CDH1 mutation carriers, a role for CDH1 germline mutation in the susceptibility to SRCC is feasible45 and warrants further investigation. Our study and others5 18 show that a single individual with early onset DGC without a strong family history is unlikely to carry a CDH1 germline mutation. Our current data set does not contain enough families with both lobular breast cancer and DGC to determine if CDH1 screening is advisable for such families. Breast cancer was reported in the first E-cadherin associated HDGC families,8 but the histologic type was not determined. Keller6 reported a family in which a germline truncating mutation carrier had a lobular breast cancer. In sporadic lobular breast cancers, loss of E-cadherin expression is commonly seen and somatic mutations are frequently seen in combination with loss of heterozygosity at the E-cadherin locus. In the 11 families included in the IGCLC penetrance study,40 nine cases of breast cancer were reported. For women, the cumulative risk of breast cancer was 39% (95% CI 12 to 84). Lobular cancer was present in all five cases where pathology was available. No CDH1 mutations were identified in the germline DNA of 65 women with lobular breast carcinoma in situ (LCIS),47 six families with both breast and gastric cancers,47 or in 13 patients with familial lobular breast cancer.24 In this study, 17 cases of breast cancer occurred in families with germline E-cadherin mutations. However, only 4/17 cases of the breast cancers could be confirmed pathologically. Of these, three were the lobular type. These data are supportive but insufficient to conclusively prove an association between lobular breast cancer and germline E-cadherin mutations. Since breast cancers were also seen in the CDH1 negative families it is possible that the cancer susceptibility in these families may be caused by CDH1 mutations missed by bi-directional sequencing or mutation events in genes that, like CDH1, have relevance for both DGC and LBC. These results are key for the development of optimal guidelines for the management of individuals within HDGC families. At present, the best preventive treatment for HDGC Table 4 Functional characterisation of missense mutations Aggregation Invasion Particle diameter (mm) T0 Particle diameter (mm) T30 Invasion index (%) Mock cells* 16.52 19.25 10.8¡1.2 Wild type 17.95 1067.00 1.2¡0.2 A298T 21.60 26.21 13.5¡0.8 W409R 20.00 32.21 22.05¡0.7 R732Q 22.08 29.22 13.2¡1.1 *A heterogeneous cell population characteristic of cells transfected with the empty vector. E-cadherin mutations in HDGC 515 www.jmedgenet.com is prophylactic gastrectomy, a procedure that is life saving but has an associated high morbidity and mortality rate. The importance of identifying the genetic basis of cancer susceptibility in HDGC families has been underscored by the recent observation of early gastric cancers in 10 prophylactic gastrectomy samples from germline E-cadherin mutation carriers.48 49 The findings suggest that prophylactic gastrectomy is currently the best treatment for germline mutation carriers and that current endoscopic screening techniques are inadequate. In 50–70% of HDGC families, cancer susceptibility is caused by unknown genetic factors and so predictive testing is not possible at present. Missense mutations remain problematic for clinical management because, without both extensive family data and functional data, it is difficult to predict the pathogenicity of a missense mutation. In the absence of such data, it may not be appropriate to use E-cadherin missense mutation status to segregate risk within an HDGC family. A total of 29 families were found not to have a mutation in the coding region of the E-cadherin gene. These families could carry a CDH1 mutation for which our assay is insensitive, such as a small intragenic deletion involving one or more exons. However, it may be more likely that most of these families bear mutations in other HDGC susceptibility genes that remain to be discovered. This view is consistent with previous observations of genetic50 51 and linkage36 heterogeneity in this disease. Unfortunately only a minority of these families are of adequate size to permit linkage mapping. Identification of novel genes that underlie HDGC will therefore rely on combination strategies that employ candidate gene screening, perhaps in combination with linkage mapping in those few families of sufficient size. The 29 CDH1 mutation negative families identified in this study represent a resource for the identification of novel HDGC genes. Identification of such genes will substantially improve the diagnostic opportunities for at risk individuals in such families, in addition to providing information about the biochemical pathways that underlie this disease. In conclusion, germline CDH1 mutations are present in close to 50% of families in which there is both a documented DGC in an individual diagnosed at age 50 or younger and a family history of gastric cancer. Testing individuals who do not meet these criteria is unlikely to yield CDH1 mutations.  Hereditary diffuse gastric cancer is caused by germline mutations in the epithelial cadherin (CDH1) gene and is characterized by an increased risk for diffuse gastric cancer and lobular breast cancer.  Objective To determine whether recurring germline CDH1 mutations occurred due to independent mutational events or common ancestry.  Design, Setting, and Patients Thirty-eight families diagnosed clinically with hereditary diffuse gastric cancer were accrued between November 2004 and January 2006 and were analyzed for CDH1 mutations as part of an ongoing study at the British Columbia Cancer Agency. Twenty-six families had at least 2 gastric cancer cases with 1 case of diffuse gastric cancer in a person younger than 50 years; 12 families had either a single case of diffuse gastric cancer diagnosed in a person younger than 35 years or multiple cases of diffuse gastric cancer diagnosed in persons older than 50 years.  Main Outcome Measures Classification of family members as carriers or noncarriers of CDH1 mutations. Haplotype analysis to assess recurring mutations for common ancestry was performed on 7 families from this study and 7 previously reported families with the same mutations.  Results Thirteen mutations (6 novel) were identified in 15 of the 38 families (40% detection rate). The 1137G>A splicing mutation and the 1901C>T (A634V) missense/splicing mutation occurred on common haplotypes in 2 families but on different haplotypes in a third family. The 2195G>A (R732Q) missense/splicing mutation occurred in 2 families on different haplotypes. The 2064-2065delTG mutation occurred on a common haplotype in 2 families. Two families from this study plus 2 additional families carrying the novel 2398delC mutation shared a common haplotype, suggesting a founder effect. All 4 families originate from the southeast coast of Newfoundland. Due to concentrations of lobular breast cancer cases, 2 branches of this family had been diagnosed as having hereditary breast cancer and were tested for BRCA mutations. Within these 4 families, the cumulative risk by age 75 years in mutation carriers for clinically detected gastric cancer was 40% (95% confidence interval [CI], 12%-91%) for males and 63% (95% CI, 19%-99%) for females and the risk for breast cancer in female mutation carriers was 52% (95% CI, 29%-94%).  Conclusions Recurrent CDH1 mutations in families with hereditary diffuse gastric cancer are due to both independent mutational events and common ancestry. The presence of a founder mutation from Newfoundland is strongly supported.  Conclusions Published online June 3, 2007 (doi:10.1001/jama.297.21.2360).  Gastric cancer is the second most common cause of cancer death worldwide.1 Two major histopathological variants of this cancer have been described: an intestinal type and a diffuse type.2 A decline in the overall incidence of gastric cancer can be attributed primarily to a decrease of the intestinal variant of gastric cancer with the diffuse type remaining stable or possibly even increasing.3  The histopathologic appearance of diffuse gastric cancer specimens reveals a pattern of isolated, mucin-filled tumor cells within the wall of the stomach.4 Decreased expression of the epithelial cadherin (CDH1) gene (Online Mendelian Inheritance in Man No. 192090) in cases of diffuse gastric cancer may account for morphological differences between intestinal and diffuse variants. Epithelial cadherin is a transmembrane glycoprotein and plays a major role in epithelial architecture, cell adhesion, and cell invasion. CDH1 was first associated with gastric cancer when somatic mutations were identified in diffuse gastric cancer specimens.5 Since then, germline mutations in CDH1 have been found in families with autosomal dominant susceptibility to hereditary diffuse gastric cancer (HDGC).6- 17  An autosomal dominant cancer susceptibility syndrome, HDGC has an average age of onset of 38 years for clinically detectable diffuse gastric cancer.7,18 Germline mutations in CDH1 are found in 30% to 40% of clinically defined families with HDGC from different ethnic backgrounds, predominantly from low-incident populations.18,19 The CDH1 mutation spectrum is heterogeneous and includes point mutations, small deletions, and insertions distributed along the entire coding sequence (Table 1).8,20 The identification of CDH1 mutations offers the opportunity of cancer risk-reduction strategies for unaffected at-risk individuals.19,21 Along with a risk of diffuse gastric cancer, there is an excess of lobular breast cancer in families with clinically defined HDGC.8,9,22- 24 This is not unexpected because loss of CDH1 expression is a cardinal feature of lobular breast cancer and HDGC and both somatic CDH1 mutations and promoter hypermethylation are found frequently in lobular breast cancer but only rarely in infiltrating ductal carcinoma.25,26  In this study, mutation status was assessed in 38 families with diffuse gastric cancer. The analysis of the 38 families revealed 13 different CDH1 mutations in 15 of these families (40% detection rate). Eight of these mutations were classified as recurring; these included 2 mutations (1 novel, 1 previously reported10) that were found in more than 1 family in this study and 6 additional mutations that had been previously reported.8,11,19,27- 29 This was unexpected because earlier mutation screenings typically reported only novel mutations in single families. Of the more than 50 published pathogenic germline CDH1 mutations, only the 187C>T (R63X) (arginine63stop) truncating mutation,9,30 the 1018A>G (T340A) (threonine340alanine) mutation,12,31 the 1792C>T (R598X) (arginine598stop) truncating mutation,9,13,30 the 1901C>T (A364V) (alanine634valine) mutation,19,27 and the 1003C>T (R335X) (arginine335stop) truncating mutation9,32 have been found in more than 1 family. The latter was found in 3 families due to independent mutational events as demonstrated by haplotype analysis.9 In this study, haplotype analysis was used to determine if 5 of the 8 recurrent mutations within this series occurred due to independent mutational events or founder effects. For 4 of these mutations, at least 2 apparently unrelated families shared the mutation and an associated haplotype at the CDH1 locus. A novel recurrent CDH1 mutation was initially found in 2 families from the southeast region of Newfoundland and subsequently in 2 additional families, both originating from this same region, implying a founder effect. The implications of these data on the genetic testing and counseling of families with HDGC, as well as the novel opportunities created by the discovery of founder mutations, appear herein.  Methods Patient Ascertainment As part of an ongoing study at the British Columbia Cancer Agency, 38 families diagnosed clinically with HDGC were accrued between November 2004 and January 2006 and were analyzed to estimate the frequency and penetrance of mutations in the CDH1 gene. When available, ethnicities for families are included (Table 2). In cases in which families are of multiple ethnicities, ethnicity is ascribed based on the branch of the family that carries the CDH1 mutation. Ethnicities were self-reported by probands. Ethnic background is relevant to ascribing founder effects to mutations.  The majority of families met previously published inclusion criteria.8,9 Six families had a single case of diffuse gastric cancer diagnosed in a person younger than 35 years; 26 families had at least 2 gastric cancer cases with 1 case diagnosed as diffuse gastric cancer in a person younger than 50 years; 5 families had at least 3 gastric cancer cases at any age, 1 of which was confirmed as diffuse gastric cancer; and 1 family had 2 confirmed diffuse gastric cancer cases, both diagnosed in persons older than 50 years. The majority of families were referred to this study through genetics clinics in North America. Pedigrees and medical records were collected by genetic counselors at the site of origin. When available, pathology and other medical reports were reviewed by study investigators for probands and affected family members.  Mutation screening results were reported to the proband (or next of kin if the proband was deceased) with appropriate counseling and psychological support by the referring genetic counselor. Genetic counseling and carrier testing was then offered to other adult family members. When prophylactic gastrectomies were performed, pathology reports were requested; only abnormalities from gastrectomies analyzed according to a previously published protocol were considered.33  Investigations were performed after approval by the clinical research ethics board of the University of British Columbia. Informed consent was obtained from each individual, or next of kin in the case of a deceased individual.  Mutation Analysis Genomic DNA was purified from peripheral blood leukocytes, using the Puregene DNA Purification Kit (Gentra Systems, Minneapolis, Minn) according to the manufacturer's instructions. The entire coding sequence and flanking intronic portions of the CDH1 gene were screened by denaturing high-performance liquid chromatography analysis using a 3500HT WAVE DNA fragment analysis system (Transgenomic Inc, San Jose, Calif). The denaturing high-performance liquid chromatography and polymerase chain reaction (PCR) primers, PCR settings, amplicon length, and resolution temperatures for denaturing high-performance liquid chromatography analysis were as described previously.9 Bidirectional sequencing of purified PCR products (Qiagen, Mississauga, Ontario) was performed using the ABI BigDye Terminator Sequencing Kit version 3.1 (Applied Biosystems, Foster City, Calif) and an ABI Prism 3100 or 310 Genetic Analyzer (Applied Biosystems).  Missense Mutation Analysis To assess epithelial cadherin function, aggregation and Matrigel invasion assays (BD Biosciences, Mississauga, Ontario) were performed as previously described.14,19 Briefly, Chinese hamster ovary (CHO) cells, which do not express endogenous CDH1, were transfected with constructs expressing wild-type, R749W, or E781D CDH1. Aggregation was assessed using an inverted microscope following incubation of cells in an agar gel at 37°C for 24 to 48 hours. Prior to assessment of invasion, cells were incubated on top of collagen gels for 22 hours at 37°C. Invasion indices are expressed as ratios between the number of invasive cells inside the gel and the total number of cells. Missense mutations were further analyzed using SIFT (Sorting Intolerant From Tolerant) software version 2 (Fred Hutchinson Cancer Research Center, Seattle, Wash), which uses evolutionary conservation to predict the potential effect of amino acid substitution.34,35  Splicing Analysis Splicing effects were predicted using NNSPLICE software version 0.9 (Berkeley Drosophila Genome Project, Splice Site Prediction by Neural Network, Berkeley, Calif) and NetGene2 software (Center for Biological Sequence Analysis, Lyngby, Denmark).36,37 These programs assign ideal splice site scores of 1 for donor and acceptor sites.  Reverse transcriptase–PCR (RT-PCR) was performed on putative novel splicing and missense/splicing mutations. The RNA was extracted from white blood cells or gastrectomy specimens using Trizol (Invitrogen, Burlington, Ontario) according to the manufacturer's instructions. Using the SuperScript First-Strand Synthesis System (Invitrogen), 1 μg of RNA was reverse transcribed. The PCR amplification was performed with gene-specific primers (EC5F5′-TGGCCAAGGAGCTGACACAC-3′ and EC7R 5′-GATCTTGGCTGAGGATGGTG-3′ for 715 G>; EC11F5′AACTGGCTGGAGATTAATCCG-3′ and EC15R 5; CAATTTCATCGGGATTGGCAG-3′ for 2195G>A and 2164 + 5G>A) using a PTC-200 DNA Engine Thermal Cycler (MJ Research, Waltham, Mass). The PCR products were either cloned into pCR-TOPO vectors (Invitrogen) or purified from 2% agarose gels followed by sequencing as described above.  Missense mutations with potential splicing effects that had not been previously described were subjected to minigene analysis to validate splicing effects predicted in silico. Exons 6 and 14 with flanking intronic sequence were amplified using specific primers (MG6F5′-TATTACTCGAGATGCCTGGCCAGGAAGTCATATATT GAT-3′ and MG6R5-ATATAGATATCCCGGGAGTTTGAAAGTAGCCTGGA-3 for exon 6; MG14F5′-TATTACTCGAGCCCACTCCCCATAGCTGGTTAT G and MG14R 5′-ATATAGATATCAATAAGTAGGCTCTCCAAGACACTG-3′ for exon 14) containing either XhoI (forward primer) or EcoRV (reverse primer) linkers and unidirectionally cloned into pSPL3 vectors. Selected constructs were transfected into COS (Cercopithecus aethiops) cells with the Lipofectin Reagent (Invitrogen). The RNA was isolated after 24 hours using Trizol (Invitrogen) according to the manufacturer's instructions. One microgram of RNA was reverse transcribed using MMLV (moloney murine leukemia virus) RT (Invitrogen) followed by PCR amplification using vector-specific primers (SD65′-TCTGAGTCACCTGGACAACC-3′ and SA25′-ATCTCAGTGGTATTTGTGAGC-3′). The PCR products were analyzed on a 2% agarose gel and sequenced as described above.  Haplotype Determination Haplotype analyses were performed using microsatellite markers D16S318, D16S3107, D16S3025, D16S496, D16S3067, D16S3095, and D16S752 surrounding the CDH1 region (Figure 1). Each marker was amplified with PCR using a PTC-200 DNA Engine Thermal Cycler (MJ Research). The PCR products were electrophoresed in an ABI Prism 3100 Genetic Analyzer (Applied Biosystems) and analyzed with Genescan version 3.1 software (Applied Biosystems). Primer sequences for microsatellite markers D16S3025, D16S496, D16S3095, and D16S752 have been previously reported.9 Individual amplification primers for markers D16S318, D16S3107, and D16S3067 are listed in the genome database (http://www.gdb.org).  Penetrance Analysis Pedigree data from families F14, F15, SF6, and SF7 were used to estimate the penetrance of CDH1 using a conditional likelihood approach implemented in the Mendel program, as previously described.38 The conditional likelihood of the pedigree was maximized given the phenotype of the family at ascertainment and the genotype of the index case. The models were parameterized in terms of age-specific log relative risks of gastric and breast cancer compared with background population incidence rates. The cumulative risk of gastric cancer for men and women and the breast cancer risk in women were then estimated from the cumulative incidence estimates. Only cases of clinically detected gastric cancer were considered in this analysis; patients undergoing prophylactic gastrectomy were censored at age of surgery and so occult carcinomas did not contribute to penetrance estimates. To protect the anonymity of the families shown in the pedigrees, unaffected individuals are represented by a sex-neutral symbol (circle within a square). Information about affected individuals is fully displayed. Individuals are not shown in birth order. Complete pedigree data will be made available by the authors upon request for specific research use consistent with protection of participant privacy and prior consent.  Results Mutation Screening Mutational analysis of CDH1 was performed for 38 families. Fifteen of the 38 families were found to have CDH1 germline mutations (40% detection rate) (Table 2). All mutation carriers were heterozygous for a specific mutation, consistent with autosomal dominant inheritance. Fourteen of the 15 families who tested positive for a mutation had at least 2 gastric cancer cases with 1 case diagnosed as diffuse gastric cancer in a person younger than 50 years (14/26; 54%). One of the 15 families had 2 confirmed diffuse gastric cancer cases, both diagnosed in persons older than 50 years. Thirteen different mutations were found in these 15 families: 6 truncating mutations (2 nonsense, 3 deletions, 1 insertion), 2 splicing mutations, 3 missense/splicing mutations, and 2 missense mutations. Six of the 13 mutations identified herein are novel: 1682-1683insA; 1913G>A (W638X) (tryptophan638stop); 2164 + 5G>A (splicing); 2245C>T (R749W) (arginine749tryptophan); 2343A>T (E781D) (glutamic acid781aspartic acid); 2398delC. The remaining 7 mutations have been previously reported: 715G>A (G239R) (glycine239arginine, missense/splicing); 283C>T (Q95X) (glutamine95stop); 1137G>A (splicing); 1397-1398delTC; 1901C>T (A634V, missense/splicing); 2064-2065delTG; 2195G>A (R732Q) (arginine732glutamine, missense/splicing).8,10,11,19,27- 29 The 6 truncating mutations were assumed to be deleterious.  To confirm pathogenicity of the 2 novel missense mutations, 2245C>T (R749W) and 2343A>T (E781D), aggregation and invasion were assayed using CHO cells. These are standard methods to assess function of epithelial cadherin missense variants through comparison with wild-type epithelial cadherin, which promotes cell aggregation and inhibits cell invasion.14,19 Introduction of wild-type CDH1 into CHO cells resulted in cell aggregation. However, CHO cells expressing the 2 missense variants failed to produce compact cellular aggregates in soft agar. The CHO cells expressing wild-type CDH1 had minimal invasion into collagen matrices as expected (mean [SD] invasion index, 0.63% [0.15%]). By contrast, the CHO cells expressing the 2245C>T (R749W) and 2343A>T (E781D) mutations had significantly higher mean (SD) invasion indices of 5.4% (0.15%) and 7.8% (0.36%), respectively (P<.01 for both mutations by normal test), demonstrating impaired function of epithelial cadherin. The missense mutation 2245C>T (R749W) is predicted to be pathogenic using SIFT prediction software and this arginine residue is conserved in homologous proteins in cows, rats, mice, dogs, and Chinese soft-shell turtles. The SIFT analysis of the 2343A>T (E781D) mutation suggested that this substitution would be tolerated. However, we believe that this mutation is pathogenic because the functional in vitro results indicate that it affects epithelial cadherin function.  Of the 2 splicing mutations, 1 had been shown previously by RT-PCR to affect splicing (1137G>A).10 The second (2164 + 5G>A) was predicted by in silico analysis to affect splicing at the native donor splice site. This was validated by us through RT-PCR analysis on RNA extracted from white blood cells, which revealed that this mutation caused deletion of exon 13 (228 base pairs).  The 3 mutations classified as missense/splicing have been described previously as pathogenic missense mutations through in vitro functional assays and SIFT analysis.8,14,19,28 However, further analysis by ourselves or others (in the case of 1901C>T [A634V]39) indicated that these mutations affect splicing. In silico analysis of 715G>A (G239R) and 2195G>A (R732Q) mutations predicted that both of these mutations created new acceptor splice sites. By contrast, similar analysis of the 2 missense mutations (2245C>T [R749W] and 2343A>T [E781D]) did not reveal any splicing effects. Our RT-PCR analysis of the 715G>A (G239R) and 2195G>A (R732Q) missense/splicing mutations using RNA extracted from white blood cells and a gastrectomy specimen, respectively, confirmed the in silico–predicted splicing effects. The 715G>A (G239R) mutation caused deletion of the first 29 base pairs from exon 6 and the 2195G>A (R732Q) mutation resulted in complex splicing and deletion of 32 base pairs at the start of exon 14. Previous RT-PCR analysis of the 1901C>T (A634V) missense/splicing mutation demonstrated that this nucleotide substitution causes deletion of the last 37 nucleotides of exon 12 and subsequent frameshift.39 Novel splicing effects were further validated by minigene analysis, which showed that the 715G>A (G239R) mutation created a preferential splice site in exon 6 and that the 2195G>A (R732Q) mutation activated a cryptic acceptor splice site in exon 14. In comparison, minigene analysis of the 22245C>T (R749W) missense mutation showed normal splicing only.  Haplotype Analysis Two mutations were found in multiple families in this study. The F3 and F4 families had a single nucleotide substitution (1137G>A) previously reported to result in complex splicing errors.10 Families F14 and F15 were both found to have a single nucleotide deletion (2398delC) in exon 15. Both families F14 and F15 came from the Canadian province of Newfoundland.  To determine whether recurring mutations represented independent CDH1 mutational events or were due to common ancestry, haplotype analysis was performed on 5 of the 12 described mutations. These include the mutations that were found in more than 1 family in this series (2398delC and 1137G>A) along with 3 other previously reported mutations (1901C>T [A634V], missense/splicing; 2064-2065delTG; 2195G>A [R732Q], missense/splicing) found in this study for which samples from previously reported families were available (Table 3).8,10,19,27  Seven microsatellite markers surrounding the CDH1 gene were selected and haplotypes were determined in several family members. These markers span a region of approximately 4 Mb (4 million base pairs of DNA) and are tightly linked to the CDH1 gene (Figure 1). The disease-associated haplotypes were deduced from allele segregation in known carriers by inspecting genotypes in the families.  The missense/splicing mutation 2195G>A (R732Q) occurred in a previously reported family (SF5)8 and in family F11 from this study. Three siblings from family F11 and an individual from family SF5 were subjected to haplotype analysis. Families F11 and SF59 had haplotypes differing at 2 microsatellite locations (D16S3107 and D16S3095) but were identical at the 3 microsatellite markers most tightly linked to CDH1 (Table 4). These data are inconclusive because the mutation could either have arisen independently in these 2 families or it is an ancient mutation with differing haplotypes resulting from 2 recombination events occurring at the more distal markers. This latter possibility is a low-frequency event; however, it may have occurred over many generations.  The missense/splicing mutation 1901C>T (A634V) had been previously reported to occur in a Portuguese individual (family SF2) diagnosed with signet ring cell carcinoma of the stomach at the age of 30 years,19 a Portuguese family with 2 cases of diffuse gastric cancer (family SF3),27 and a colorectal carcinoma–derived cell line.39 This mutation was found in family F7 of English descent. Analysis revealed that families SF2 and SF3 share virtually common haplotypes but differ at microsatellite D16S318 (Table 4), the most centromeric of the markers analyzed (Figure 1). Because D16S318 is relatively distant (approximately 1.5 Mb) from the CDH1 gene, we believe that a single recombination event has occurred in 1 of the families but that these families share a common ancestor. The putative haplotype detected in families SF2 and SF3 differed at markers D16S318, D16S3025, D16S496, D16S3067, and D16S3095 from the haplotype cosegregating with the mutation in family F7. We conclude that the mutation 1901C>T (A634V) occurred due to an independent mutational event in family F7.  The 1137G>A splicing mutation had been previously reported in a Brazilian family (SF1) of Italian origin10 and was reported in 2 families, F3 and F4. Haplotype analysis performed on an individual from each of families F3 and F4 and on 2 affected individuals from family SF1 revealed a common haplotype shared among families F3 and F4 and a different haplotype in family SF1 (Table 4). This result indicates that families F3 and F4 could share a common ancestor. However, the presence of this same mutation in family SF1 is the result of an independent mutational event.  The 2064-2065delTG mutation was identified in family F9 in this study and also had been previously identified in family SF4 of German descent.8 Haplotypes were determined in an individual from family F9 and in 2 individuals from family SF4. A putative common haplotype was found segregating with the mutation in both families, indicating that families F9 and SF4 share common ancestry. Despite extensive efforts, we have been unable to connect the pedigrees of these 2 families.  The 2398delC mutation (Figure 2) was found in families F14 (Figure 3) and F15 (Figure 4), both of Irish/English ethnicity and originating from Newfoundland. This same mutation was subsequently found in an additional family (SF6) from Newfoundland (Figure 3)8 and in family SF7 (Figure 4) in which the mother of the proband originates from Newfoundland. This cluster of 4 families has 29 cases of gastric cancer, 16 cases of breast cancer, and 31 unaffected mutation carriers, of which 12 have had prophylactic gastrectomies. Family SF6 had previously tested negative for CDH1 mutations8 but was retested when it was discovered that they originated from Newfoundland. The original false-negative result occurred due to polymorphic sites within primers used for the initial analysis (data not shown); in this study, different primer sets were used for sequencing. Although families SF6 and SF7 showed no apparent relationship to each other or to the original 2 Newfoundland families carrying this mutation, haplotype analysis revealed that all mutation carriers shared a common haplotype. This strongly suggests that the 2398delC mutation is a CDH1 founder mutation. Furthermore, all 4 families are from communities within a 100-mile radius. Families F14 and SF6 are both from an abandoned island community off the southeast coast of Newfoundland and family SF7 is from the neighboring French islands of St Pierre and Miquelon. We have been able to connect the pedigrees of family F14 and SF6 (Figure 3); however, it is uncertain whether this connection is relevant to the inheritance of the CDH1 mutation.  Penetrance Analysis The cumulative risk of gastric cancer by age 75 years in the 4 families with the 2398delC mutation was estimated to be 40% (95% confidence interval [CI], 12%-91%) for males and 63% (95% CI, 19%-99%) for females (Table 5). The cumulative risk for breast cancer for females by the age of 75 years was found to be 52% (95% CI, 29%-94%).  Clinical Follow-up We have performed 129 CDH1 mutation carrier tests for asymptomatic individuals (age range, 18-77 years) from 11 of the families (9 of the original 15 families plus 2 additional families carrying the 2398delC mutation) cited in this study. Sixty-one of these tests were from family F14. Fifty-one individuals had positive carrier tests (51/129; 40%) and 23 (45%) of these underwent prophylactic gastrectomies. A pathology report was available for 18 of the specimens and occult cancers were detected in 12 cases (67%). A single individual from family F15 had diffuse gastric cancer detected by endoscopic random biopsy while she was considering a prophylactic gastrectomy.  Comment Gastric cancer is one of the most common gastrointestinal tract malignancies worldwide and increased gastric cancer rates are observed in hereditary nonpolyposis colon cancer, particularly in East Asian families, and in individuals with Li-Fraumeni syndrome or Peutz-Jegher syndrome.40- 42  The germline defect in CDH1 underlying HDGC was first identified through a combination of linkage and mutation analysis in 3 Maori families with early onset diffuse gastric cancer6 and subsequently confirmed in families with different ethnicities.30 Combining results from this and 2 previous studies, we have found CDH1 mutations in 53.1% (24/49) of families with 2 or more cases of gastric cancer, with at least 1 diffuse gastric cancer case diagnosed before age 50 years.8,9 These criteria may be appropriate for HDGC selection in low-incidence populations but is likely too permissive for use in countries such as Japan or Korea where gastric cancer is much more common.  For families with HDGC, the uptake for carrier testing is high and current cancer risk reduction strategies include endoscopic screening and prophylactic gastrectomy. The choice of prophylactic gastrectomy is supported by the finding of early diffuse gastric cancer in the tissue specimens of 21 of 22 CDH1 mutation carriers who chose to have gastrectomies despite receiving negative endoscopy results.9,43- 45 Furthermore, in this study 23 (45%) of the 51 asymptomatic CDH1 mutation carriers chose to have prophylactic gastrectomies and additional individuals are either awaiting surgery or are consulting with surgeons to discuss this possibility. A pathology report was available in 18 cases and occult cancer was confirmed in 12 (67%). The age range of individuals undergoing this procedure ranges from 36 to 60 years. The uptake of prophylactic gastrectomies is likely due to poor early detection methods and severity of disease at the time of clinical presentation. One of the difficulties associated with early diagnosis through screening is that early lesions are often multifocal, less than 1 mm in diameter, and underlie normal gastric mucosa.33 Positron emission tomographic scanning or chromoendoscopy-directed biopsies have recently been recommended as a screening method for at-risk individuals.46 However, these detection methods appear to have low sensitivity and failed to detect early gastric cancer in 6 patients with HDGC.47 In this study, a single individual from family F15 was found to have occult cancer from an endoscopic biopsy; this is the only asymptomatic mutation carrier from this study known by us to have cancer detected by this method. Frequent screening by chromoendoscopy as the primary gastric risk reduction strategy in CDH1 mutation carriers is being used for the previously described Maori family.6,48 Further study of the risks and benefits of chromoendoscopic screening and prophylactic gastrectomy, including analysis of the long-term quality-of-life implications of gastrectomy will be required for the development of evidence-based management guidelines for this cancer susceptibility syndrome.  Prophylactic gastrectomy is a major surgical procedure associated with predictable morbidity and potential mortality rates (<1%) and thus should only be performed by expert surgeons after counseling by geneticists, dieticians, and gastroenterologists. For HDGC mutation carriers, our recommendations conform to published guidelines,18 with consideration of genetic testing and commencement of screening in the late teens or early 20s. The timing of gastrectomy is an individual decision. The penetrance of CDH1 mutations by age 75 years was somewhat lower in men (40%) than in women (63%), a difference that is similar to a previous study in which the cumulative risk of gastric cancer by age 80 years was 67% (95% CI, 39%-99%) for men and 83% (95% CI, 58%-99%) for women.38 The biologic basis for the increased penetrance among females is not known and is in variance with diffuse gastric cancer incidence rates in males and females, which are approximately equal.22 The high frequency of occult cancers in prophylactic gastrectomies in comparison with the incomplete penetrance of clinically detected cancer suggests that not all early lesions proceed to lethal carcinomas. The age range for gastric cancer in this series is 18 to 82 years and the youngest fatality from gastric cancer was at age 20 years. The youngest reported death from HDGC was in a 16-year-old adolescent from the Maori family in which the first CDH1 germline mutations were described.6 We recommend that gastrectomy should be considered in the early 20s in male mutation carriers whereas female mutation carriers would need to consider the dietary ramifications of prophylactic gastrectomy on pregnancy in their decision-making process and thus may wish to delay the procedure. For mutation carriers awaiting gastrectomy or for those in whom it is not a desirable option for personal or medical reasons, screening every 6 months is recommended. Both more refined penetrance data and an understanding of the effect of gastrectomy on quality of life will be required for the generation of evidence-based management guidelines.  For families with multiple cases of diffuse gastric cancer, mutational CDH1 screening for diagnostic purposes is difficult due to the large size of the gene and distribution of mutations along the entire gene. Recurring mutations have been described, for example the 1003C>T (R335X) mutation, which was found in 3 families due to separate mutational events in each.9,32 Other mutations (187C>T [R63X], 1018A>G [T340A], 1792C>T [R598X], and 1901C>T [A634V]) have been independently identified by multiple groups, however further analysis has not been performed to determine whether these are due to common ancestry or represent separate mutational events.9,12,13,19,27,30,31  In this study, 38 HDGC families were screened for CDH1 mutations and 13 different mutations were identified in 15 families. The mutations identified in this study are similar to mutations that previously have been identified in terms of their distribution throughout the CDH1 gene and mutation type (Table 1).  Five mutations were subjected to haplotype analysis to determine whether they occurred due to independent events or as founder effects. Four of these mutations (1901C>T [A634V], 1137G>A, 2064-2065delTG, and 2398delC) were found to be associated with identical or near identical haplotypes in more than 1 HDGC family. Of these, only the 2398delC mutation was identified multiple times within a genetically homogeneous, ethnically or geographically defined population, and thus is a likely founder mutation. Both the 1901C>T (A634V) and the 1137G>A mutations also occurred as independent mutation events in additional families. Cleft lip and congenital scalp aplasia cutis have been described in siblings with positive test results for the 1137G>A splicing mutation from the previously reported family.10 In adults, haploid insufficiency of epithelial cadherin results in diffuse gastric cancer susceptibility with no accompanying abnormalities in wound healing or skin integrity. However, at 4 to 6 weeks of embryo development, which coincides with lip and palate development, CDH1 is expressed in the nasal prominence.10 There were no known individuals with skin closure abnormalities in either family F3 or F4, which suggests that other genetic or environmental effects (eg, folate insufficiency) may influence this phenotype.  The 2398delC mutation was found on a common haplotype in 4 families (F14, F15, SF6, and SF7; Figure 3 and Figure 4). Aside from the previously reported Maori family with HDGC,6 a founder mutation in the CDH1 gene has not been described in the literature for HDGC syndrome. Two of these families originate from or near an abandoned island community off the southeast coast of Newfoundland and the other 2 families are from nearby communities (<100 miles away).  The incidence of mortality from gastric cancer in Newfoundland is the highest in Canada at 1.7 times the Canadian average.49 Within Newfoundland, the regions this family comes from (Avalon Peninsula and southeast coast) are the highest-risk areas within the province. Consanguinity is known to be increased in the Southern Avalon Peninsula,50 which increases the possibility of marriage between 2 carriers of the 2398delC mutation. Based on the phenotype of CDH1 knockout mice, which are lethal during the embryonic stage,51 we expect that homozygotes for this mutation would not be viable. However, such marriages would increase the heterozygous carrier frequency in sibships. While it is not known whether there is a higher than normal proportion of diffuse gastric cancer cases in this province, an early study describing increased incidence of gastric cancer in Newfoundland52 shows a single figure of a gastric cancer sample whose pathology matches that of signet ring diffuse gastric cancer identical to that seen in CDH1 mutation families. There has been speculation that environmental factors such as seabird populations53 may contribute to this increased risk. However, the finding of a founder population from this region suggests a genetic explanation for these increased gastric cancer rates. If we are correct in attributing the increased rate of gastric cancer in Newfoundland to these founder mutations, then we would expect that more than 40% of the gastric cancer cases in this province could be hereditary. This is in strong contrast to the rest of North America where we estimate that less than 5% of gastric cancer cases are caused by autosomal dominant cancer susceptibility genes.18,21  In addition to the consequences of CDH1 mutations for HDGC patients, female CDH1 mutation carriers are at increased risk for developing lobular breast cancer.8,9,22- 24 Penetrance studies have shown that female CDH1 germline mutation carriers have an additional risk of breast cancer, particularly lobular breast cancer, in about 39% (95% CI, 12%-84%) of patients.38 Within the families with founder mutations from this study, the cumulative risk of breast cancer was 52% (95% CI, 29%-94%), which is slightly higher than the previously reported risk.38 Family F14 carrying the 2398delC mutation has 13 members with breast cancer; all 3 available pathology reports were confirmed to have invasive lobular breast cancer. Family SF68 with the same mutation also has 2 breast cancer cases with 1 case confirmed as lobular breast cancer (Figure 3). Two branches of this extended family, along with family F11, had been misclassified as breast cancer families due to clustering of lobular breast cancer cases and subsequently tested negative for BRCA1/2 mutations (Figure 3 and Figure 4). This suggests that families with negative test results for BRCA1/2 in whom the proband has lobular breast cancer should be reevaluated for HDGC and screened for CDH1 mutations.  Magnetic resonance imaging with mammography may be more sensitive than mammography alone for detection of breast cancer in high-risk women.54 Our current recommendations for all female carriers of CDH1 mutations include referral to high-risk breast cancer screening clinics with regular magnetic resonance imaging and mammography testing starting before age 40 years. Because early lobular breast cancer cases are estrogen-receptor positive and prophylactic tamoxifen trials preferentially reduced risk for estrogen-receptor positive cancer55 and lobular carcinoma in situ,56 the recommendation of tamoxifen for women with germline CDH1 mutation carriers may be prudent if not contraindicated. Prophylactic mastectomy also could be considered and may be particularly appealing for asymptomatic female carriers from families with multiple cases of lobular breast cancer.  This extended family with the 2398delC founder mutation is a useful resource for determining risk-modifying factors in the development of diffuse gastric cancer or lobular breast cancer, such as diet or genetic polymorphisms, and for studying secondary genetic events that lead to cancer formation. The identification of this mutation could permit population-based screening of diffuse gastric cancer within specific regions of Newfoundland. Testing for the founder mutation will be particularly valuable for potential HDGC families from Newfoundland in which there is no known living relative with either diffuse gastric cancer or lobular breast cancer from whom a high-quality peripheral blood DNA sample can be obtained for full CDH1 genetic screening because testing a single mutation can be readily performed on suboptimal DNA from archival tissue samples. In this study, we have identified other ancestral mutations; however, they cannot be used to streamline CDH1 testing because they are not associated with a geographically or culturally definable founder population.  Our results confirm that between 30% and 40% of families with a positive family history of gastric cancer and more than 50% of families with 2 diffuse gastric cancer cases diagnosed prior to age 50 years will carry germline mutations in the CDH1 gene. We also have identified a potential genetic etiology for the increased incidence of gastric cancer in the Canadian province of Newfoundland and have demonstrated a rapid uptake of asymptomatic carrier tests and prophylactic gastrectomies in families testing positive for a CDH1 gene mutation.\n",
      "343\n",
      "A 41-year-old man with no familial history of gastric cancer was diagnosed as with intramucosal early gastric cancer. Two months after the first endoscopic submucosal dissection for signet-ring cell carcinoma (SRCC), the appearance of previously unrecognized multiple erosions of SRCC was noticed. Pathological examination after a total gastrectomy and Roux-en-Y reconstruction with D2 lymph node dissection were performed. Postoperative pathological examination revealed 90 and more lesions, which tempted the attending pathologist to refer to genetic tests for the predisposition though the patient had no familial history of gastric cancer. There were no mutations in all the exons of CDH1 with conventional DNA sequencing, but multiplex ligation-dependent probe amplification, and reverse transcription-polymerase chain reaction analyses disclosed a large genomic deletion (c.1566-?_1711+?del), leading to the mRNA with loss of the exon 11. Among family members, his son was found to be a carrier of this change, while his parents were negative for the familial CDH1 mutation, implying that this change is a de novo event in the proband. The present report is the first description of a de novo large genomic deletion of CDH1 gene associated with early-onset diffuse gastric cancer. When the clinician finds a relatively-young patient who has multiple SRCCs, CDH1 germline mutation should be considered, even for patients with no familial history.  Keywords: Endoscopic submucosal dissection, Signet-ring cell carcinoma, Stomach neoplasms, CDH1, Mutation Go to: Introduction Endoscopic submucosal dissection (ESD) is the preferred treatment method for locally dissecting early gastric cancer (EGC) with a negligible risk of lymph node metastasis [1]. Currently nearly 100 % curability has been achieved for EGC with radical surgery, thus ESD is also expected to be performed at a comparable success rate. Metachronous recurrence of EGC after ESD is an important problem, but recurrence can be detected with adequate follow-up endogastroduodenoscopy (EGD), and most patients can receive early endoscopic treatment again [2]. In Japan, the previous consensus has recommended curative ESD that are indicated only for small intramucosal differentiated-type EGC [1]. Recently, the criteria have been expanded to other pathological types such as the undifferentiated-type under regulated conditions [3–5].  In Japan, gastric cancer is a major cause of cancer death, and Helicobacter pylori infection is considered to be a major cause of gastric cancer [6]. Although epidemiological studies have shown that there are patients who have a significant familial history of gastric cancer, the pathogenesis of familial gastric cancer (FGC) has not been clearly explained. CDH1 germline mutations are associated with the development of autosomal cancer syndrome, namely hereditary diffuse gastric cancer (HDGC); about 25–30 % of families fulfilling the clinical criteria for HDGC established by the International Gastric Cancer Linkage Consortium (IGCLC) have constitutional alterations of the CDH1 gene [7]. The CDH1 gene maps to chromosome 16q22.1 and consists of 16 exons that encode E-cadherin, which is a major component of adherens junctions. Carrying the abnormal CDH1 gene confers more than an 80 % lifetime risk of developing gastric cancer.  In 2012, the first conclusive case of a de novo CDH1 germline mutation (c.1792 C>T (R598X)) in a woman whose daughter was diagnosed with early-onset diffuse gastric cancer was reported [8]. On the other hand, CDH1 germline mutations can also be identified in sporadic early-onset gastric cancer in less than 4 % of patients who are 35 years of age at the time of diagnosis, presenting as de novo mutations [9]. Although the incidence of gastric cancer is relatively high in Japan, the detection rate of CDH1 germline mutations in Japanese patients with FGC is low compared to that in European patients [10].  We report the first case of early-onset diffuse gastric cancer associated with a de novo large genomic deletion of CDH1 gene, and the pathological feature including gross appearance of the surgical specimen of the stomach was pathognomonic.  Go to: Case report An asymptomatic 41-year-old man was admitted to our hospital with EGC revealed by cancer screening EGD. There were no abnormalities in the physical examination or laboratory data, and no medical history of malignant tumors. He smoked 20–60 cigarettes a day for 22 years. An enzyme immunoassay to qualitatively detect serum IgG antibodies to Helicobacter pylori was negative. A protruding lesion, 5-mm in size, with erosion was observed on the greater curvature of the antrum and no other lesions were observed endoscopically. Biopsy from the erosion indicated signet-ring cell carcinoma (SRCC). ESD was performed on the lesion in accordance with clinical trial (JCOG1009/1010) [5], resulting in en bloc resection. Precise pathological examination of 3- to 4-mm slices from the entire specimen revealed a lesion, 13 × 13 mm in size, which was a SRCC limited in the mucosal layer (Fig. 1a, b). Lymphovascular involvement was not observed and horizontal/vertical margins were negative. According to the above-mentioned pathological analysis, the definitive diagnosis was f Stage IA SRCC (L, Gre, 0-IIb, pT1a, cN0, cH0, cP0, cM0) and resection was thought to be curative [1].  Fig. 1 Fig. 1 a Lesion resected with ESD. Resected specimen was 40 × 38 mm. The lesion was step-sectioned at 3- to 4-mm intervals, and then examined pathologically. Although SRCCs at the biopsy site disappeared (as an arrow indicates), ... Follow-up EGD performed 2 months later revealed no local recurrences at the ESD site, but three new erosions appeared on the gastric angle and antrum (Fig. 2a, b). Biopsies from these erosions again indicated multiple lesions of SRCC. Apparently they were likely to be intramucosal, but there was a high incidence of metachronous multiple lesions. We considered the possibility of HDGC that we could not control multiple SRCCs by repeated ESD or distal gastrectomy similar to a previous report [11]. After the explanation about the risk for recurrence or metachronous lesions in any part of the stomach, next month, a total gastrectomy and Roux-en-Y reconstruction with D2 lymph node dissection were carried out as the patient wished. Postoperative pathological examination revealed that 90 and more lesions, each less than 5-mm in size, showing type 0-IIb SRCCs in the mucosal layer, and they were distributed in the whole stomach (Fig. 3). The lesions may be missed by a cursory look. The reason for the discrepancy between preoperative endoscopic findings and final pathological diagnosis would be the difficulty in locating endoscopically SRCC foci in the superficial lamina propria, just beneath normal-looking surface epithelium or gastric pits. The tumor cells had a low proliferative index and reduced expression of E-cadherin with immunohistochemistry (Fig. 4c). We suspected the involvement of a genetic abnormality of CDH1, although he had no familial history of gastric cancer and did not meet clinical criteria for HDGC defined by the IGCLC [7].  Fig. 2 Fig. 2 EGD view 2 months after ESD. EGC is visible as the arrows indicate. a The posterior wall of the lower gastric body, near the gastric angle. b The lesser curvature of the gastric angle and the distal antrum Fig. 3 Fig. 3 Surgical specimen. Postoperative pathological examination revealed 90 and more lesions. White circles indicate infiltrating SRCCs Fig. 4 Fig. 4 a, b Germline copy number change in the CDH1 gene of a patient. Since there were no point mutation of CDH1, the DNAs were tested for large genomic deletions in the CDH1 gene using the SALSA P083-B1 CDH1 multiplex ligation-dependent probe amplification ... After informed consent was obtained, DNA was extracted from the peripheral blood of the patient. First, a polymerase chain reaction (PCR)-direct sequencing analysis was performed for all the exons of CDH1, but no mutations were identified. Next, a multiplex ligation dependent probe amplification (MLPA) analysis was performed as previously described [12] (Fig. 4a). This revealed that the exon 11 of the CDH1 transcript was heterozygously deleted in this case; thus, a heterozygous c.1566-?_1711+?del germline mutation existed in the proband. For validation, CDH1 copy numbers in exons 11 were measured using TaqMan copy number assays according to the manufacturer’s instructions, and the copy number was 1 (the normal copy number is 2) (Fig. 4b).  A program of genetic counseling and DNA testing was provided to other members of his family (Fig. 5). We could not contact his brother. Predictive genetics testing was performed on his parents and his son. Among family members, his 15-year-old son was a carrier of the mutation, and his 71-year-old father and 65-year-old mother were negative for the familial CDH1 mutation. These results lead to the conclusion that a large genomic deletion of CDH1 gene identified in this proband and in his son probably occurred de novo in the proband. The proband’s 13-year-old daughter has not yet been tested.  Fig. 5 Fig. 5 Pedigree of family with germline CDH1 alteration. Squares indicate males; circles indicate females. Solid symbols indicate the gastric cancer patient. Symbols with a slash indicate deceased individuals. An arrow points to the proband. The numbers below ... Go to: Discussion E-cadherin is a calcium-dependent cell membrane protein involved in cell–cell adhesion and confers cell polarity [13]. CDH1 mutations are associated with a risk of early-onset diffuse gastric cancer, in addition to increased risk of lobular breast cancer and signet-ring colon cancer [7]. There was a report of six patients from one large kindred who underwent total gastrectomy for CDH1 mutation without any evidence of gastric cancer from detailed preoperative imaging studies, including high magnification endoscopy with methylene blue chromoendoscopy and multiple random biopsies. Each patient was found to have multiple sites of invasive diffuse SRCC without lymph node metastases [14]. The guideline recommends advising CDH1 mutation positive patients with normal gastric biopsies to undergo prophylactic gastrectomy once the genetic testing results are known and once individuals are older than 20 years. Rare cases of clinically significant diffuse gastric cancer have been reported in affected families before the age of 18, but the overall risk of diffuse gastric cancer before the age of 20 is low [15]. In the present case, the proband’s children should receive surveillance EGD in the future.  To date, over 100 kindreds with CDH1 germline mutations have been identified, and few cases of a concluded de novo CDH1 germline mutation have been reported [8]. However, we think there are many unidentified patients who were not tested or in whom mutations could not be identified with conventional DNA sequencing. Importantly, 4 % of these mutation positive families exhibited large germline deletions of CDH1 that were not detectable with conventional DNA sequencing [16]. Analyses of large genomic deletions with MLPA are recommended in cases where simple DNA sequencing is unrevealing [16, 17]. We propose that screening for germline large rearrangements of CDH1 should be included in CDH1 genetic testing for FGC in young patients with diffuse gastric cancer [12].  The present report is the first description of a de novo large genomic deletion of the CDH1 gene associated with the production of a truncated CDH1 protein. The c.1566-?_1711+?del germline mutation observed in this case has not been reported previously either as a somatic or a germline mutation, so we think this is novel and further characterization of its RNA would be interesting. Importantly, it was verified that this mutation passes the gene on to the proband’s offspring without change.  The ESD technique has rapidly permeated worldwide for treating EGC, but obviously it is not suitable for multiple occurrences such as in CDH1 mutation positive patients. As this instructive case shows, unusual morphologic feature may suggest prepared endoscopists should suspect genetic predisposition. In such a case, genetic counseling should also be provided. A prudent follow-up is necessary not only for the proband but also for the proband’s family. When a clinician finds the relatively-young patient who has multiple SRCCs, unusual conditions such as CDH1 germline mutation should be considered, even if patients have no familial histories of gastric cancer.  A cDNA encoding the cell-cell adhesion molecule E-cadherin was transfected into highly invasive epithelial tumor cell lines of dog kidney or mouse mammary gland origin. Transfectants with a homogeneously high expression of E-cadherin showed a reproducible loss of activity in two types of in vitro invasion assays. Invasiveness of these transfectants could be reinduced specifically by treatment with anti—E-cadherin antibodies. In vivo, they formed partly differentiated tumors, instead of fully undifferentiated tumors. Alternatively, a plasmid encoding E-cadherin-specific antisense RNA was introduced into noninvasive ras-transformed cells with high endogenous E-cadherin expression. The resulting down-regulation, albeit partial, rendered the cells Invasive. These data provide direct evidence that E-cadherin acts as an invasion suppressor molecule.  Gastric cancer (GC) is the second leading cause of cancer-related deaths worldwide, affecting close to one million people per year.1 The majority of patients with GC present with advanced disease at diagnosis (stages III and IV), rendering the prognosis extremely poor, with a 5-year overall survival rate of less than 25%.2–4 Most patients with GC are asymptomatic during the early stages of disease, thus delaying the initial diagnosis. Although, some studies have pinpointed improvements on patient survival with perioperative and adjuvant treatment modalities, surgical resection is still the primary curative treatment for localized GC; however, less than 50% of patients are eligible for resection.5,6 During the last decade, several genetic and epigenetic changes underlying gastric carcinogenesis have been elucidated.7,8 Nonetheless, their clinical usefulness is limited because of the lack of systematic genetic-clinical correlations.  GC is histologically classified into two major types, intestinal and diffuse.9 The intestinal type is characterized by well-differentiated glandular structures, whereas the diffuse type consists of individually infiltrating neoplastic cells that may have signet ring morphology, does not form glandular structures, and has a poorer patient prognosis.10,11 The majority of GCs (90%) appear in a sporadic setting. The remaining 10% of GCs show familial clustering, and of these, only 1% to 3% constitute hereditary forms.12 Among GCs with familial aggregation and with histology information, the following specific syndromes can be identified: familial intestinal GC (FIGC) and hereditary diffuse GC (HDGC; Online Mendelian Inheritance in Man No. 137215).13 So far, no germline defects have been associated with FIGC, whereas germline mutations and deletions of E-cadherin (CDH1) are the underlying genetic defect in 45% of families with clinical diagnosis of HDGC.14–16  Furthermore, 70% of CDH1 mutation–negative HDGC probands display germline monoallelic CDH1 RNA downregulation (allelic imbalance) reinforcing the role of CDH1 locus in this disease.17 HDGC tumors appear when complete somatic CDH1 inactivation is acquired, leading to reduced or absent E-cadherin expression.18,19 This occurs through second hit mechanisms, pursuing the Knudson's model of tumor suppressor gene inactivation.20,21 CDH1 promoter hypermethylation is the most frequent second hit inactivation mechanism in HDGC primary tumors (50% to 70% of tumors), whereas a second mutation or deletion (loss of heterozygosity [LOH]/intragenic deletions) was less frequently identified.22–24  In the sporadic context, the frequency of CDH1 somatic mutations is controversial, and percentages between 3% and greater than 50% have been reported for GCs with a diffuse component.25–28 LOH at the CDH1 locus was noted in diffuse and intestinal histotypes, with rates ranging from 11% to 39% and 36% to 46%, respectively.29–31 Epigenetic silencing through CDH1 promoter hypermethylation has been reported in frequencies that vary from 50% to 83% in diffuse tumors and from 6.25% to 50% in intestinal tumors.28–32 Importantly, the presence of concomitant CDH1 inactivation mechanisms was rarely described in this setting.28,29,33 Nevertheless, most of these studies have been limited by the series size, the type of molecular mechanisms studied, different methodologic approaches, and the lack of extensive correlations with clinical parameters, thus hampering potential translation of genetic data into medical practice.34  The goal of this study was to analyze the impact of somatic CDH1 alterations in prognosis and survival of patients with GC. In this study, we performed a comprehensive analysis of somatic CDH1 mutations, LOH, and promoter hypermethylation in 246 patients with sporadic and familial GC. Additionally, we analyzed the relationship between somatic alterations underlying CDH1 inactivation and survival rates, clinicopathologic characteristics of patients with GC, and E-cadherin expression in tumors.  PATIENTS AND METHODS  Patients All 246 patients with GC enrolled onto this study were admitted at the Division of General Surgery and Surgical Oncology, University of Siena (Siena, Italy) and were classified as having sporadic (n = 174) or familial (n = 72) GC. Patients with familial GC were classified as having HDGC (n = 19) or FIGC (n = 53) using clinical criteria defined by the International Gastric Cancer Linkage Consortium13 (Appendix Table A1, online only). The presence of CDH1 germline mutations was discarded in all patients with HDGC in this cohort. Patients with GC classification more than pT1N+ were candidates for adjuvant chemotherapy using standard protocols. Prospectively collected clinicopathologic and follow-up data were available. The median follow-up time for surviving patients was 92.1 months. Informed consent was obtained from all patients, and the study was approved by the hospital's ethics committee (Appendix, online only).  DNA Extraction Tissue areas for DNA extraction were histologically verified to contain a minimum of 70% to 80% of neoplastic cells. DNA was isolated from frozen material using the Puregene DNA Purification Kit (Gentra Systems, Minneapolis, MN).  CDH1 Promoter Hypermethylation Analysis CDH1 promoter methylation analysis was carried out in 160 base pairs upstream of the translation start site, encompassing 17 CpG sites, as previously described23 (Appendix, online only). Primer sequences are listed in Appendix Table A2 (online only).  CDH1 Somatic Mutation Screening Fifty nanograms of tumor DNA from 86 diffuse GCs were subjected to mutation screening at the CDH1 hotspot region (exons 7 to 10), as previously described.23 Each mutation was confirmed by an independent polymerase chain reaction amplification followed by sequencing analysis. Primer sequences are listed in Appendix Table A2 (online only).  LOH Analysis LOH analysis was performed using three intragenic CDH1 markers (the promoter −161C/A [rs16260], the exon 13 2076T/C [rs1801552], and the 3′-untranslated region single-nucleotide polymorphisms [rs1801026]) and three proximal and distal microsatellite markers (D16S3025, D16S496, and D16S3067) flanking the CDH1 locus, as described.23 Only informative markers were considered for LOH analysis, and positive samples were repeated twice (Appendix). Primer sequences are listed in Appendix Table A2 (online only).  E-Cadherin Immunohistochemistry Formalin-fixed, paraffin-embedded tissues comprising 207 GCs were stained with mouse E-cadherin monoclonal antibody HECD-1 (dilution 1:200; Zymed, San Francisco, CA) followed by incubation with avidin-biotin complex and diaminobenzidine. E-cadherin immunoreactivity was evaluated on tumor and normal tissues considering the predominant expression pattern—normal (complete membrane staining), aberrant (cytoplasmic and heterogeneous staining), or absent (no staining).  Statistical Analysis Analyses were performed using commercially available statistical software (SPSS, version 14.0; SPSS, Chicago, IL). Statistical associations between clinicopathologic characteristics and CDH1 somatic alterations were assessed using the χ2 test for categorical variables and the t test or analysis of variance for continuous variables. Survival curves were estimated using the Kaplan-Meier method and were compared using the log-rank test. Multivariate analysis was performed using a Cox proportional hazards regression model by considering the following risk factors: sex, age (older v the median age or younger), tumor location (other v antrum), Lauren histotype (nonintestinal v intestinal), depth of tumor invasion (pT2-4 v pT1), lymph node involvement (pN1-3 v pN0), presence of systemic metastasis (M1 v M0), and R category (R1 or R2 v R0). Postoperative mortality was assessed, with deaths unrelated to tumor recurrence considered censored observations at the time of death. P < .05 was considered statistically significant.  RESULTS  After screening of somatic CDH1 promoter hypermethylation, LOH, and mutations in exons 7 to 10, GCs were clustered in the following groups regarding CDH1 alterations: patients with methylation only (named epigenetic); patients with LOH or mutation, with/without methylation (named structural); and patients without CDH1 alterations (named negative).  CDH1 Structural and Epigenetic Alterations in Overall GC Overall, 77 (31.3%) of 246 GCs carried somatic CDH1 alterations. Epigenetic alterations were found in 51 (20.7%) of 246 GCs, and structural alterations were detected in 26 (10.6%) of 246 GCs. Specifically within structural alterations, LOH alone was detected in 18 (7.3%) of 246 GCs, mutation alone was found in three (1.2%) of 246 GCs (c.1109A>G [p.Asp370Gly], c.IVS9+5G>A, and c.1105_1106 insACCAAC), and five (2%) of 246 GCs presented LOH concomitantly with CDH1 promoter hypermethylation (Table 1, Appendix Figs A1 and A2, online only).  Table\t Table 1. Epigenetic and Structural CDH1 Alterations in Patients With Sporadic and Familial GCs  Table 1. Epigenetic and Structural CDH1 Alterations in Patients With Sporadic and Familial GCs  Setting and Histologic Type\tNo. of Patients\tCDH1 Alterations Overall Frequency of CDH1 Alterations* Negative Patients Structural Epigenetic Methylation Only LOH Only Mutation Only Concomitant No.\t%\tNo.\t%\tNo.\t%\tNo.\t%\tNo.\t%\tNo.\t% Sporadic\t\t\t\t\t\t\t\t\t\t\t\t\t     Intestinal\t107\t15\t14.0\t10\t9.3\t0†\t0.0\t3‡\t2.8\t28\t26.2\t79\t73.8     Diffuse/mixed\t67\t17\t25.4\t3\t4.5\t3\t4.5\t0\t0.0\t23\t34.3\t44\t65.7     Total\t174\t32\t18.4\t13\t7.5\t3\t1.7\t3\t1.7\t51\t29.3\t123\t70.7 Familial\t\t\t\t\t\t\t\t\t\t\t\t\t     FIGC§\t53\t9\t17.0\t5\t9.4\t0†\t0.0\t2‡\t3.8\t16\t30.2\t37\t69.8     HDGC§\t19\t10\t52.6\t0\t0.0\t0\t0.0\t0\t0.0\t10\t52.6\t9\t47.4     Total\t72\t19\t26.4\t5\t6.9\t0\t0.0\t2\t2.8\t26\t36.1\t46\t63.9 Total\t246\t51\t20.7\t18\t7.3\t3\t1.2\t5\t2.0\t77\t31.3\t169\t68.7 Abbreviations: FIGC, familial intestinal gastric cancer; GC, gastric cancer; HDGC, hereditary diffuse gastric cancer; LOH, loss of heterozygosity.  *This frequency was calculated assuming that intestinal GCs were negative for somatic CDH1 mutations (three of 174 GCs; 1.7%).  †Intestinal GCs were assumed to lack CDH1 somatic mutations.  !Methylation and LOH occurred concomitantly.  §A statistically significant association was found when comparing FIGCs or HDGCs for three classes of alterations—patients with epigenetic alterations only, patients with structural alterations, and negative patients (P = .005, Fisher's exact test; Appendix Table A3, online only)—because more than 50% of HDGC tumors carry CDH1 promoter methylation. LOH was defined when the LOH index was greater than or less than 1.04 ± 0.13 for D16S3025, 1.0 ± 0.67 for D16S496, and 1.06 ± 0.11 for D16S3067 as reference.  Patients with tumors with CDH1 structural alterations displayed poorer overall survival (P = .017; Fig 1; all cases) than patients with negative tumors or tumors carrying CDH1 epigenetic alterations.  figure Fig 1. Kaplan-Meier curves showing the probability of overall survival for patients with gastric cancer (GC), according to CDH1 alterations (epigenetic, structural, or negative) and stratified according to GC clinical settings (sporadic or familial) and histologic type (intestinal or diffuse).  CDH1 Structural and Epigenetic Alterations in Different Clinical Settings and Histologic Types of GC We evaluated the distribution of somatic CDH1 alterations among clinical settings (sporadic and familial) and histotypes (intestinal and diffuse) of GC and correlated these findings with overall patient survival. The overall frequency of CDH1 alterations was similar in sporadic (51 of 174 tumors; 29.3%) and familial (26 of 72 tumors; 36.1%) tumors (Table 1). Specifically within the sporadic setting, 32 (18.4%) of 174 GCs displayed epigenetic alterations and 19 (10.9%) of 174 GCs presented structural alterations. Within the familial setting, 19 (26.4%) of 72 GCs harbored epigenetic alterations, whereas seven (9.7%) of 72 GCs displayed structural alterations (Table 1; Appendix Table A3, online only). No significant correlation was observed between GC clinical settings and overall/specific CDH1 alterations (P > .05; Appendix Table A4, online only). However, patients with tumors with CDH1 structural alterations, from both sporadic and familial settings, had a poorer survival rate than patients with negative tumors or with CDH1 epigenetic alterations (P = .046 and P = .002, respectively; Fig 1).  Regarding GC histotypes independent of the clinical setting, the overall frequency of CDH1 alterations was 27.5% (44 of 160 GCs) in intestinal GCs and 38.4% (33 of 86 GCs) in diffuse GCs (Table 2). Whereas within intestinal tumors, 15% carried epigenetic alterations and 12.5% carried structural alterations, in diffuse tumors, 31.4% carried epigenetic alterations and 7% carried structural alterations. Although diffuse tumors carried epigenetic alterations more often than structural alterations (P = .007; Table 2), these molecular features did not confer different overall survival (P = .230, Fig 1). In contrast, and although intestinal tumors carried similar frequencies of epigenetic and structural alterations, Kaplan-Meier plots showed that patients with intestinal tumors carrying structural alterations had a lower probability of survival than patients with intestinal tumors negative for or carrying CDH1 epigenetic alterations (P = .041; Fig 1). In fact from the 26 GCs with structural alterations, 20 (76.9%) were of the intestinal type (P = .007; Table 3).  Table\t Table 2. Epigenetic and Structural CDH1 Alterations in Intestinal and Diffuse GCs  Table 2. Epigenetic and Structural CDH1 Alterations in Intestinal and Diffuse GCs  GC Histotype\tNo. of Patients\tCDH1 Alterations Overall Frequency of CDH1 Alterations Negative P Epigenetic Structural Negative P No.\t%\tNo.\t%\tNo.\t%\tNo.\t%\tNo.\t% Intestinal\t160\t24\t15.0\t20\t12.5\t116\t72.5\t.007\t44\t27.5\t116\t72.5\t.108 Diffuse\t86\t27\t31.4\t6\t7.0\t53\t61.6\t\t33\t38.4\t53\t61.6\t Abbreviation: GC, gastric cancer.  Table\t Table 3. Comparison of Clinicopathologic Variables With CDH1 Alterations in Patients With GC  Table 3. Comparison of Clinicopathologic Variables With CDH1 Alterations in Patients With GC  Variable\tTotal No. of Patients (N = 246)\tCDH1 Overall Alterations CDH1 Epigenetic and Structural Alterations Positive (n = 77) Negative (n = 169) P\tEpigenetic (n = 51) Structural (n = 26) Negative (n = 169) P No. of Patients\t%\tNo. of Patients\t%\tNo. of Patients\t%\tNo. of Patients\t%\tNo. of Patients\t% Age, years\t\t\t\t\t\t\t\t\t\t\t\t\t     Mean\t68.05\t69.1\t\t67.6\t\t\t69.7\t\t67.9\t\t67.6\t\t     Standard deviation\t10.9\t11.8\t\t10.4\t\t\t13.3\t\t8.5\t\t10.4\t\t Sex\t\t\t\t\t\t.110\t\t\t\t\t\t\t.004     Male\t143\t51\t66.2\t92\t54.4\t\t28\t54.9\t23\t88.5\t92\t54.4\t     Female\t103\t26\t33.7\t77\t45.6\t\t23\t45.1\t3\t11.5\t77\t45.6\t Liver or peritoneal metastases\t\t\t\t\t\t.920\t\t\t\t\t\t\t.218     Present\t20\t6\t7.8\t14\t8.3\t\t2\t3.9\t4\t15.4\t14\t8.3\t     Absent\t226\t71\t92.2\t155\t91.7\t\t49\t96.1\t22\t84.6\t155\t91.7\t Radicality of resection*\t\t\t\t\t\t.502\t\t\t\t\t\t\t.049     R0\t181\t54\t70.1\t127\t75.1\t\t40\t78.4\t14\t53.8\t127\t75.1\t     R1/2\t65\t23\t29.9\t42\t24.8\t\t11\t21.6\t12\t46.1\t42\t24.8\t Extent of gastrectomy\t\t\t\t\t\t.680\t\t\t\t\t\t\t.100     Partial\t150\t45\t58.4\t105\t62.1\t\t34\t66.7\t11\t42.3\t105\t62.1\t     Total\t96\t32\t41.6\t64\t37.9\t\t17\t33.3\t15\t57.7\t64\t37.9\t Lymphadenectomy\t\t\t\t\t\t1.0\t\t\t\t\t\t\t.737     D1\t91\t28\t36.4\t63\t37.3\t\t17\t33.3\t11\t42.3\t63\t37.3\t     D2/3\t155\t49\t63.6\t106\t62.7\t\t34\t66.7\t15\t57.7\t106\t62.7\t Lauren histotype\t\t\t\t\t\t.108\t\t\t\t\t\t\t.007     Intestinal\t160\t44\t57.1\t116\t68.6\t\t24\t47.1\t20\t76.9\t116\t68.6\t     Diffuse\t86\t33\t42.9\t53\t31.4\t\t27\t52.9\t6\t23.1\t53\t31.4\t Depth of invasion\t\t\t\t\t\t.436\t\t\t\t\t\t\t.695     pT1\t17\t3\t3.9\t14\t8.3\t\t2\t3.9\t1\t3.8\t14\t8.3\t     pT2\t93\t31\t40.3\t62\t36.7\t\t22\t43.1\t9\t34.6\t62\t36.7\t     pT3-4\t136\t43\t55.8\t93\t55.0\t\t27\t52.9\t16\t61.5\t93\t55.0\t Lymph node involvement\t\t\t\t\t\t.021\t\t\t\t\t\t\t.042     Absent (pN0)\t72\t15\t19.5\t57\t33.7\t\t8\t15.7\t7\t26.9\t57\t33.7\t     Present (pN+)\t173\t62\t80.5\t111\t65.7\t\t43\t84.3\t19\t73.1\t111\t65.7\t Stage\t\t\t\t\t\t.029\t\t\t\t\t\t\t.155     I\t57\t10\t13\t47\t27.8\t\t6\t11.8\t4\t15.4\t47\t27.8\t     II\t41\t15\t19.5\t26\t15.4\t\t10\t19.6\t5\t19.2\t26\t15.4\t     III\t106\t41\t53.2\t65\t38.5\t\t28\t54.9\t13\t50.0\t65\t38.5\t     IV\t40\t10\t13\t30\t17.8\t\t6\t11.8\t4\t15.4\t30\t17.8\t GC familial aggregation\t\t\t\t\t\t.371\t\t\t\t\t\t\t.372     Positive\t72\t26\t33.8\t46\t27.2\t\t19\t37.3\t7\t26.9\t46\t27.2\t     Negative\t174\t51\t66.2\t123\t72.8\t\t32\t62.7\t19\t73.1\t123\t72.8\t E-cadherin IHC†\t\t\t\t\t\t.275\t\t\t\t\t\t\t.479     Absent\t30\t12\t19.0\t18\t12.5\t\t8\t20\t4\t17.4\t18\t12.5\t     Aberrant\t152\t46\t73.0\t106\t73.6\t\t30\t75\t16\t69.6\t106\t73.6\t     Normal\t25\t5\t7.9\t20\t13.9\t\t2\t5\t3\t13.0\t20\t13.9\t Abbreviations: GC, gastric cancer; IHC, immunohistochemistry.  *R0 resection indicates a microscopically margin-negative resection.35 R1/R2 resections indicate different grades of positivity for tumor after surgery (primary tumor, regional nodes, and microscopic/macroscopic margin involvement).35  †Only 207 patients were analyzed for IHC; the percentages of CDH1-positive, -negative, epigenetic, and structural alterations were calculated by adjusting the total number of alterations to 63, 144, 40, and 23, respectively.  We observed that within sporadic and familial settings, patients with tumors carrying structural alterations had worse overall survival. Moreover, we also verified that patients with intestinal-type tumors carrying structural alterations, independent of the clinical setting, had worse overall survival. Therefore, we next assessed whether the worse overall survival of patients with intestinal tumors with structural alterations was a feature of familial and/or sporadic settings.  Intestinal tumors belonging to sporadic and familial (FIGC) settings did not differ in the frequency of epigenetic (14% sporadic, 17% FIGC) and structural alterations (12.1% sporadic, 13.2% FIGC; Table 1, Appendix Table A3, Fig 2). Nonetheless, the Kaplan-Meier plots showed statistical differences in terms of overall survival only for patients belonging to FIGC families. The patients with FIGC carrying tumors with structural alterations had the poorest survival rate (P < .001; Fig 2). Although it did not reach statistical significance (P = .148), patients with intestinal tumors belonging to the sporadic setting and carrying structural alterations also had a poorer survival rate compared with patients carrying negative/epigenetic alterations (Table 1, Appendix Table A3, Fig 2).  figure Fig 2. Details of the molecular analysis performed in gastric cancer (GC) samples and survival rates of patients with GC. Kaplan-Meier curves show the probability of overall survival for patients with familial intestinal GC (FIGC), hereditary diffuse GC (HDGC), sporadic intestinal GC, and sporadic diffuse GC according to CDH1 alterations (epigenetic, structural, or negative).  Within diffuse-type tumors, tumors belonging to the sporadic setting displayed epigenetic (25.4%) and structural (8.9%) alterations, whereas HDGC tumors (52.6%) harbored exclusively epigenetic alterations (Table 1, Appendix Table A3). Kaplan-Meier plots showed no association between the type of CDH1-specific alterations and overall survival of patients with sporadic diffuse or HDGC tumors (P = .259 and P = .631, respectively; Fig 2). However, despite the lack of statistical significance, the few patients with sporadic diffuse-type tumors carrying structural alterations had the lowest survival rates (P = .259; Fig 2).  CDH1 Structural and Epigenetic Alterations and Clinicopathologic Features of GC We next tried to understand whether the presence of overall or specific CDH1 alterations correlated with clinicopathologic parameters of tumors and patients (Table 3). This analysis revealed that patients with tumors carrying overall CDH1 alterations displayed significantly more frequently lymph node metastases (P = .021) and more advanced tumors (P = .029), in particular invasive stage III. The other clinicopathologic parameters, including age, sex, liver or peritoneal metastases, radicality of resection, extent of gastrectomy, lymphadenectomy, depth of invasion, familial aggregation, and even Lauren histotype, were not significantly associated with the presence of overall CDH1 alterations (Table 3).  When considering the specific type of CDH1 alterations in the different GC groups (epigenetic v structural v negative), we observed that patients carrying tumors displaying epigenetic CDH1 alterations more frequently had the diffuse histotype (P = .007) and 84.3% had lymph node metastases (P = .02). We also observed that patients carrying tumors with structural CDH1 alterations more frequently had intestinal histotype (20 of 26 patients, 76.9%; P = .007) and were mainly males (23 of 26 patients, 88.5%; P = .004), and 46.1% had been submitted to R1/2 resections (compared with 21.6% with epigenetic alterations and 24.8% negative for alterations; P = .09; Table 3).  CDH1 Structural and Epigenetic Alterations and E-Cadherin Immunoexpression Pattern We also evaluated E-cadherin expression and correlated it with tumor histology and CDH1 alterations. Of 207 patients analyzed, 73.4% showed aberrant expression, 14.5% displayed complete loss of E-cadherin, and 12.1% retained the protein at the cell membrane (Table 4). In both sporadic and familial settings, there was a significant association between complete loss of E-cadherin expression and diffuse histology (sporadic: P < .001; familial: P = .017; Table 4).  Table\t Table 4. E-Cadherin Immunoexpression in Sporadic and Familial GCs  Table 4. E-Cadherin Immunoexpression in Sporadic and Familial GCs  GC Setting and Histologic Type\tNo. of Patients\tE-Cadherin Immunoexpression P Absent Aberrant Normal No.\t%\tNo.\t%\tNo.\t% Sporadic\t147\t\t\t\t\t\t\t<.001     Intestinal\t95\t4\t4.2\t74\t77.9\t17\t17.9\t     Diffuse/mixed\t52\t14\t26.9\t36\t69.2\t2\t3.8\t Familial\t60\t\t\t\t\t\t\t.017     FIGC\t43\t5\t11.6\t32\t74.4\t6\t14\t     HDGC\t17\t7\t41.2\t10\t58.8\t0\t0\t Total\t207\t30\t14.5\t152\t73.4\t25\t12.2\t NOTE. Scoring criteria were as follows: absent, E-cadherin not expressed; aberrant, cytoplasmic or heterogeneous staining of E-cadherin; and normal, complete E-cadherin membrane staining.  Abbreviations: FIGC, familial intestinal gastric cancer; GC, gastric cancer; HDGC, hereditary diffuse gastric cancer.  In addition, there were no differences when comparing E-cadherin immunohistochemistry (IHC) pattern in GCs carrying overall or specific CDH1 alterations with the pattern in those without alterations (P = .479 and P = .275, respectively; Table 3). These results revealed that the CDH1 alterations analyzed do not generate a specific E-cadherin IHC pattern and, more importantly, that 60% of GCs without E-cadherin expression and approximately 70% of GCs with aberrant expression are negative for CDH1 alterations (Appendix Table A5, online only).  DISCUSSION  GC is a highly heterogeneous disease where even similar clinical and pathologic features lead to distinct outcomes.7,9,11,36 These observations indicated that staging systems, based on clinical and pathologic findings, may have reached their limit of usefulness and impelled the need for molecular biomarkers, as an added value, to predict patients' outcome and treatment.  The present study encompasses a large series of sporadic and familial GCs (negative for CDH1 germline mutations) systematically analyzed for somatic CDH1 structural and epigenetic inactivating mechanisms. Furthermore, this study systematically correlates the molecular data with patient survival, clinicopathologic parameters, and E-cadherin IHC expression.  The main findings of the present study indicate the following: somatic CDH1 alterations occur in approximately 30% of all GCs analyzed (approximately 20% epigenetic, approximately 10% structural); CDH1 structural alterations underlie the worst survival rate of patients with GC overall; patients with FIGC with tumors with structural alterations had the worst overall survival rate; HDGC tumors harbored exclusively CDH1 promoter hypermethylation (approximately 50%); specific CDH1 alterations underlie distinct GC clinicopathologic features; and overall or specific CDH1 alterations are not associated with a distinct pattern of E-cadherin immunoexpression. These and other findings will be discussed in more detail here.  The results herein reported may change the paradigm thus far described for CDH1 alterations in GC. In the sporadic context, E-cadherin abnormalities have been, to date, mainly associated with the diffuse type of GC, and CDH1 somatic mutations and promoter methylation have each been reported in more than 50% of these patients.26,27,29,33,34,37 In diffuse GCs, we report epigenetic alterations (methylation only) in 25.4%, somatic mutations in 4.5%, and LOH in 4.5% (structural alterations). These frequencies are lower than most earlier reported frequencies, but are similar to those recently reported in several independent series.28,30,31 Discrepancies are most likely a result of different methodologies used. Interestingly, we demonstrate that CDH1 somatic epigenetic and structural alterations are as frequent in intestinal as in diffuse GC, suggesting histotype independence.  In the familial setting, tumors from HDGC families lacking germline CDH1 mutations displayed exclusively CDH1 promoter methylation in frequencies comparable to those previously published for HDGC carriers of CDH1 germline mutations.23 We herein report for the first time, to the best of our knowledge, that tumors from FIGC families also present somatic CDH1 epigenetic and structural alterations in frequencies similar to those of patients with sporadic GC.  Importantly, GCs with epigenetic or structural CDH1 alterations or negative for CDH1 alterations revealed an unequal impact on patient survival. Patients with tumors with CDH1 structural alterations displayed a significantly poorer survival rate than patients with tumors negative for CDHI alterations or patients with tumors with epigenetic CDH1 alterations. In line with these findings, Gamboa-Dominguez et al38 observed that patients with GC displaying CDH1 exon 8/9 deletions (structural) have a worse clinical evolution and a shorter overall survival. An adverse prognostic effect of CDH1 promoter hypermethylation in patients with diffuse GC has also been reported, but no structural alterations were reported in that series.34  When we explored in depth the survival rates of patients belonging to different clinical settings and histologic types, the most striking finding was that patients with FIGC with tumors with structural alterations had the worst overall survival. This observation indicates that CDH1 somatic alterations are a novel and unexpected feature of FIGCs and, more importantly, allows the stratification of patients with FIGC into subsets with completely different clinical outcomes. Available data on familial history associated with survival of patients with GC has remained largely inconclusive.39–41 One of the latest reports addressing this issue demonstrated a favorable prognosis for patients with a positive family history of GC, whereas others did not.39–41 These controversial results may have been affected by different strategies in selecting the study population and, mainly, by inconsistent definitions and clinical criteria for family history, precluding comparison of trustworthy results. In light of our own findings, molecular variables such as CDH1 alterations may be crucial to better define the survival of patients with family history.  Apart from the impact of CDH1-specific alterations on survival of patients with GC, these alterations also underlie other clinicopathologic features. Tumors with CDH1 epigenetic alterations were more often of the diffuse histotype, and patients more frequently displayed lymph node metastases. These observations suggest that superextended lymphadenectomy may be an adequate procedure in these patients.42 However, patients with tumors with CDH1 structural alterations more often had the intestinal type (76.9%), and despite the generally accepted better prognosis,43 these patients had the worst survival, as previously discussed. Furthermore, these patients were mainly men and had been more frequently subjected to R1/2 radical resections, indicating the presence of a more aggressive disease. Therefore, screening of CDH1 alterations at the biopsy stage, concomitantly with histologic classification and both faster and more radical surgery, may bring benefit to the approximately 10% of patients with GC who belong to this specific group (Appendix Fig A3, online only).  In parallel, we also verified that approximately 68% of patients with altered E-cadherin expression were negative for CDH1 alterations, showing that IHC, per se, is not an efficient method to infer E-cadherin molecular alterations and indicating other transcriptional/post-transcriptional regulatory mechanisms.44–47 Following this idea, we have recently reported that miR-101 downregulation with consequent EZH2 upregulation constitutes an additional mechanism by which E-cadherin becomes dysfunctional, mainly in intestinal-type GC retaining allele(s) untargeted by classical CDH1 inactivating mechanisms.28  In conclusion, our data provide, for the first time to our knowledge, a detailed analysis of somatic CDH1 alterations in different clinical settings and histologic types of GC, highlighting subsets of patients with distinct clinical outcomes. In particular, this work defined a group of patients (FIGC with CDH1 structural alterations) with the worst prognosis among all GCs analyzed. The presence of CDH1 epigenetic and structural alterations in a diagnostic/preoperative biopsy may provide clinical utility and improve patient management, particularly to infer the prognosis of GC and the pattern of tumor dissemination. Germline CDH1 point or small frameshift mutations can be identified in 30–50% of hereditary diffuse gastric cancer (HDGC) families. We hypothesized that CDH1 genomic rearrangements would be found in HDGC and identified 160 families with either two gastric cancers in first-degree relatives and with at least one diffuse gastric cancer (DGC) diagnosed before age 50, or three or more DGC in close relatives diagnosed at any age. Sixty-seven carried germline CDH1 point or small frameshift mutations. We screened germline DNA from the 93 mutation negative probands for large genomic rearrangements by Multiplex Ligation-Dependent Probe Amplification. Potential deletions were validated by RT–PCR and breakpoints cloned using a combination of oligo-CGH-arrays and long-range-PCR. In-silico analysis of the CDH1 locus was used to determine a potential mechanism for these rearrangements. Six of 93 (6.5%) previously described mutation negative HDGC probands, from low GC incidence populations (UK and North America), carried genomic deletions (UK and North America). Two families carried an identical deletion spanning 193 593 bp, encompassing the full CDH3 sequence and CDH1 exons 1 and 2. Other deletions affecting exons 1, 2, 15 and/or 16 were identified. The statistically significant over-representation of Alus around breakpoints indicates it as a likely mechanism for these deletions. When all mutations and deletions are considered, the overall frequency of CDH1 alterations in HDGC is ∼46% (73/160). CDH1 large deletions occur in 4% of HDGC families by mechanisms involving mainly non-allelic homologous recombination in Alu repeat sequences. As the finding of pathogenic CDH1 mutations is useful for management of HDGC families, screening for deletions should be offered to at-risk families.  Go to: INTRODUCTION Hereditary diffuse gastric cancer (HDGC) is an autosomal dominant disorder that accounts for <1% of all cases of gastric cancer (GC). Although uncommon, this disease constitutes an important health issue due to its severity, its high penetrance, early age at presentation and the unavailability of effective screening tools. Diffuse gastric cancer (DGC) is the most important cause of cancer lethality in mutation positive HDGC families with lobular breast cancer being a secondary medical concern (1).  Heterozygous germline point or small frameshift mutations in E-cadherin gene (CDH1) [OMIM +192090], a calcium-dependent cell-to-cell adhesion molecule and a tumor suppressor protein, are the only germline molecular defect associated with HDGC (2–4). Overall, carriers of CDH1 germline mutations have a cumulative GC risk, before age 75, of 40–67% for men and 63–83% for women and a risk for lobular breast cancer of 39–52% (5,6). Clinical expression of the disease often occurs before the third decade of life (1), but pre-clinical multifocal tumors, which may have the potential for metastatic spread, are already present in the stomachs of young asymptomatic CDH1 mutation carriers (7,8).  Clinical criteria have been established, in 1999, by the International Gastric Cancer Linkage Consortium (IGCLC) to select families for mutation screening in the CDH1 gene, based mainly on early age of disease presentation and diffuse histology of the tumors (9). These have been later adapted in an attempt to enlarge the population to which this screening could apply, in countries with low incidence of GC (10). Overall, 30–40% of HDGC families harbor CDH1 germline mutations, but this frequency is highly variable between countries with different incidences of GC, with low incidence countries like UK, North America and Canada, displaying nearly 50% of HDGC mutation positive families, while high incidence countries like Portugal and Italy, displaying around 10% of HDGC mutation positive families (4,6,11).  Approximately two-thirds of families, screened worldwide, remain genetically unexplained and the concern related with the management of CDH1 mutation negative gastric cancer families persists. Since the required samples for full genome linkage, in any GC family, are difficult to obtain, a candidate gene approach to identify novel susceptibility genes has been more frequently used. Putative tumor suppressor genes, which are commonly inactivated in sporadic GC and/or associated with GC development, could also represent good candidate susceptibility genes to familial GC. Previous studies including ours, however, have lead us to rule out RUNX3, Caspase-10, SMAD4, HPP1, and Desmoglein 2 as major GC predisposition genes in families with aggregation of gastric carcinoma (12–14). Virtually all diffuse gastric carcinomas, both hereditary and sporadic, independently of whether they carry a CDH1 mutation, display identical morphological features as well as aberrant patterns of expression of E-cadherin (unpublished data). As no other gene has been so far shown to play a major genetic role in the disease, we hypothesize that other genetic events at the CDH1 locus, potentially missed by standard screening techniques, such as large genomic rearrangements, deletions or insertions could be found in families with multiple cases of DGC. Examples of this type of molecular alterations have been recently reported for other cancer associated syndromes. Large genomic rearrangements have been described in hMLH1 and hMSH2 in 10–20% of HNPCC mutation negative families, in APC in 15% of FAP mutation negative families and in lower frequencies in BRCA1 and BRCA2 in breast cancer families initially identified as negative, as well as in CDKN2A in melanoma negative families (15–21).  Based on these observations, we aimed at characterizing CDH1 genomic rearrangements in a large population of HDGC patients, negative for CDH1 point mutations. Moreover, by using a larger series of families screened for CDH1 point mutations and rearrangements, in five reference centers world-wide and arising in countries with different GC incidence rates, we aimed at clarifying the frequency of CDH1 germline alterations and its relationship with GC incidence as well as with clinical criteria for selection of patients at risk.  Go to: RESULTS Multiplex ligation-dependent probe amplification analysis for the CDH1 gene  We collected, from five reference centers, 93 DNA samples from families with aggregation of DGC and negative for CDH1 germline point mutations. These families were selected based on their risk to carry a germline alteration in CDH1 and the clinical criteria described in Materials and Methods section.  We found that six out of the 93 (6.5%) probands showed abnormal multiplex ligation-dependent probe amplification (MLPA) features when compared with the controls, displaying an ∼70% signal reduction in one or more exons of the CDH1 gene and, suggesting the presence of large deletions affecting the CDH1 locus (Fig. 1, Table 1). No abnormalities were observed in the remaining 87 probands.  Figure 1. Figure 1. MLPA output from the six HDGC families carrying germline deletions of the CDH1 locus. The first 17 bars represent signal obtained with MLPA probes within the CDH1 gene. Please note that for exon 1, three probes were used (1a, 1b and 1c). CDH1 deletions ... Table 1. Table 1. Table summarizing HDGC families analyzed for large CDH1 rearrangements (gray columns) and compilation of frequency of CDH1 alterations indicating geographic origin grouped by GC incidence rates After reproducing these results in an independent MLPA reaction, we verified that four probands (families 1 to 4) presented deletions at the 5′-end of the gene: in three probands deletions encompassed at least exon 1, intron 1 and exon 2 and, in one proband a smaller deletion that encompassed one of the three probes designed for CDH1 exon 1 was found. The remaining two probands (families 5 and 6) displayed deletions at the further 3’end of the gene: in one proband a deletion of the genomic sequence encompassing exons 14 and 16 was observed, an in the other a deletion of exon 16 only was observed (Fig. 1).  Fine mapping of the deletions breakpoints  A combination of array CGH and PCR was utilized to map the putative breakpoints in five of the six probands (Fig. 2). The array CGH indicated the regions of chromosome 16 at which the breakpoints were located. PCR primers were then designed to span the breakpoints and the resulting PCR products sequenced (Table 2). Family 1 and family 2 showed identical MLPA patterns as well as virtually identical array CGH results. Sequencing of the breakpoint revealed that these two families harbored an identical 193 593 bp deletion encompassing the full sequence of CDH3 and extending to position IVS2+57 595 in CDH1. Family 3 was found to carry a 5671 bp deletion with breakpoints at position −3809 upstream of the CDH1 transcription start site (TSS) and at position IVS2+742 in CDH1. Family 4 harbored a 150 bp deletion encompassing the TSS of CDH1, with breakpoints located 125 bp upstream and 25 bp downstream of the TSS. Array CGH was not done for this proband since the PCR encompassing CDH1 exon 1 produced both a normal sized and a shorter band that was visible by dHPLC and further used to clone the breakpoints for the deletion by direct sequencing. Family 5 carried an 8078 bp deletion ranging from IVS13-2738 to 20 bp downstream of the stop codon of CDH1, as confirmed by long-range PCR using specific primers flanking the breakpoints (Fig. 3A). Family 6 was found to carry an 828 bp deletion ranging from IVS15+3097 to 223 bp downstream of the stop codon of CDH1 (Fig. 3B).  Figure 2. Figure 2. Array CGH profiles obtained for HDGC families carrying large CDH1 deletions. Examples are shown for four of the six families. See Tables Tables22 and and33 for details on the deletion found in each family. Deletions are represented by ... Figure 3. Figure 3. Sequencing chromatogram of the mapped breakpoints in family 5 (A) and family 6 (B). Comparison with the human genome reference sequence. (A) Deletion of 8078 bp without insertion of P or N nucleotides. (B) Deletion of 828 bp and insertion of a triplet ... Table 2. Table 2. Summary of HDGC families displaying CDH1 germline deletions Bioinformatics analysis of breakpoints and deletion containing sequences  Assuming that deletions are more likely to occur due to recombination events, we analyzed the genomic sequences flanking the boundaries of these five deletions, which are predicted to constitute target regions for breaks and recombinations to occur. We performed a search for Alu elements which have been reported to mediate non-allelic recombination events in several other malignancies. Evidence for non-homologous end joining (NHEJ) and non-allelic homologous recombination (NAHR) has been described in atypical NF1 microdeletions. These deletions were traced back to short sequences with similarities to Alu elements near the deletion junction (22). Moreover, deletions in the PMP22 gene have been described to be a product of homologous recombination between AluS elements present in intronic regions of the gene (23). We also performed a search for the presence of cryptic sequences recognizable by the RAG1/2 recombinase near the breakpoints and the insertion of P and/or N nucleotides in the rejoining sites. Such sequence features, reminiscent of V(D)J recombination sites, are expected to indicate whether cleavage by the RAG1/2 recombinase may occur near these joining sites (24,25).  In three families carrying two different deletions we found that deletions overlap Alu repeats (Table 3). In families 1 and 2, bearing the same deletion at chr16:67193822–67387415, both deletion boundaries fall within annotated Alu repeats, AluSp near the start and AluSg near the end. The sequences of the two repeats are highly similar (percentage identity >75%) (Table 3, Fig. 4A). In family 3, carrying a deletion at chr16:67324886–67330557, the start of the deletion maps inside an AluSx repeat while the end map into a repeat AluSg and, the sequences of these two repeats display an identity of ∼80% (Table 3, Fig. 4B).  Figure 4. Figure 4. Analysis of sequences flanking the deletion breakpoints. (A and B) Homology analysis of AluS type elements flanking deletions in families 1, 2 and 3. (C) Ensembl based scheme displaying the repetitive elements flanking the deletion in family 5. Table 3. Table 3. Deletion coordinates and distance from deletion ends to the closest Alu containing sequences For the deletion at chr16:67328695–67328844, present in family 4, an AluJo repeat is present 304 bp upstream from the start of the deletion, and another AluJo is present 1497 bp downstream the end of the deletion. The closest repeat external to the deletion is a MIR named MIRb, 1235 bp downstream of the end-point but no sequence identity can be seen in the −150/+50 and −50/+150 bp fragments centered, respectively, on the start and the end of the deletion. An interesting feature of this 150 bp deletion is the presence of microsinteny/microhomology around the breakpoints due to the presence of five nucleotides sequence (CTCTC) between the upstream and downstream breakpoints.  Family 5 displays a deletion at chr16:67416845–67424923 that does not overlap Alu repeats, although it occurs in the proximity of this type of repetitive regions (Table 3). The closest repeats both upstream and downstream are AT rich sequences, ∼300 bp distant from the deletion (Fig. 4C). Moreover, 344 bp upstream of the deletion start a FLAM_C repeat exists and 442 bp downstream of the endpoint there is an AluJb sequence (Table 3). The alignment of the −150/+150 bp surrounding the deletion ends did not show any identity.  The deletion in family 6, at chr16:67424298–67425126, is unlikely to have been produced by homology-dependent mechanisms, since a small (3 bp) sequence insertion at the junction between the deletion breakpoints was identified (Fig. 3B). For this deletion, the closest Alu repeats found on both sides of the deletion are an AluSq 50 bp upstream and an AluJb 239 bp downstream of the deletion’s start and end, which share a sequence identity >77% (Table 3).  In a range of +/−5 Mb flanking the five deletions reported in this study, we did not find evidence for the presence of RAG1/2 consensus sequences accompanied or not by the insertion of P and/or N nucleotides in the rejoining sites. Nevertheless, the presence of Alus overlapping 6 out of the 12 points analyzed (the ends of the six deletion sequences) is statistically significant (z-score 3.3, P-value 0.0005) as tested by performing a randomization analysis with 1000 random sampling of 12 random locations (see Materials and Methods).  Clinical features of deletion-carrying HDGC families  The proband in family 1 was a 61-year-old female of European Irish/German/Scottish/Danish ancestry presenting with lobular breast cancer. Her daughter and son, as well as a grand-daughter developed DGC at the age of 40, 37 and 28, respectively, and are already diseased. The deletion was only looked for in the proband that is currently alive.  In family 2, the proband, of unknown origin but currently living in Canada, was a female who developed DGC at 38 and later at 43 years old and the only patient tested in this family. Her two sisters also developed DGC at the ages of 30 and 35, respectively, but were not tested for the presence of the same deletion.  The presence of the same deletion in two different families could be explained two ways. Either of these families had a common ancestor or the deletions arose independently. To clarify this issue, we performed haplotype analysis with seven microsatellite markers surrounding the CDH1 gene and verified that an allele for each of the microsatellite markers was common between the two families, demonstrating the existence of a common ancestor carrying this deletion (Fig. 5).  Figure 5. Figure 5. Haplotype analysis of family 1 and family 2 probands. (A) Scheme of chromosome 16 with reference to microsatellite markers used in the haplotype analysis and to the breakpoints of the deletion. (B) Common haplotypes displayed by probands of family 1 and ... The proband in family 3, was a 34-year-old female of European Lithuanian origin, with DGC. Her mother was also affected by the same disease at uncertain age, and her grand-father developed pancreatic cancer at the age of 43 and none of them was tested for the deletion. The proband in family 4, was a 40-year-old male of Hispanic origin who developed DGC at the age of 40. His 24-year-old brother died from exactly the same disease and his father died at 42 with a pancreatic cancer, neither has been tested for the deletion. Carrier tests have been performed in ten other family members and one of which was positive.  In family 5, the proband was a patient of European ancestry, who developed DGC at 33 years. His two sisters developed DGC at 33 and 35 years. At least one second degree relative developed GC, but no more information was available. So far, only the proband was tested, but five other family members are currently being tested for the presence of the deletion. As for family 6, the proband was a 51-year-old patient of European ancestry who developed DGC. His mother died of GC of unconfirmed histology at age of 52. Two sisters were recently diagnosed with DGC by surveillance endoscopy, although only one of them was tested for the deletion and proved to be a carrier. This sister’s offspring, constituted of two asymptomatic individuals, also tested positive for the deletion. Four other second degree relatives died of cancer, one with DGC, two of GC of unknown histology and one of lobular breast cancer.  Clinical criteria, frequency of CDH1 alterations and GC incidence  In the present study, we analyzed 93 families based on two clinical criteria: (i) three or more DGC in first degree relatives diagnosed at any age or; (ii) two or more GC in first degree relatives with at least one DGC diagnosed before age 50; and found that 6.5% of these families carry large CDH1 germline deletions.  With the purpose of clarifying the relative contribution of germline CDH1 point mutations and deletions in a large series of families fulfilling the aforementioned criteria, we collected data not only on the 93 negative families, herein analyzed for genomic rearrangements, but also on 67 HDGC families carrying CDH1 point mutations, from five different reference centers, and originated from countries with low, moderate and high GC incidence rates.  From the series of 160 probands, 73 (45.6%) were proved to carry germline CDH1 alterations: 41.9% point mutations and 3.8% large deletions (Fig. 6).  Figure 6. Figure 6. Frequency of CDH1 alterations in a series of 160 HDGC families, subdivided by GC incidence rates as well as by type of gene alteration (point or small frameshift mutations and large deletions). In low incidence countries namely in the North America, Canada, UK and Holland, the frequency of germline CDH1 alterations was 51.6%, while in moderate (Germany) and high (Portugal and Italy) incidence countries the frequency of alterations was 25 and 22.2%, respectively (Fig. 6). Large deletions were only identified in countries with low incidence of GC and account for 9% of all CDH1 alterations in HDGC families from such countries (Table 1).  Go to: DISCUSSION In this report, we demonstrate that germline large deletions affecting the CDH1 locus are one of the mechanisms underlying HDGC. Our data supports the addition of MLPA based screening for such deletions to clinical screening for CDH1 mutations.  The CDH1 locus, like the hMLH1, hMSH2, BRCA1, APC and CDKN2A loci, is a region susceptible to genomic rearrangements leading to increased cancer susceptibility (18,19,21,26).  The analysis of deletions found in our HDGC patients, revealed that two out of five deletions showed extensive sequence identity between the two breakpoints, strongly suggesting that NAHR was the mechanism of their formation (Families 1, 2 and 3). The deletion breakpoints of these two deletions fall within Alu repeats similar to what has been previously reported for hMLH1, hMSH2, BRCA1 and infrequently for APC (18,19,26). Alu repetitive sequences have long been believed to give rise to genomic deletions promoting recombinational instability (27). PMP22 deletions have been described to be a product of homologous recombination between intronic AluS elements in hereditary neuropathies (23), but the type of Alu sequences involved in deleterious deletions underlying cancer-associated syndromes have not been systematically explored in the literature. Nevertheless, at least one of the Alu sequences located at large deletion breakpoints in hMLH1 gene are described to be of the AluSx type (28). In our study, we analyzed the type of Alu sequences associated with CDH1 large deletions in HDGC, and found that six breakpoints overlap AluS elements: three overlap AluSg, two overlap AluSp and one overlaps an AluSx (Table 3).  One of the deletions herein identified occurs in a context of microhomology and could have arisen by microhomology-mediated recombination, a mechanism of NHEJ. This mechanism is associated with very short stretches of sequence identity (a few bp) between the two ends of the breakpoint junctions (29,30), rather than with long stretches of sequence identity at these locations (31,32). Moreover, it involves the double strand breakage of DNA followed by end joining in the absence of extensive sequence homology. A similar microhomology-mediated intra-strand repair mechanism has been previously described to occur either in pathological conditions such as trisomy in a family displaying an inverted duplication/deletion of 2p25.1–25.3 (33) or in recombination events underlying genomic variation during evolution (34). The same putative mechanism underlying the type of deletion and end re-joining, found in this family, has been described in a recent report where the treatment of mice, with specific chemicals or radiation during late spermatogenic stages, generates very short deletions, duplications, or inversions with microhomology around breakpoints. The ligation by NHEJ mediated by microhomology, is raised in the aforementioned report, as the major path of repair (35), and provides insight on how this specific CDH1 deletion may have initially arisen.  Another of our deletions is unlikely to have been produced by homology-dependent mechanisms, since a small sequence insertion (3 bp) at the junction between the deletion breakpoints was identified. Since no extensive homology exists in the vicinity of breakpoints, it is likely that this deletion have been mediated by NHEJ. Identical deletions involving small insertions (3, 4, 6 bp), of unknown origin at the site of the breakpoints, have been previously identified within copy number variants (CNVs) breakpoint-region sequences, in normal population (32,34).  In contrast to all other CDH1 deletions described so far in this report, one does not seem to be caused by NAHR, microhomology-mediated recombination or small sequence insertions at the junction site. This deletion is flanked by upstream and downstream low complexity AT rich sequences, as well as FLAM_C repeat and an AluJb sequence. Similar features surrounding deletions have been previously described to occur in CNVs described in apparently normal individuals (34), but the mechanisms involved in their generation is not known. The AT rich sequences could mediate a microhomology-mediated deletion mechanism based on their base composition bias, though further evidence would be needed to prove this.  Deletions found in HDGC families mainly fall into three separate classes: products of NAHR, in which recombination between Alu containing sequences leads to deletion of the sequence between them; products of microhomology-mediated recombination, a mechanism of NHEJ; and results of small insertions of unknown origin at the site of the breakpoint junction. These deleterious deletions in CDH1 appear as the pathogenic counterpart of apparently normal evolution generated CNVs (32,34).  The probands studied (n = 160) fulfilled clinically used criteria for HDGC (see Materials and Methods). We found that 45.6% (73/160) of them displayed CDH1 alterations (Fig. 6). In comparison, only 7/123 (5.7%) of the probands of families who did not meet the above testing criteria, had point mutations or small frameshift mutations, and none had deletions detectable through MLPA (data not shown). Based on this analysis, we have demonstrated that the clinical criteria herein used (three or more DGC in first degree relatives diagnosed at any age or two or more GC in first degree relatives with at least one DGC diagnosed before age 50) will identify most families with HDGC associated germline CDH1 alterations, with a mutation pick up rate of ∼46%. Based on these criteria, ∼4% of HDGC patients display large CDH1 germline deletions (Fig. 6).  Two of the families shared the same large deletion that included the whole of the CDH3 gene and the first two exons of CDH1. Individuals with homozygous germline truncating mutations in the CDH3 gene present with hypotrichosis with juvenile macular dystrophy (HJMD) syndrome (36,37). Heterozygous carriers of such mutations are phenotypically normal and have not been reported to have increased GC risk. Haplotype analysis supports a shared ancestor as the source of the (chr16:67193822–67387415) in these two families. We have previously used this approach to demonstrate that recurrent HDGC associated CDH1 germline point mutations can arise from a common ancestor (6).  In conclusion, 6.5% of probands from strictly defined HDGC families who previously tested negative for CDH1 mutations carried heterozygous germline deletions. As these can be readily identified using MLPA, an assay frequently used in clinical cancer genetics laboratories, we suggest that MLPA be added to the testing strategy for all families who meet clinical criteria for HDGC.   Development of malignant tumours is in part characterized by the ability of a tumour cell to overcome cell–cell adhesion and to invade surrounding tissue. E-cadherin is the main adhesion molecule of epithelia1, 2, 3 and it has been implicated in carcinogenesis because it is frequently lost in human epithelial cancers4, 5, 6. Re-establishing the functional cadherin complex in tumour cell lines results in a reversion from an invasive to a benign epithelial phenotype7. However, it remained unresolved whether the loss of E-cadherin-mediated cell adhesion was a cause or a consequence of tumour progression in vivo. Here we report that the loss of E-cadherin expression coincides with the transition from well differentiated adenoma to invasive carcinoma in a transgenic mouse model of pancreatic beta-cell carcinogenesis (Rip1Tag2)8. Intercrossing Rip1Tag2 mice with transgenic mice that maintain E-cadherin expression in beta-tumour cells results in arrest of tumour development at the adenoma stage, whereas expression of a dominant-negative form of E-cadherin induces early invasion and metastasis. The results demonstrate that loss of E-cadherin-mediated cell adhesion is one rate-limiting step in the progression from adenoma to carcinoma.  We have studied tumour progression in a transgenic mouse line that expresses the SV40 T antigen under the control of the insulin promoter in the beta-cells of pancreatic islets of Langerhans (Rip1Tag2)8. These mice reproducibly develop tumours in a multistage tumorigenesis pathway involving islet hyperplasia (approx70% of all islets)9,10, the formation of new blood vessels (angiogenesis, approx20%)11,12 and, finally, solid beta-cell tumours (insulinomas) that can be further distinguished into well differentiated, benigntumours (adenomas, approx2%) and de-differentiated, invasive tumours (carcinomas, approx0.5%). In beta-cell adenomas, tumour cells are arranged as epithelial structures in cords and ribbons. The tumour's encapsulation, the lack of local invasion, as well as the relatively normal nuclear morphology and nuclear to cytoplasmic ratio define these tumours as benign (Fig. 1A, a and b). In contrast, beta-cell carcinomas are recognized by their local invasion, by remnants of successive fibrous capsules that by expansive growth are embedded in the mass, by cellular anaplasia, nuclear atypia, and high nuclear to cytoplasmic ratio (Fig. 1Ac). Metastases are usually not found in these mice, probably because they succumb to hypoglycaemia with increased tumour mass.  Figure 1: Histopathological and immunohistochemical analyses of tumorigenesis.Figure 1 : Histopathological and immunohistochemical analyses of tumorigenesis. Unfortunately we are unable to provide accessible alternative text for this. If you require assistance to access this image, or to obtain a text description, please contact npg@nature.com A, Transition from beta-cell adenoma to beta-cell carcinoma in Rip1Tag2 transgenic mice. a–c, Histopathological analysis of beta-cell tumour development by haematoxylin/eosin staining (H&E). d–f, Immunofluorescence analysis of E-cadherin expression in the different stages of beta-cell tumorignesis. B, E-cadherin-negative carcinoma cells are insulin-producing beta-cells. a–c, Immunohistochemical analysis of E-cadherin expression in the multiple stages of beta-cell tumorigenesis. d–f, Immunohistochemical analysis of insulin expression on adjacent sections to a–c. A, Adenoma; C, carcinoma; E, exocrine pancreas; I, normal islet of Langerhans. Scale bars, 25 mum.  High resolution image and legend (99K)  Immunohistochemical analysis of the different stages of Rip1Tag2 tumorigenesis reveals that E-cadherin is expressed at the surface of beta-cells in normal islets of Langerhans and in the acini cells of the exocrine pancreas (Fig. 1Ad). E-cadherin expression is unaltered in adenoma cells (Fig. 1Ae). In contrast, E-cadherin is completely absent in all carcinomas examined (Fig.1Af; >200 tumours were analysed). Notably, in a subset of tumours E-cadherin expression is focally lost concomitant with the loss of benign hallmarks (data not shown), indicating that the expression of E-cadherin is downregulated with the transition from beta-cell adenoma to beta-cell carcinoma. This loss of E-cadherin expression is due to transcriptional repression (ref. 13; data not shown).  The tumour cells that have lost E-cadherin expression at the carcinoma stage still express insulin, indicating that they are indeed beta-tumour cells (Fig. 1B). However, insulin expression is absent in a subset of highly anaplastic carcinomas (approx30% of the carcinomas) suggesting a process of de-differentiation. Expression of two more members of the cadherin family, N-cadherin (Fig. 2) and R-cadherin (data not shown) is not downregulated with the transition from adenoma to carcinoma. Furthermore, the epithelial polarity marker Na/K-ATPase (Fig. 2) and the tight junction protein ZO1 (data not shown) are only modestly downregulated or disorganized in the invasive tumours. No change in the organization of F-actin and fibronectin is apparent, and vimentin, a marker for mesenchymal cells, is not expressed in the carcinoma stages (data not shown), indicating that the transition from adenoma to carcinoma in Rip1Tag2 transgenic mice does not represent a complete epithelial–mesenchymal conversion. Of the proteins analysed, loss of E-cadherin expression is the only notable change in this process.  Figure 2: Expression of epithelial markers during the transition from beta-cell adenoma to carcinoma in Rip1Tag2 transgenic mice.Figure 2 : Expression of epithelial markers during the transition from |[beta]|-cell adenoma to carcinoma in Rip1Tag2 transgenic mice. Unfortunately we are unable to provide accessible alternative text for this. If you require assistance to access this image, or to obtain a text description, please contact npg@nature.com Immunofluorescent staining of adjacent histological sections from adenoma (a, c, e) or carcinoma (b, d, f) with antibodies against E-cadherin (a, b), N-cadherin (c, d) or Na/K-ATPase (e, f). A, Adenoma; C, carcinoma; E, exocrine pancreas. Scale bar, 25 mum.  High resolution image and legend (67K)  To aess whether the loss of E-cadherin-mediated cell adhesion is required for tumour progression in vivo we sought to maintain expression of E-cadherin throughout beta-cell tumorigenesis in Rip1Tag2 transgenic mice. Five independent transgenic mouse lines were generated that specifically expressed E-cadherin under the control of the rat insulin promoter in pancreatic beta-cells (Rip1E-cad). None of them exhibited abnormalities in islet development and islet physiology (data not shown). All five Rip1E-cad mouse lines were intercrossed with Rip1Tag2 transgenic mice, and changes in tumour development in single-transgenic (Rip1Tag2) and double-transgenic (Rip1Tag2 times Rip1E-cad) mice within the same litters were monitored. In single-transgenic mice, beta-cell tumours developed without any significant aberrations from the well studied pathway (see above). In double-transgenic mice, tumour volumes were moderately reduced, whereas the number of tumours per mouse did not significantly change (Table 1). Histopathological analysis revealed that the incidence of carcinoma formation was reduced from 26.5% of all tumours in single-transgenic mice to 7.8% in double-transgenic mice (Table 1; P < 0.001, chi-square test). Notably, the few carcinomas in the double-transgenic mice had all lost E-cadherin protein expression, including expression from the transgene (Fig. 3), indicating that beta-tumour cells have developed a strong selection against E-cadherin-mediated cell adhesion. The downregulation of E-cadherin transgene expression might be due to destabilization of E-cadherin message or protein, because neither the transgene was lost nor insulin promoter activity was downregulated in these tumour cells (data not shown). These results indicate that loss of E-cadherin expression is a rate-limiting step in the progression to the carcinoma stage in vivo.  Figure 3: Forced expression of E-cadherin blocks the transition from adenoma to carcinoma.Figure 3 : Forced expression of E-cadherin blocks the transition from adenoma to carcinoma. Unfortunately we are unable to provide accessible alternative text for this. If you require assistance to access this image, or to obtain a text description, please contact npg@nature.com Immunohistochemical analysis of E-cadherin expression in adenomas (a, c) and carcinomas (b, d) of single-transgenic Rip1Tag2 (a, b) and double-transgenic Rip1Tag2 times RipE-cad mice (c, d). The few carcinomas found in Rip1Tag2 times Rip1E-cad double-transgenic mice (d) have focally lost E-cadherin expression, including expression from the E-cadherin transgene (beta-tumour cells that still express E-cadherin are indicated by arrows). A, Adenoma; C, carcinoma; E, exocrine pancreas. Magnification: times200.  High resolution image and legend (46K)  Table 1: Impact of E-cadherin function on beta-cell tumorigenesis in vivoTable 1 - Impact of E-cadherin function on |[beta]|-cell tumorigenesis in vivo Full table   Next we investigated whether loss of E-cadherin function is sufficient to promote tumour invasion in vivo. Rip1Tag2 mice were intercrossed with transgenic mice that carry a dominant-negative form of E-cadherin, lacking the extracellular domain, under the control of the insulin promoter (Rip1dnE-cad)14. Expression of the truncated form of E-cadherin in normal mice displaces both E- and N-cadherin from pancreatic beta-cells resulting in a perturbation of beta-cell aggregation during islet development14. However, in adult mice normal islets form, most likely because N-cadherin, in contrast to E-cadherin, is no longer displaced from beta-cells after birth (U.D. and H.S., unpublished results). Tumour incidence or tumour volume were not significantly changed between double-transgenic Rip1Tag2 times Rip1dnE-cad mice and single-transgenic Rip1Tag2 littermates. However, the incidence of carcinoma formation was dramatically increased from 24.5% of all tumours in single-transgenic mice to 50.6% in double-transgenic mice (Table 1; P < 0.001, chi-square test). In addition, one fourth of the double-transgenic mice developed metastases to the pancreatic lymph nodes (3.5% of solid tumours), an invasive phenotype that was never observed in single-transgenic Rip1Tag2 mice8, 9, 10, 11, 12 (Table 1). Histopathological analysis reveals two types of invasive tumours in the Rip1Tag2 times Rip1dnE-cad double-transgenic mice (Fig. 4): tumours that clearly overgrow their capsule and invade into the surrounding tissue, yet exhibit a relatively normal cellular morphology (referred to as early carcinoma), and invasive tumours that exhibit all hallmarks of highly malignant tumour cells, including nuclear atypia and cellular anaplasia (referred to as late carcinoma). Some of these invasive tumours contain tumour cells of both normal and malignant cellular phenotype indicating a gradual early–late transition during tumour progression (Fig. 4e). The lymph node metastases of double-transgenic mice were all of the late carcinoma phenotype (Fig. 4). Insulin expression indicates that the carcinomas and the lymph node metastases have originated from beta-cell tumours (Fig. 4). In contrast to the carcinomas of single-transgenic Rip1Tag2 mice, endogenous E-cadherin is still expressed in the invasive carcinoma stages of double-transgenic Rip1Tag2 times Rip1dnE-cad mice (Fig. 4). The dominant-negative form of E-cadherin appears to induce subcellular redistribution and destabilization of endogenous E-cadherin in late carcinoma and in metastasis (Fig. 4). Taken together, abrogation of cadherin-mediated cell adhesion is sufficient to induce early tumour invasion and metastasis.  Figure 4: Forced expression of dominant-negative E-cadherin induces tumour-cell invasion and metastasis.Figure 4 : Forced expression of dominant-negative E-cadherin induces tumour-cell invasion and metastasis. Unfortunately we are unable to provide accessible alternative text for this. If you require assistance to access this image, or to obtain a text description, please contact npg@nature.com a–c, H&E staining of histological sections containing early carcinoma, late carcinoma, and lymph node metastasis from Rip1Tag times Rip1dnE-cad mice as indicated. Arrowheads mark lymphoid tissue that has been overgrown by metastatic beta-tumour cells. d–f, Higher magnification microphotographs of H&E staining of different tumour stages in double-transgenic Rip1Tag2 times Rip1dnE-cad mice as indicated. g–i, Immunohistochemical analysis of insulin expression on sections adjacent to d–f. j–l, Immunohistochemical analysis of endogenous E-cadherin expression in the different tumour stages using antibodies specific for the extracellular domain of E-cadherin. C, Carcinoma; E, exocrine pancreas; M, metastasis. Magnifications: a–c, times100; d–l, times200.  High resolution image and legend (93K)  The present findings provide the first direct evidence for a causal role of E-cadherin in the progression from benign adenoma to malignant carcinoma in vivo. The data suggest that E-cadherin counteracts the tumour cell's transition to an invasive phenotype by maintaining intercellular adhesion and epithelial organization. E-cadherin-mediated cell adhesion has no direct effect on the control of tumour cell proliferation; the incidence of both mitosis and apoptosis does not differ significantly between the different tumour phenotypes (mitosis: 22 plusminus 5.6% for adenoma versus 18.3 plusminus 5.3% for carcinoma; apoptosis: 0.43 plusminus 0.18% for adenoma versus 0.32 plusminus 01.2 for carcinoma). These results are surprising because the mere loss of cell adhesion is not expected to enable cells to cross the basal lamina and to invade surrounding tissue. It is likely that the loss of E-cadherin-mediated cell adhesion may convey additional signals to induce tumour-cell invasion. Recently, it has been demonstrated that beta-catenin, in addition to its role in cadherin-mediated cell adhesion, is part of the Wnt signalling pathway15, 16, 17. Cytosolic beta-catenin is translocated to the nucleus, where it binds Tcf/Lef-1 transcription factors and associates with a number of promoter sequences and potentially affects transcription18, 19, 20. Activation of this pathway by mutation of either beta-catenin or APC has been shown to play an important role in the development of colon carcinoma and melanoma21, 22, 23. The observation that beta-catenin can transduce signals to the nucleus raises the possibility that changes in E-cadherin-mediated cell adhesion may modulate gene expression and thus tumour phenotype. Furthermore, E-cadherin's transcriptional repression in human cancers and in Rip1Tag2 transgenic mice needs to be investigated. Chromatin rearrangement, hypermethylation, and loss of transcription factor binding coincide with supression of E-cadherin promoter activity in invasive carcinoma cells24, 25, 26; however, the molecular signals responsible remain unknown. Thus, unravelling the regulation of E-cadherin expression and the communication pathway between cell adhesion and gene expression may lead to a better understanding of how tumour cells control their invasive phenotype.  Top of page Methods Transgenic mice. The generation and phenotypic characterization of Rip1Tag28, 9, 10, 11, 12 and Rip1dnE-cad14 transgenic mice has been described previously. Rip1E-cad transgenic mice were generated according to standard procedures27. The transgene was generated by inserting a 3.0 kilobase (kb) XbaI/HindIII complementary DNA fragment encoding mouse E-cadherin (a gift from M. Takeichi, Kyoto) into the plasmid pRip1Tag8 from which the fragment encoding the SV40 early region was removed by digestion with XbaI and HindIII. Five independent founder lines were found to express the transgene in pancreatic beta-cells and were subsequently backcrossed to C57/BL6 for at least 2 generations.  Tumour volumes. Glucose (5%, w/v) was added to the drinking water of all tumour-bearing mice to counteract hypoglycaemia resulting from insulinoma development. Mice were killed between 10 and 16 weeks of age, tumour numbers and diameters were determined, and the tumour volume was calculated assuming a spherical tumour.  DNA synthesis and apoptosis assays. For BrdU staining, tissues were prepared as above, except that mice were injected intraperitoneally with 100 mug of bromodeoxyuridine (BrdU) per gram of body weight 2 hours before being killed. Histological sections were immunostained as previously described28. The mitotic index equals the percentage of BrdU-positive nuclei. Detection of apoptotic cells using the TUNEL-technique was performed as previously described28. The apoptotic index is the percentage of TUNEL-positive nuclei.  Histopathological analysis. Tissue embedding in JB4 plastic and preparation of semi-thin sections was as described10. For immunohistochemical analysis pancreas were removed and fixed in HBS-Ca2+ (HEPES-buffered saline, 1 mM CaCl2) containing 4% paraformaldehyde for 2 h at 4 °C14. Tissues were incubated overnight at 4 °C in HBS-Ca2+/30% sucrose, embedded in O.C.T. compound (Tissue Tek), and frozen in liquid nitrogen. Immunostaining was as described14. Specific binding of primary antibodies was visualized using either the ABC Vector-horseradish peroxidase kit according to the manufacturer's recommendations or fluorescence-labelled secondary antibodies as indicated. Antibodies were monoclonal rat anti-mouse E-cadherin (Zymed; 1:150), monoclonal rat anti-mouse N-cadherin14 (a gift from M. Takeichi, Kyoto), polyclonal guinea pig anti-insulin (DAKO, 1:150), and rabbit polyclonal serum against mouse Na/K-ATPase (a gift from W. J. Nelson, Stanford, 1:500). Biotin-conjugated donkey anti-rat and anti-rabbit IgG (1:500) and Cy3-streptavidin (1:750) were purchased from Jackson ImmunoResearch Laboratories.  Hereditary diffuse gastric cancer (HDGC), characterized by susceptibility to gastric signet ring cell carcinomas (SRCCs) and caused by _CDH1_ germline mutations, is rare in the Japanese. We present here a Japanese family with HDGC identified by comparative genomic hybridization (CGH) analysis. A 55 -year-old woman was treated with completion gastrectomy for multiple SRCCs, and pathological examination revealed approximately 200 foci of SRCC with loss of E-cadherin expression. Her 30-year-old son had surveillance endoscopy and was found to have multiple SRCCs. He underwent total gastrectomy, and 32 foci of SRCC with loss of E-cadherin expression were histologically found. Although no point mutations were detected in _CDH1_ by sequencing, CGH revealed a 275-kb deletion involving exons 716 of _CDH1_ in both patients. While only a few HDGCs have been reported in East Asia, patients with multiple SRCC may need to be offered appropriate genetic counseling and testing in this area.  ### Keywords  Hereditary diffuse gastric cancerGastric cancerE-cadherinComparative genomic hybridization  ### Electronic supplementary material  The online version of this article (doi:[10.1007/s10120-013-0298-y](http://dx.doi.org/10.1007/s10120-013-0298-y)) contains supplementary material, which is available to authorized users.  ## Introduction  Hereditary diffuse gastric cancer (HDGC) is an autosomal dominant disease associated with multiple signet ring cell carcinomas (SRCCs) and is caused by a germline mutation in the E-cadherin gene (_CDH1_). Guilford et al. [1] first reported that HDGC is caused by truncating mutations of _CDH1_ in New Zealand Maori families with early-onset, multiple SRCCs. In Western countries, inherited gastric cancers (GCs) are thought to account for 13% of all GCs [2]. Clinically, approximately 25% of families fulfilling the criteria for the diagnosis of HDGC have inactivating _CDH1_ germline mutations [2, 3]. In contrast, in East Asian countries, including Japan, Korea, and China, with high incidences of GCs [4], HDGC has rarely been reported [5, 6]. Recently, Yamada et al. [7] reported two germline alterations in the _CDH1_ gene in two Japanese familial GCs. However, HDGC is still rarely diagnosed in East Asian countries, and it is still unknown whether HDGC is really rare or overlooked because of the high incidence of coincidental familial GC. We report here clinical characteristics and genomic analysis of a Japanese HDGC family with a _CDH1_ germline mutation.  ## Case reports  A 55-year-old female patient was referred to our hospital for treatment of multiple SRCCs detected by endoscopic examination during an annual health check. She had a past history of intramucosal SRCC at the age of 34 and had undergone distal gastrectomy with _Billroth I_ reconstruction in another hospital. She had no past history of other malignancies, including lobular breast cancer. Family history of GC was noted, affecting the patients father and paternal grandfather (Fig.1). Esophagogastroduodenoscopy (EGD) in our hospital detected 1012 small pale mucosal patches, mainly in the greater curvature of the remnant stomach (Fig.2a), but no atrophic gastritis indicative of _Helicobacter pylori_ infection. The size of each focus was endoscopically estimated as less than 10mm. In one of the largest lesions, narrow band imaging with magnification showed a wavy-shaped irregular microvessel pattern suggesting undifferentiated adenocarcinoma (Fig.2b). All the biopsy specimens from six lesions demonstrated SRCC. Completion gastrectomy was performed, and the entire gastric mucosa was histologically examined.  [Open image in new window![](https://static- content.springer.com/image/art%3A10.1007%2Fs10120-013-0298-y/MediaObjects/10120_2013_298_Fig1_HTML.gif)](https ://static- content.springer.com/image/art%3A10.1007%2Fs10120-013-0298-y/MediaObjects/10120_2013_298_Fig1_HTML.gif)  Fig.1  Pedigree chart. The presence of the deletion mutation (Fig.4) was confirmed in patients 1 and 2 in two generations  [Open image in new window![](https://static- content.springer.com/image/art%3A10.1007%2Fs10120-013-0298-y/MediaObjects/10120_2013_298_Fig2_HTML.gif)](https ://static- content.springer.com/image/art%3A10.1007%2Fs10120-013-0298-y/MediaObjects/10120_2013_298_Fig2_HTML.gif)  Fig.2  Clinicopathological findings in patient 1. **a** Conventional endoscopy revealed pale lesions in the greater curvature of the remnant stomach (_arrow_). **b** Magnifying endoscopy with narrow-band imaging showed irregular microvessels. **c** Histology of the tumor showing SRCC proliferating in the upper layer of the fundic gland mucosa. **d** Periodic acid-Schiff (PAS) staining highlighted intracytoplasmic mucin in SRCC. **e** Immunohistochemistry for E-cadherin. Although normal gastric epithelial cells showed clear membrane staining, SRCCs showed loss of immunoreactivity. **f** Gastrectomy mapping study. Approximately 200 SRCC foci were observed in the resected specimen, predominantly near the greater curvature  Histopathological examination revealed approximately 200 SRCC foci, and their maximum size was 10mm in diameter (Fig.2f). None of the lesions showed submucosal invasion, and tumor cells were mostly confined to the upper mucosal layer (Fig.2c). Periodic acid-Schiff (PAS) staining highlighted intracytoplasmic mucin in SRCC (Fig.2d). Immunohistochemistry was performed using anti-E-cadherin antibody (NCH-38; 1:100 dilution; DAKO, Glostrup, Denmark) as previously described [8]. The results showed loss of E-cadherin expression in SRCCs. (Fig.2e). These histopathological findings were exactly identical to the findings in the HDGC cases reported by Guilford et al. [1]. No lymph node metastasis was observed.  Based on the pathological findings of the proband (patient 1), we suspected that she might have HDGC. Therefore, we performed surveillance endoscopy of two sons of the proband. Neither son had a history of malignancy. EGD for the 30-year-old elder son (Patient 2) detected three tiny pale areas at the body- antrum junction of the stomach (Fig.3a, b). Apart from these tiny pale patches, no other endoscopic findings were suggestive of SRCC, but biopsy specimens obtained from each of the three pale areas revealed SRCC. Based on these findings, we clinically diagnosed HDGC in both patients, and total gastrectomy was performed on patient 2. Histopathological assessment of the entire gastric mucosa revealed a total of 32 SRCCs (Fig.3f) along with loss of E-cadherin expression in SRCCs (Fig.3c, d, e). No apparent findings of recurrence or distant metastasis have been noted on the follow-up thus far. EGD in the younger son identified no significant endoscopic findings.  [Open image in new window![](https://static- content.springer.com/image/art%3A10.1007%2Fs10120-013-0298-y/MediaObjects/10120_2013_298_Fig3_HTML.gif)](https ://static- content.springer.com/image/art%3A10.1007%2Fs10120-013-0298-y/MediaObjects/10120_2013_298_Fig3_HTML.gif)  Fig.3  Clinicopathological findings in patient 2. **a** Conventional endoscopy revealed pale lesions at the body-antrum junction of the stomach (_arrows_). **b** Magnifying endoscopy with narrow-band imaging. **c** Histological analysis showed SRCC in the fundic gland mucosa. **d** PAS staining highlighted intracytoplasmic mucin in SRCC. **e** Immunohistochemistry revealed lack of E-cadherin expression in SRCC. **f** Gastrectomy mapping study. A total of 32 signet ring cell carcinoma foci were detected exclusively in the fundic gland area. The finding was similar to that in patient 1  ## The presence of a 275-kb deletion involving _CDH1_  In order to perform genetic analysis, genomic DNA was extracted from peripheral leukocytes of patient 1, patient 2, and a healthy volunteer. Genomic DNA was also extracted from biopsy specimens obtained from the cancer site(s) in the stomachs of both patients. All 16 exons of _CDH1_ were sequenced by PCR-direct sequencing from both directions using previously reported primers [9]. Promoter methylation was analyzed by bisulfite modification and quantitative real-time methylation specific PCR (MSP) as previously described [10]. Primer sequences for the methylated DNA were 5-TCG TTT TGG GGA GGG GTT C-3 (forward) and 5-CAA ATA AAC CCC GAA AAC ACC G-3 (reverse), and those for the unmethylated DNA were 5-GGA GGT ATT GTT TTT TGT ATT-3 (forward) and 5-AAC AAA CCA TCA ACT CCA-3 (reverse). However, no _CDH1_ mutation or aberrant methylation was detected in peripheral lymphocytes or biopsy specimens of patient 1.  Array-comparative genomic hybridization (CGH) analysis was performed according to the manufacturers protocol using genomic DNA of peripheral leukocytes and human reference DNA (Caucasian, male #5190-4370, female #5190-4371, Agilent Technologies, Santa Clara, CA). DNA was digested with _Alu_I and _Rsa_I, labeled with Cy5 and Cy3, respectively, using a SureTag DNA Complete Labeling Kit (Agilent Technologies), and hybridized to a SurePrint G3 Cancer CGH+SNP 4180K microarray (Agilent Technologies). The microarray was scanned with an Agilent G2565BA microarray scanner (Agilent Technologies), and the scanned data were processed using the Agilent CytoGenomics Software (Agilent Technologies). Regions that were covered by five probes or more and showed a mean log2 ratio &gt;0.5 or &lt;0.5 were considered to have a gain or loss of the copy numbers, respectively. The CGH analysis revealed a 275-kb deletion in both patients 1 and 2 that involved exons 716 of _CDH1_ (Fig.4a, b). There were no other genomic regions that had altered copy numbers commonly in patients 1 and 2 but not in a Japanese healthy control (Supplementary Table1). The deletion of exons 716 of _CDH1_ was further confirmed by copy number analysis of _CDH1_ exon 4, 6, 7, 8, 10, 13, and 16 using TaqMan copy number assays (Life Technologies, Grand island, NY) (Fig.4c).  [Open image in new window![](https://static- content.springer.com/image/art%3A10.1007%2Fs10120-013-0298-y/MediaObjects/10120_2013_298_Fig4_HTML.gif)](https ://static- content.springer.com/image/art%3A10.1007%2Fs10120-013-0298-y/MediaObjects/10120_2013_298_Fig4_HTML.gif)  Fig.4  A 275-kb deletion involving exons 716 of _CDH1_. **a** Array CGH profiles obtained from peripheral leukocyte DNA of patient 1, patient 2, and a healthy control (33-year-old Japanese male without history of gastric cancer or family history of gastric cancer). The profiles of patients 1 and 2 showed a 275-kb deletion. Deleted regions and the _CDH1_ locus are shown by a _green square_ and a _red bar_, respectively. A deletion of an exactly identical region was observed in both patients 1 and 2. **b** The location of the breakpoint within the _CDH1_ gene. The 5-most probe of the deleted region (probe y, A_16_P03169109) was located on exon 7, and the 3-most probe of the intact region (probe x, A_16_P03169108) was located on exon 6. **c** Copy numbers of _CDH1_ exons 4, 6, 7, 8, 10, 13, and 16 in patient 1, Pptient 2, and a healthy control analyzed by TaqMan copy number assays. Deletion of _CDH1_ exons 716 was confirmed in patients 1 and 2. The TaqMan probes for exons 4, 6, 7, 8, 10, 13, and 16 were Hs00296979_cn, Hs01313772_cn, Hs01453316_cn, Hs00466712_cn, Hs01438786_cn, Hs01952025_cn, and Hs02507752_cn, respectively. For the seven exons above, copy numbers of patients 1 and 2 relative to a healthy control were calculated by Ct method  ## Discussion  We presented a Japanese HDGC family with a 275-kb deletion involving exons 716 of _CDH1_. From the initial EGD, we strongly suspected patient 1 to suffer from HDGC according to the clinical course and histopathological findings, which, although they did not fulfill the first diagnostic criteria for HDGC formulated by the International Gastric Cancer Linkage Consortium (IGCLC) in 1999 [11], fulfilled the criteria updated in 2009 [2], individuals with diffuse gastric cancer before the age of 40 with or without family history. We therefore recommended the probands offspring to undergo EGD, and one son had three tiny pale areas at the body-antrum junction of the stomach that were confirmed to be SRCCs. Other than the tiny pale patches on endoscopy, there were no other endoscopic findings suggestive of SRCC. In the IGCLC guidelines, it is recommended that any endoscopically visible lesions should be targeted and that random sampling of six biopsies should additionally be taken for each of the following anatomical zones: antrum, transitional zone, body, fundus, and cardia. In total, a minimum of 30 biopsies is recommended. However, we were able to perform successful targeted biopsies of a minimal number of biopsy sites in patient 1 (_n_=6) and 2 (_n_=3) because good preparation was obtained as previously described [12]. Based on the presence of multiple SRCCs, patient 2 was also diagnosed with HDGC and was successfully treated with total gastrectomy.  The existence of numerous SRCCs strongly indicated that this family has HDGC. Although the expression of E-cadherin was absent or reduced in the tumor cells, we were unable to detect any small mutations in any exons of _CDH1_ or to detect aberrant methylation of _CDH1_ in DNA collected from the peripheral leukocytes of the two patients. Generally, 2530% of HDGC families are known to have germline mutations of the _CDH1_ gene [2]; 77.9% of all reported _CDH1_ germline mutations are truncating mutations, and their locations span the whole length of the _CDH1_ gene without any hot spots [13]. At the same time, 4% of reported _CDH1_ germline mutations exhibit large deletions that are not detectable by conventional DNA sequencing [14]. Therefore, we continued to search for a large deletion of the _CDH1_ gene in patients 1 and 2, and, by CGH analysis, we detected a pathogenic 275-kb deletion in both patients 1 and 2. To our knowledge, this is the first case of HDGC harboring a large germline deletion in the range of hundreds of kilobases involving _CDH1_.  GCs have marked geographic variation in their incidence, with high incidences in East Asian countries and low incidences in Western countries. On the other hand, the majority of HDGC with _CDH1_ mutations have been reported in the Western countries. To our knowledge, there have been only two Japanese HDCG families with germline _CDH1_ mutations reported so far [7]. However, it still remains unclear whether HDCG truly occurs less in Japan than in Western countries. The rarity of HDGC in Japan might be due to insufficient surveillance rather than reflecting the actual incidence. The high incidence of sporadic GCs is likely to increase false-positive familial GCs and make it difficult to identify hereditary GCs. The IGCLC determined diagnostic criteria for HDGC, but these criteria are likely to be too broad to be used in areas with high GC incidence, including Japan [2, 11]. In addition, the prevalence of large deletions, as suggested by the study of Yamada et al. [7] and here, might have made detection of the _CDH1_ mutation difficult.  Although HDGC has been regarded as rare in East Asia, these two recent reports show that there are HDGC families in East Asia and suggest the importance of the analysis of large deletions for cases with multiple SRCCs. Genetic counseling and testing may be advised for patients with multiple SRCCs and their family members even in East Asia.\n",
      "2918\n",
      "The Nrf2 (nuclear factor erythroid 2 [NF-E2]-related factor 2 [Nrf2])–Keap1 (Kelch-like erythroid cell-derived protein with CNC homology [ECH]-associated protein 1) signaling pathway is one of the most important cell defense and survival pathways. Nrf2 can protect cells and tissues from a variety of toxicants and carcinogens by increasing the expression of a number of cytoprotective genes. As a result, several Nrf2 activators are currently being tested as chemopreventive compounds in clinical trials. Just as Nrf2 protects normal cells, studies have shown that Nrf2 may also protect cancer cells from chemotherapeutic agents and facilitate cancer progression. Nrf2 is aberrantly accumulated in many types of cancer, and its expression is associated with a poor prognosis in patients. In addition, Nrf2 expression is induced during the course of drug resistance. Collectively, these studies suggest that Nrf2 contributes to both intrinsic and acquired chemoresistance. This discovery has opened up a broad spectrum of research geared toward a better understanding of the role of Nrf2 in cancer. This review provides an overview of (1) the Nrf2–Keap1 signaling pathway, (2) the dual role of Nrf2 in cancer, (3) the molecular basis of Nrf2 activation in cancer cells, and (4) the challenges in the development of Nrf2-based drugs for chemoprevention and chemotherapy.  Keywords: Nrf2, Keap1, chemoprevention, chemoresistance, ARE, oxidative stress The transcription factor nuclear factor erythroid 2 (NF-E2)-related factor 2 (Nrf2) belongs to the Cap ‘N’ Collar (CNC) family that contains a conserved basic leucine zipper (bZIP) structure. The main function of Nrf2 is to activate the cellular antioxidant response by inducing the transcription of a wide array of genes that are able to combat the harmful effects of extrinsic and intrinsic insults, such as xenobiotics and oxidative stress. As a result, Nrf2 has traditionally been regarded as the cell's main defense mechanism and a major regulator of cell survival. Activation of the Nrf2 defense response has been shown to protect against neurodegenerative diseases, aging, diabetes, photo-oxidative stress, cardiovascular disease, inflammation, pulmonary fibrosis, acute pulmonary injury, and cancer (for review, see Motohashi and Yamamoto 2004; Jeong et al. 2006; Zhang 2006; Kensler et al. 2007; Lau et al. 2008).  Recent studies, however, have demonstrated that Nrf2 promotes the survival of not only normal cells but also cancer cells. Accumulation of Nrf2 in cancer cells creates an environment conducive for cell growth and protects against oxidative stress, chemotherapeutic agents, and radiotherapy. This phenomenon has been termed the “dark side of Nrf2” (Lau et al. 2008; Wang et al. 2008b). This discovery has opened up a broad spectrum of research geared toward a better understanding of the role of Nrf2 signaling in cancer and has set a new paradigm for the development of pharmacological reagents targeting Nrf2 for cancer prevention and treatment.  Go to: The Nrf2–Keap1 (Kelch-like erythroid cell-derived protein with CNC homology [ECH]-associated protein 1)–ARE (antioxidant response element) signaling pathway Nrf2  Nrf2 contains seven functional domains, known as Neh1–Neh7 (Fig. 1A). Of these, the Neh2 domain, located in the N terminus of Nrf2, is the major regulatory domain. Neh2 contains seven lysine residues that are responsible for ubiquitin conjugation (Zhang et al. 2004) as well as two binding sites (termed ETGE and DLG motifs) that help regulate Nrf2 stability (McMahon et al. 2006; Tong et al. 2006b). The ETGE and DLG motifs interact with Keap1, which is a substrate adaptor protein for the Cullin 3 (Cul3)-dependent E3 ubiquitin ligase complex that represses Nrf2 by promoting its ubiquitination and subsequent proteasomal degradation (Cullinan et al. 2004; Kobayashi et al. 2004; Zhang et al. 2004; Furukawa and Xiong 2005).  Figure 1. Figure 1. Conserved domains of Nrf2 and Keap1. (A) Nrf2 contains seven domains, known as Neh1–Neh7. The Neh2 domain contains two binding motifs, DLG and ETGE, which interact with Keap1. The Neh4, Neh5, and Neh3 domains are important for the transactivation ... The Neh1 and Neh6 domains have also been reported to regulate the stability of Nrf2. Neh1 contains a CNC-type bZIP DNA-binding motif that allows Nrf2 to bind DNA and dimerize with other transcription factors (Moi et al. 1994). Additionally, the Neh1 domain has been shown to interact with UbcM2, a ubiquitin-conjugating enzyme, to regulate the stability of Nrf2 (Plafker et al. 2010). The Neh6 domain contains two binding sites (DSGIS and DSAPGS motifs) for the β-transducin repeat-containing protein (β-TrCP). β-TrCP acts as a substrate adaptor for the Skp1–Cul1–Rbx1/Roc1 ubiquitin ligase complex. Phosphorylation of the DSGIS motif by GSK-3 increases the ability of β-TrCP to ubiquitinate Nrf2 and promote its rapid turnover (McMahon et al. 2004; Rada et al. 2011; Chowdhry et al. 2013).  The Neh3, Neh4, and Neh5 domains interact with coactivators to enable the transactivation of Nrf2 target genes. The Neh3 domain binds to the chromo-ATPase/helicase DNA-binding protein family member CHD6, which functions as an Nrf2 transcriptional coactivator (Nioi et al. 2005). The Neh4 and Neh5 domains have been shown to interact with the CH3 domains of CBP (CREB-binding protein) to facilitate the transactivation of Nrf2 target genes (Katoh et al. 2001; Zhu and Fahl 2001). Recently, however, a seventh Neh domain (Neh7, amino acids 209–316) was identified and shown to interact with the retinoic X receptor α, an Nrf2 repressor, and repress Nrf2 target gene transcription (Wang et al. 2013).  Keap1  Nrf2 is primarily regulated by Keap1, a substrate adaptor for a Cul3-containing E3 ubiquitin ligase. Keap1 possesses three functional domains, including a broad complex/tramtrack/bric-a-brac (BTB) domain, an intervening region (IVR), and a Kelch domain, also known as the double glycine repeat (DGR) domain (Fig. 1B). The BTB domain binds Cul3 and is required for Keap1 dimerization (Zipper and Mulcahy 2002; Lo et al. 2006). The Kelch/DGR domain is critical for maintaining the interaction between Nrf2 and Keap1 by interacting with the Neh2 domain of Nrf2 (Itoh et al. 1999; McMahon et al. 2004). The IVR links the BTB and Kelch/DGR domains and contains several cysteine residues that have been proposed to regulate Keap1 activity (Kobayashi et al. 2004; Ogura et al. 2010). Thus, each of the three domains is thought to play a unique role in mediating Nrf2 ubiquitination and repression.  Under basal conditions, Nrf2 is primarily localized in a complex with Keap1 via direct protein–protein interactions between the Keap1 Kelch domain and the ETGE and DLG motifs on the Neh2 domain of Nrf2 (Fig. 2). Keap1 has been shown to bind to the ETGE motif with a higher affinity than to the DLG motif (McMahon et al. 2006; Tong et al. 2006a). Based on these observations, a two-site substrate recognition hinge-and-latch model describing the interaction between Nrf2 and Keap1 was developed (McMahon et al. 2006; Tong et al. 2006a,b). The model suggests that Keap1 recruits Nrf2 via the ETGE motif (hinge), and once this interaction has been established, the DLG motif (latch) docks onto an adjacent unoccupied Kelch repeat domain on Keap1. Two Keap1 molecules position the seven ubiquitin-accepting lysine residues that are located between the DLG and ETGE motifs of Nrf2 in a favorable position and promote Nrf2 polyubiquitination and its subsequent proteasomal degradation by the 26S proteasome (Zhang et al. 2004; McMahon et al. 2006; Tong et al. 2007). Therefore, the Keap1–Cul3–E3 ubiquitin ligase complex tightly regulates Nrf2 protein to maintain it at a low level. Conversely, recent evidence from our laboratory demonstrated that USP15 (ubiquitin-specific peptidase 15), a deubiquitinating enzyme, also plays an important role in mediating the ubiquitination and degradation of Nrf2. USP15 deubiquitinates Keap1, stabilizes the Keap1–Cul3–E3 ligase complex, and enhances its E3 ligase activity, which ultimately leads to the degradation of Nrf2 (Villeneuve et al. 2013).  Figure 2. Figure 2. Schematic model of the Nrf2–Keap1 signaling pathway. Under basal conditions, Keap1 binds to the ETGE and DLG motifs on Nrf2 and brings Nrf2 into Keap1–Cul3–E3 ubiquitin ligase complex, leading to ubiquitination and subsequent degradation ... In response to a diverse array of stimuli, it has been proposed that critical cysteine residues, especially Cys151, within Keap1 can be covalently modified, allowing Nrf2 to evade Keap1-mediated ubiquitination (Fig. 2). The human Keap1 protein contains 27 cysteine residues that can be oxidized to sulfenic acid, form disulfides, or be covalently adducted by electrophiles (Dinkova-Kostova et al. 2002; Zhang and Hannink 2003; McMahon et al. 2010). The modification of thiols on Keap1 is thought to alter its conformation and results in the release of Nrf2 from the low-affinity binding site (DLG motif); however, Nrf2 remains attached to Keap1 by the ETGE motif. These changes are thought to prevent Nrf2 ubiquitination (McMahon et al. 2006; Tong et al. 2006a,b). Consequently, Keap1 molecules become saturated with Nrf2 that is no longer targeted for degradation, and newly synthesized, free Nrf2 translocates to the nucleus. In the nucleus, Nrf2 dimerizes with members of the masculoaponeurotic fibrosarcoma (Maf) protein family that have been shown to facilitate the binding of Nrf2 to AREs located within the regulatory regions of a wide variety of genes involved in cytoprotection and metabolism (Itoh et al. 1997; Nguyen et al. 2000; Motohashi et al. 2004; Hirotsu et al. 2012). The ARE is a cis-acting DNA enhancer sequence with the consensus sequence 5′-RTGABnnnGCR-3′, where conserved nucleotides are in capitals, and the “n” represents any nucleotide (Rushmore et al. 1991; Wasserman and Fahl 1997; Hayes and McMahon 2001). The Nrf2–Maf heterodimer recruits transcriptional coactivators that promote the transcription of genes involved in (1) regulating the synthesis and metabolism of glutathione, such as glutamate–cysteine ligase catalytic subunit (GCS); (2) antioxidant proteins specializing in neutralizing reactive species such as glutathione peroxidase (GPX); (3) drug-metabolizing enzymes like UDP-glucuronosyl-transferase 1A1; (4) xenobiotic transporters, including multidrug resistance protein 1 (MRP1); and (5) numerous other stress response proteins (for review, see Zhang and Gordon 2004; Hayes et al. 2010). By inducing the expression of this battery of genes, Nrf2 is able to augment a wide range of cell defense processes, thereby enhancing the overall capacity of cells to detoxify potentially harmful entities. As such, the Nrf2–Keap1 pathway is generally considered a major cellular defense pathway.  Go to: The dual role of Nrf2 in cancer Tumor suppressor functions of Nrf2: ‘the good side of Nrf2’  Several studies using Nrf2 knockout mice (Nrf2−/−) show that Nrf2 protects against chemical carcinogen-induced tumor formation in the stomach, bladder, and skin. For example, Nrf2-null mice are more likely to develop gastric neoplasia after exposure to benzo(a)pyerene compared with wild-type mice (Ramos-Gomez et al. 2001). Higher tumor burdens were reported in the intestines of Nrf2-deficient mice challenged with azoxymethane followed by dextran sodium sulfate compared with wild-type mice (Osburn et al. 2007; Khor et al. 2008). In addition, Nrf2-deficient mice had a higher incidence of bladder tumors following exposure to N-nitrosobutyl(4-hydroxybutyl)amine (Fahey et al. 2002) as well as an increased incidence of skin tumors after exposure to 7,12-dimethylbenz(a)anthracene or 12-O-tetradecanoylphorbol-13-acetate, two potent carcinogens (Xu et al. 2006). The mechanism by which Nrf2 protects against chemical-induced carcinogenesis may be due in part to its ability to reduce the amount of reactive oxygen species (ROS) and DNA damage in cells (Hirayama et al. 2003; Morito et al. 2003).  Further evidence supporting the protective role of Nrf2 comes from studies with mice harboring a single-nucleotide polymorphism (SNP) in the promoter region of the mouse Nrf2 gene. Mice with this SNP have reduced expression of Nrf2 and are more susceptible to hyperoxia-induced lung damage (Cho et al. 2002). The human NRF2 gene also harbors a SNP in its promoter region (rs6721961) (Yamamoto et al. 2004; Marzec et al. 2007). Individuals with this SNP have significantly lower NRF2 messenger RNA (mRNA) levels and an increased risk of developing non-small-cell lung cancer (NSCLC) (Suzuki et al. 2013).  Oncogenic functions of Nrf2: ‘the dark side of Nrf2’  Although a wide body of evidence indicates that activation of Nrf2 protects against a variety of toxicants and diseases, the prolonged activation of Nrf2 has been shown to favor the progression of several types of cancers. Nrf2 has been shown to be constitutively elevated in lung, breast, head and neck, ovarian, and endometrial carcinomas (Singh et al. 2006; Shibata et al. 2008a,b; Wang et al. 2008b; Jiang et al. 2010; Kim et al. 2010; Solis et al. 2010; Zhang et al. 2010). The prognosis of patients with tumors expressing high levels of Nrf2 in the clinic is poor (Shibata et al. 2008a; Solis et al. 2010; Sasaki et al. 2012) partly due to Nrf2's ability to enhance cancer cell proliferation and promote chemoresistance and radioresistance. In addition, Nrf2 expression is induced during the course of drug resistance. Collectively, these studies suggest that Nrf2 contributes to both intrinsic and acquired chemoresistance (Shibata et al. 2008a,b; Singh et al. 2008; Solis et al. 2010; Zhang et al. 2010; Inoue et al. 2012).  Nrf2 and cancer cell proliferation The elevated levels of Nrf2 in cancer cells have been shown to promote cancer cell proliferation. Recently, Mitsuishi et al. (2012) demonstrated that Nrf2 is indeed important for the ability of A549 lung cancer cells to proliferate. They performed microarray analysis to identify Nrf2 target genes involved in cancer cell proliferation and identified several genes involved in the pentose phosphate pathway, including glucose-6-phosphate dehydrogenase (G6PD), phosphogluconate dehydrogenase (PGD), transketolase (TKT), and transaldolase 1 (TALDO1), which are responsible for nicotinamide adenine dinucleotide phosphate (NADPH) regeneration. In addition, other metabolic genes, including malic enzyme 1 (ME1), phosphoribosyl pyrophosphate amidotransferase (PPAT), methylenetetrahydrofolate dehydrogenase 2 (MTHFD2), and isocitrate dehydrogenase 1 (IDH1), were also identified as transcription targets of Nrf2. Nrf2 directly activated G6PD, PGD, TKT, TALDO1, ME1, and IDH1 by binding to their respective AREs. These proteins support glucose flux and generate purines, which are the building blocks of DNA and RNA and are important for accelerating proliferation in cancer cells.  Nrf2 may also promote cancer cell proliferation and tumorigenesis by maintaining the redox balance and generating antioxidants in cancer cells. Recent studies have revealed that glutathione is critical for cell proliferation (Reddy et al. 2007a,b; Ishimoto et al. 2011). Mitsuishi et al. (2012) also found that, in A549 cells, the cellular levels of glutamine are elevated and that a substantial amount of the glutamine was used for the generation of glutathione. Therefore, the enhancement of glutathione synthesis is another important effect of Nrf2 in accelerating cancer cell proliferation.  Nrf2 and chemoresistance and radioresistance Several studies have demonstrated that cancer cells with elevated levels of Nrf2 are less sensitive to common chemotherapeutic agents such as etoposide, carboplatin, cisplatin, 5-fluorouracil, and doxorubicin (Singh et al. 2006; Ohta et al. 2008; Shibata et al. 2008a; Wang et al. 2008b; Homma et al. 2009; Jiang et al. 2010; Lister et al. 2011). Consistent with this notion, ectopic expression of Nrf2 in cancer cell lines that have low basal levels of Nrf2 renders them more resistant to a variety of anti-cancer agents, whereas siRNA-mediated inhibition of Nrf2 in cells with high levels of Nrf2 has been shown to reverse drug resistance (Shibata et al. 2008a; Wang et al. 2008b; Homma et al. 2009; Jiang et al. 2010; Lister et al. 2011). Furthermore, cells that have developed resistance to chemotherapeutic agents have also been shown to express high levels of Nrf2. For example, human ovarian cancer cells selected for resistance to doxorubicin have elevated levels of Nrf2 compared with wild-type cells, and depletion of Nrf2 restores drug sensitivity in the resistant cells (Shim et al. 2009). In vivo xenografts derived from Nrf2 silenced lung cancer cells were more susceptible to platin-based chemotherapeutics compared with xenografts with control cells (Singh et al. 2008; Jiang et al. 2010).  Constitutively high levels of Nrf2 have also been shown to protect cancer cells against ionizing radiation and confers radioresistance in NSCLC cells (Singh et al. 2010). A549 and H460 NSCLC cells transfected with Nrf2 siRNA were more susceptible to ionizing radiation, which was measured by an increase in protein oxidation and the total protein carbonyl content, than cells transfected with a control siRNA. Taken together, these findings suggest that Nrf2 plays a role in not only intrinsic chemoresistance but also acquired resistance in cancer cells and that selectively inhibiting Nrf2 may improve the clinical outcome of patients receiving chemotherapy or radiation therapy.  All of these findings suggest that Nrf2 has a dual role in cancer. Based on numerous studies, it appears that transient activation of Nrf2 in normal cells (where the Nrf2–Keap1 axis is intact) is protective; however, constitutive activation of Nrf2, as seen in cancer, enhances the survival and progression of cancer cells (Lau et al. 2008). A recent study supports this hypothesis. Satoh et al. (2013) found that Nrf2-deficient mice exposed to the carcinogen urethane exhibited a relative increase in tumor foci 8 wk after urethane administration. However, after 16 wk, tumors in Nrf2-null mice showed less advanced malignancy (Satoh et al. 2013).  Go to: Molecular basis of Nrf2 activation in cancer cells Several mechanisms have been reported for the increased activity of Nrf2 in cancers, including (1) somatic mutations in KEAP1, CUL3, or NRF2; (2) epigenetic silencing of Keap1; (3) aberrant accumulation of proteins that disrupt the interaction between Nrf2 and Keap1; (4) transcriptional up-regulation of NRF2 through oncogene-dependent signaling; and (5) modification of Keap1 by metabolic intermediates (for review, see Mitsuishi et al. 2013). A detailed description of each mechanism is discussed below.  Somatic mutations in KEAP1, NRF2, or CUL3  Gain-of-function mutations in NRF2 and loss-of-function mutations in KEAP1 and CUL3 have been identified in several human cancers (Padmanabhan et al. 2006; Singh et al. 2006; Nioi and Nguyen 2007; Ohta et al. 2008; Shibata et al. 2008a; Li et al. 2011; Ooi et al. 2013; Sato et al. 2013). Mutations in the KEAP1 gene were initially identified in two human lung adenocarcinoma cell lines (Padmanabhan et al. 2006). The mutations involved a glycine-to-cysteine substitution in the Kelch/DGR domain of Keap1. The mutant Keap1 exhibited reduced affinity to Nrf2, and, consequently, Nrf2 was constitutively activated in these cells. Since then, multiple somatic mutations have also been identified in the Kelch or IVR domains of the Keap1 protein in NSCLC cell lines and human tissues from NSCLC patients (Singh et al. 2006; Ohta et al. 2008). The decreased Nrf2/Keap1 interaction resulted in the constitutive activation of Nrf2 and induction of ARE-containing genes. KEAP1 gene mutations have also been identified in breast cancer cells (Sjoblom et al. 2006) and patients with gallbladder (Shibata et al. 2008a), liver (Shibata et al. 2008b), ovarian (Konstantinopoulos et al. 2008), endometrial (Wong et al. 2008), and lung papillary (Li et al. 2011) cancers. As more studies accessing the status of KEAP1 in different cancers are undertaken, it is likely that these studies will reveal that KEAP1 mutations occur frequently in many cancer types.  Although Nrf2 mutations occur less frequently than Keap1 mutations, mutations in the NRF2 gene have also been identified in several cancers, including lung, head and neck, and esophageal carcinoma (Shibata et al. 2008b, 2011; Kim et al. 2010). Shibata et al. (2008b) recently identified NRF2 somatic mutations in 11 of 103 patients with lung cancer and in three of 12 patients with head and neck cancers. Of the 14 NRF2 mutations described in the study, six were within the DLG motif and eight were within the ETGE motif (Shibata et al. 2008b). Mutations in this region impair the two-site substrate recognition of Keap1, resulting in the stabilization of Nrf2, increased nuclear translocation of Nrf2, and activation of Nrf2 target genes (for review, see Mitsuishi et al. 2013).  Recently, somatic mutations in CUL3 were identified in hereditary type 2 papillary renal cell carcinoma (Ooi et al. 2013). While CUL3 has a role in regulating the Nrf2/Keap1 pathway, CUL3 also has many other targets. Thus, mutation of CUL3 likely has many downstream effects, one of which is up-regulation of Nrf2. Interestingly, all of the KEAP1, NRF2, and CUL3 mutations identified thus far are somatic. No genomic amplification or deletion of NRF2 has been reported in cancer thus far.  Epigenetic silencing of Keap1  Epigenetic modifications in KEAP1 have also been shown to promote the accumulation of Nrf2. The promoter region of the KEAP1 gene is hypermethylated in lung (Wang et al. 2008a; Muscarella et al. 2011), prostate (Zhang et al. 2010), malignant glioma (Muscarella et al. 2011), and colorectal cancers (Hanada et al. 2012). The methylation of specific CpG sites within the promoter region of KEAP1 affects KEAP1 expression by inducing local chromatin remodeling and restricting the ability of the transcriptional machinery to bind to the necessary DNA sequences (for review, see Copple 2012). Therefore, hypermethylation inhibits KEAP1 gene expression, which results in the accumulation of Nrf2. These epigenetic modifications in KEAP1 confer a growth advantage in cancer cells. In fact, the epigenetic abnormalities in the KEAP1 gene in lung cancers and malignant gliomas have been associated with poor clinical prognoses in patients (Muscarella et al. 2011).  Several microRNAs (miRNAs) have been shown to be involved in the regulation of NRF2, including miR-144, miR-28, and miR-200a. miRNAs are short, single-stranded, noncoding RNAs that regulate gene expression by sequence-specific binding to mRNA that either inhibits translation or causes degradation of mRNA (Bartel 2004). miR-144 was the first miRNA to be fully characterized as a negative regulator of Nrf2 expression (Sangokoya et al. 2010). miR-144 decreased the expression of NRF2 and its target genes by targeting two distinct sites at positions 265–271 and 370–377 of the 3′ untranslated region of NRF2 (Sangokoya et al. 2010). Similarly, MCF-7 breast cancer cells ectopically expressing miR-28 exhibited lower levels of NRF2 mRNA and protein (Yang et al. 2011). miR-200a was shown to negatively regulate the stability of KEAP1 mRNA and the level of Keap1 protein, suggesting that miR-200a indirectly up-regulates Nrf2 by suppressing Keap1 levels (Eades et al. 2011). Supporting a role for these miRNAs in cancer, aberrant expression of miR-28 and miR-144 has been reported in various cancers, including lymphoma, glioma, and squamous cell carcinoma (Bryan et al. 2012).  Interactions between Nrf2 and other signaling pathways  A growing body of evidence suggests that in tumors, increased levels of Nrf2 can occur in the absence of genomic alterations in the NRF2 and KEAP1 genes. Numerous studies have revealed that a wide variety of proteins can activate the Nrf2–Keap1 pathway by altering Nrf2–Keap1 binding (for review, see Yates et al. 2009; Wakabayashi et al. 2010). In this section, we focus on recent evidence highlighting the interactions between the Nrf2–Keap1 complex and p21, p62, and other proteins (Fig. 3).  Figure 3. Figure 3. Cross-talk between Nrf2 and other proteins. The substrate adaptor sequestosome 1 protein (p62) modulates the Nrf2–Keap1 signaling pathway by directly interacting with Keap1. p62 sequesters Keap1 in autophagosomes, which results in a decrease in ... Nrf2 and p21  Studies have demonstrated that p53 negatively regulates Nrf2-mediated gene transcription (Faraonio et al. 2006). Specifically, p53 has been shown to suppress the transcription of x-ct, NQO1, and GST1, three Nrf2 target genes (Faraonio et al. 2006). In this study, various cell lines were transfected with expression vectors containing Nrf2 or p53. Cotransfection with Nrf2 and p53 resulted in decreased expression of the three target genes compared with cells transfected with Nrf2 alone. A report from our laboratory examining the relationship between p21 (a direct downstream target of p53) and Nrf2, however, demonstrated that the basal and inducible activity of Nrf2 is subject to positive regulation by p21 (Chen et al. 2009). p21 associates with the DLG motif within Nrf2, thereby disrupting the ability of Keap1 to properly bind and promote the ubiquitination of Nrf2. As a result, Nrf2 is stabilized in response to p21 up-regulation. Conversely, in the absence of p21, the basal and inducible expression of Nrf2 target genes is reduced in vitro and in vivo (Fig. 3). These findings may suggest that under unfavorable conditions, strong induction of p53 inhibits Nrf2 by an undefined mechanism, which reduces the antioxidant defense and cell survival pathways in order to promote cell death. On the other hand, under mild stress conditions, when p53 is only weakly induced, p21 is increased, stalling the cell cycle at the G1/S-phase checkpoint and eliciting the cytoprotective response to promote cell survival through (1) the induction of DNA damage repair functions of p53 and (2) activation of the Nrf2-mediated cytoprotective response.  Nrf2 and p62  Several laboratories, including ours, have demonstrated that sequestosome 1 protein (p62/SQSTM) can modulate the activity of Nrf2 (Copple et al. 2010; Jain et al. 2010; Komatsu et al. 2010; Lau et al. 2010). p62 is a scaffold protein that binds to polyubiquitinated proteins and targets protein aggregates and damaged organelles for degradation via the autophagy pathway. p62 contains an STGE-binding motif that is similar to the Nrf2 ETGE motif between amino acids 349 and 354 (Jain et al. 2010). p62 directly interacts with the Kelch domain of Keap1 via its STGE motif, thereby disrupting the proper confirmation of the Keap1–Nrf2 complex (Copple et al. 2010; Jain et al. 2010; Komatsu et al. 2010; Lau et al. 2010). p62 is also capable of binding to LC3, which is associated with the autophagosome membrane, thereby providing a link between the Nrf2–Keap1 pathway and the autophagy pathway (Komatsu et al. 2010). Our laboratory found that ectopic overexpression of p62 or blockage of autophagosomal flux sequestered Keap1 in autophagosomes via direct interaction between Keap1 and p62. The sequestration of Keap1 into autophagosomes resulted in a decrease in Nrf2 ubiquitination, an increase in Nrf2 stability, and, ultimately, the enhanced expression of ARE-bearing genes (Fig. 3). This aberrant accumulation of Nrf2 by autophagy deregulation is p62-dependent and Keap1-Cys151-independent, which we named the noncanonical mechanism of Nrf2 activation. The pathological importance of prolonged Nrf2 activation through the noncanonical mechanism is exemplified in studies with knockout mice showing that accumulation of Nrf2 is the major cause of liver damage in autophagy-deficient mice (Taguchi et al. 2012). Consistent with the notion that Keap1 is degraded through the autophagy–lysosome pathway (Taguchi et al. 2012), siRNA-mediated knockdown of p62 increased the half-life of Keap1 twofold, increased the levels of Keap1, lowered the level of Nrf2, and reduced the expression of Nrf2 target genes (Copple et al. 2010; Lau et al. 2010, 2013). This noncanonical mechanism of Nrf2 activation caused by autophagy deregulation adds another dimension to the regulation of the Nrf2–Keap1 pathway.  Recently identified proteins that interact with Keap1  The list of proteins that interact with Nrf2 or Keap1 and therefore modulate Nrf2 levels is continuously expanding. Recent studies revealed that the Wilms tumor gene on the X chromosome (WTX) tumor suppressor protein physically binds to Keap1 to competitively inhibit Nrf2 ubiquitination (Camp et al. 2012). PALB2 has also been shown to associate with Keap1 and sterically inhibit Nrf2 ubiquitination (Ma et al. 2012). Very recently, Hast et al. (2013) identified the protein dipeptidyl peptidase 3 (DPP3) that binds to Keap1, inhibits Nrf2 ubiquitination, and drives Nrf2-dependent transcription in cancer cells (Fig. 3). All of these proteins contain an ETGE motif, suggesting that certain ETGE-containing proteins are capable of up-regulating Nrf2 by competing with Nrf2 for Keap1 binding and suppressing Keap1-mediated ubiquitination of Nrf2. Therefore, the functional roles of these proteins may derive at least partially from activation of the Nrf2 pathway.  Transcriptional up-regulation of Nrf2 by oncogene-dependent signaling  Until recently, it was believed that Nrf2 activity is primarily regulated at the protein level through ubiquitination and degradation, based on the observations that a vast majority of Nrf2 activators enhance Nrf2 at the protein level without affecting the mRNA levels of NRF2. A recent study, however, demonstrated that Nrf2 expression might be constitutively elevated by oncogenic activation in cancer cells via transcriptional up-regulation of NRF2 (DeNicola et al. 2011). Activation of oncogenic alleles of KRAS, BRAF, and C-MYC (KRASG12D, BRAFV619E, and C-MYCERT12) increased the mRNA level of NRF2 and its target genes 60% and resulted in a more reduced intracellular redox environment. Furthermore, results from studies employing Jun and myc siRNA indicate that KRASG12D and BRAFV619E may up-regulate the transcription of NRF2 via Jun and Myc. However, more concrete mechanistic studies are needed to dissect the molecular mechanisms underlying the enhanced expression of NRF2 in response to oncogenic activation of KRAS, BRAF, and C-MYC.  Metabolic activation of Nrf2  Fumarate, a Kreb cycle metabolite, was recently shown to form adducts with Keap1 and activate the Nrf2 signaling pathway in the hereditary form of type 2 papillary renal cell cancer (Ooi et al. 2011). In the Krebs cycle, fumarate is normally metabolized to malate by the enzyme fumarate hydratase (FH). Homozygous FH loss-of-function mutations have been shown to cause the hereditary form of type 2 papillary renal carcinoma in humans (for review, see Eng et al. 2003). The loss of the FH enzyme activity causes the accumulation of high levels of fumarate in the affected cells. Mechanistic analysis revealed that fumarate modifies cysteine residues within Keap1, abrogating its ability to ubiquitinate Nrf2 and causing prolonged activation of Nrf2, which is likely to promote the progression of hereditary type 2 papillary renal cell carcinoma (Adam et al. 2011; Ooi et al. 2011).  Go to: Pharmacological modulators of the Nrf2–Keap1 signaling pathway The discovery of mechanisms underlying the regulation of the Nrf2/Keap1 pathway has led to the development of agents that can manipulate Nrf2 in order to obtain therapeutic benefit. Compounds that increase the activity of Nrf2 are being tested for disease prevention, whereas Nrf2 inhibitors are being developed as an adjuvant therapy to enhance the efficacy of chemotherapeutic drugs.  Nrf2 activators  Many Nrf2 activators are naturally occurring, plant-derived phytochemicals. Some examples of natural Nrf2 activators include sulforaphane (SF), curcumin, epigallocatechin-3-gallate, resveratrol, cafestol, kahweol, cinnamonyl-based compounds, zerumbone, garlic oganosulfur compounds, lycopene, and carnosol (for review, see Jeong et al. 2006; Lau et al. 2008; Kensler and Wakabayashi 2010). These chemopreventive compounds exert their effects by inducing the Nrf2-mediated defense response, including activation of phase II detoxification enzymes, antioxidants, and transporters that protect cells from subsequent carcinogenic insults.  One of the most extensively investigated natural products that target the Nrf2–Keap1 signaling pathway is SF, an isothiocyanate present in cruciferous vegetables such as broccoli (Kensler et al. 2000). In a mouse cancer model, topical application of SF protected mice from developing skin tumors after they were exposed to several different carcinogens (Xu et al. 2006). SF also inhibited tobacco-induced lung carcinogenesis (Conaway et al. 2005) and reduced the formation of aberrant crypt foci and pancreatic carcinogenesis following carcinogen exposure (Chung et al. 2000; Kuroiwa et al. 2006).  In humans, several cohort and case control studies have shown that there is an inverse correlation between broccoli consumption and the risk of developing colon, lung, breast, liver, and prostate cancer (Kensler et al. 1987, 2000, 2005; Spitz et al. 2000; Seow et al. 2002; Ambrosone et al. 2004; Joseph et al. 2004). Broccoli sprouts contain glucosinolates, which are the precursors of isothiocyanates such as SF. The most abundant glucosinolate found in broccoli is glucoraphanin, the precursor to SF. A phase II clinical trial study in China evaluating the chemopreventive properties of broccoli sprout preparations containing >60% glucoraphanin determined that SF reduced aflatoxin–DNA adducts in humans (Kensler et al. 2005). The success of these studies has led to several phase II clinical trials to study SF and broccoli sprouts as chemopreventive agents (http://www.clinicaltrials.gov).  Curcumin is another well-investigated chemopreventive natural product that is capable of activating Nrf2 (Balogun et al. 2003). Dietary administration of curcumin enhanced the expression of Nrf2 and blocked DNA adduct formation, oxidative stress, and inflammation in the livers and lungs of mice treated with benzo(a)pyrene (Garg et al. 2008; Thimmulappa et al. 2008).  Several synthetic Nrf2 activators have also been developed. Oltipraz (4-methyl-5-[2-pyrazinyl]-1,2-dithiole-3-thione) is a synthetic Nrf2 activator that has been shown to inhibit chemical-induced carcinogenesis in a variety of organs in rodents, including the colon, kidney, liver, stomach, and bladder (Kensler et al. 1987; Roebuck et al. 1991; Rao et al. 1993, 1996; Moon et al. 1994; Clapper et al. 1995; Nishikawa et al. 1998; Iida et al. 2004; Sharma et al. 2006). Oltipraz reduced the number of pulmonary adenomas, gastric cancer, and forestomach tumors in Nrf2 wild-type mice treated with benzo(a)pyrene but not in Nrf2-null mice, suggesting that Nrf2 is in fact important for its chemopreventive effects (Rao et al. 1993; Ramos-Gomez et al. 2003; Sharma et al. 2006). Oltipraz also inhibited N-nitrosobutyl (4-hydroxybutyl) amine (BBN)-induced urinary bladder cancer in an Nrf2-dependent manner (Iida et al. 2004). These promising preclinical studies led to phase I and II clinical trials in humans; however, the results are inconclusive. In a randomized, double-blind phase II clinical trial in China, 1 mo of weekly administration of 500 mg of oltipraz reduced the levels of a toxic hydroxylated metabolite of aflatoxin (Kensler et al. 1998a,b; Wang et al. 1999). However, another randomized, double-blind trial in smokers did not show significant differences in the amount of polyaromatic hydrocarbon DNA adduct levels in lung epithelial cells and blood among trial groups (Kelley et al. 2005). Side effects such as flatulence, gastrointestinal irritation, and paresthesia in fingertips stopped the evaluation of this agent.  Dimethyl fumarate (DMF) is another synthetic Nrf2 activator that has been shown to alkylate critical cysteine residues on Keap1, prevent Nrf2 ubiquitination, and promote Nrf2 stabilization and subsequent activation of Nrf2 target genes (Phillips and Fox 2013). In phase III clinical trials for multiple sclerosis (MS), BG-12, an oral preparation of DMF, led to a reduction in the annual relapse rate for MS (Gold et al. 2012). In addition, DMF protected heart tissues from ischemia–reperfusion injury (Ashrafian et al. 2013). As a result, BG-12 (Tecfidera) has been approved by the Food and Drug Administraion for the treatment of MS in March 2013 (http://www.fda.gov). Although the cancer-preventive properties of DMF in humans have not been tested, recent studies have also shown that DMF may have potential as an anti-cancer agent in several different types of cancer cells, including melanoma, head and neck carcinoma, glioblastoma, and colon cancer (for review, see Chen and Kirsner 2011).  Nrf2 inhibitors  Given the role of constitutive Nrf2 activation in promoting cancer progression and mediating resistance to cancer therapy, the pharmacological inhibition of Nrf2 signaling has recently emerged as a promising approach for cancer therapy, especially for cancers with elevated levels of Nrf2. Recently, our laboratory screened many plant extracts to identify Nrf2 inhibitors and found that brusatol, a component of Brucea javanica seeds, is able to inhibit the Nrf2 pathway (Ren et al. 2011). Brusatol decreased the protein levels of Nrf2 across a panel of mammalian cells at nanomolar concentrations and decreased the expression of Nrf2 target genes in cancer cells. Consequently, brusatol decreased the intracellular levels of glutathione by suppressing the expression of GCLM and GCLC, which encode for GCS, a glutathione synthesis enzyme. Additionally, brusatol decreased the intracellular concentration of cisplatin, presumably through enhanced expression of Mrp1, a drug efflux protein highly expressed in cancer cells. As a result, brusatol enhanced the cytotoxic effects of several chemotherapeutic agents both in vitro and in vivo. Currently, the molecular mechanism by which brusatol inhibits Nrf2 is under investigation.  Several other small molecules have been found to suppress the Nrf2 pathway, including ascorbic acid, luteolin, ochratoxin A, trigonelline, and all-trans retinoic acid (ATRA) (for review, see Magesh et al. 2012). ATRA and other RA receptor α (RARα) agonists have been shown to inhibit the basal and inducible activity of Nrf2 in vitro and in vivo (Wang et al. 2007). Mechanistically, in the presence of ATRA, Nrf2 forms a complex with RARα. The Nrf2:RARα complex is not able to bind to the ARE and thus decreases the ability of Nrf2 to activate ARE-driven genes. Intriguingly, although these compounds were reported to inhibit Nrf2, they also have been shown to act as Nrf2 activators in other studies. Thus, additional research is needed to establish the specificity and mechanism of action of these and other putative Nrf2 inhibitors before they can be used as therapeutics for cancer patients.  Future directions in the development of Nrf2 modulators  Given the dual role of Nrf2 in cancer, it is natural to wonder whether Nrf2 activation can lead to cancer. There have been a substantial number of studies assessing the ability of Nrf2 activators to promote cancer. There is no evidence to date suggesting that the agents used to activate the Nrf2 pathway have adverse impacts on tumor growth. Administration of oltipraz in rats, for example, did not affect hepatic tumor yield or burden, and similar results were seen with other Nrf2 activators (Maxuitenko et al. 1993). Therefore, transient activation of Nrf2 in cells with an intact Nrf2–Keap1 axis by pharmacological activators is safe for the purpose of chemoprevention.  One of the major concerns with the Nrf2 activators that are currently being used as pharmacological agents, however, is cytotoxicity due to off-target effects. Given the increased tendency for these chemopreventive agents to react with cysteines, multiple signaling pathways can conceivably be modulated. For example, chemopreventive agents such as CDDO-Im, SF, and dithiolethione have been shown to affect multiple pathways. Bardoxolone methyl, a CDDO derivative, was recently withdrawn from a phase III clinical trial in end stage renal disease patients with type II diabetes following an excess of unspecified serious adverse effects and mortality among the treated patients (Tayek and Kalantar-Zadeh 2013; Zhang 2013). The reactive site on bardoxolone can undergo Michael addition reactions readily with a range of nucleophiles. Thus, it is possible that the adverse events associated with bardoxolone are due to off-target events and unlikely to be due to the activation of the Nrf2–Keap1 signaling pathway.  Several studies have been published that will help improve the specificity of Nrf2-based therapies. The identification of the cocrystal structure of Keap1 in complex with the Neh2 domain of Nrf2, for example, will provide opportunities to design molecules that specifically and selectively interfere with the binding of Keap1 and Nrf2 (McMahon et al. 2006; Tong et al. 2006a). Small mimetics of the ETGE or DLG domains of Nrf2, for example, may serve this purpose. The demonstration that the expression of NRF2 and KEAP1 is also controlled by specific miRNAs provides additional targets to manipulate the Nrf2–Keap1 pathway. Furthermore, mounting evidence suggests that Nrf2 can cross-talk with other pathways important for cell survival (for review, see Wakabayashi et al. 2010). These converging points may offer great opportunities for pharmacological intervention to control the level of Nrf2 and its downstream effects.  Oxidative and mutagenic damage to DNA, proteins, and lipids occurs continuously as a result of exposure to environmental pollutants, ionizing radiation, endogenous reactive oxygen species (ROS), electrophilic intermediates of xenobiotic metabolism, and inflammation. Damage from these exogenous and endogenous insults, if not repaired, can contribute to various pathologies including cancer (1, 2). The transcription factor Nrf2 mediates gene induction of protective phase-2 detoxifying enzymes and its activation by thiol-active compounds, found in a variety of natural foods and dietary supplements, has been advocated as a means to prevent cancer (3, 4). By contrast, Nrf2-deficient mice are highly susceptible to chemically induced carcinogenesis of multiple organs (5, 6). Under normal, nonstressed circumstances, low cellular concentrations of Nrf2 are maintained by proteasomal degradation, through a Keap1-Cullin 3-Roc1-dependent mechanism in which Keap1 serves as the substrate adaptor subunit in the E3 holoenzyme. Activation of Nrf2 allows it to escape proteolysis and to rapidly accumulate in the nucleus where it stimulates gene activation (7–9). Hence, experimental strategies to target Keap1, including siRNA knockdown and small molecule modulation, have also been used as a means to activate Nrf2 (10, 11).  Enzymes that are encoded by Nrf2 target genes, such as NAD(P)H quinine oxidoreductase-1 (NQO1) and peroxiredoxins (PRDX), are cytoprotective under normal conditions. Paradoxically, these enzymes are increased in many cancers, thereby affording these cells with protective capabilities to environmental stresses, which is a hallmark of malignancy (12, 13). This observation implicates aberrant Nrf2 activation in the process of human cancer development. In this regard, recent studies have revealed genetic alterations in KEAP1 that can lead to uncontrolled Nrf2 activation in cancers of the lung (14–16). Although polymorphisms have been identified in the promoter region of the Nrf2 gene (17, 18), which are thought to predispose individuals to lung injury and oxidative related diseases, no conclusive findings on the occurrence of genetic variations in the coding region have yet been published. Given that the intermolecular association between Nrf2 and Keap1 is mandatory for Nrf2 degradation and repression, we hypothesized that subtle amino acid changes in the coding region of Nrf2 might also affect proper Keap1-Nrf2 interaction and hence Nrf2 turnover in cancer.  Previous Section Next Section Results NRF2 Mutation in Human Cancer. We sequenced the entire coding region of NRF2 in 85 cancer cell lines and found mutations in two lung cancer cell lines (LK2 and EBC1) and one oral cancer cell line (HO1-U1) [Fig. 1 A and B and supporting information (SI) Table S1]. We also examined 103 primary lung cancers of various histological subtypes and 12 primary head and neck (HN) cancers, and observed mutations in 11 lung tumors (10.7%) and 3 HN tumors (25%). All mutations (14/115, 12.2%) in primary tumors were missense amino acid substitutions and determined to be of somatic origin (Fig. 1A, Fig. S1 and Table S1), whereas no synonymous somatic alteration was detected. These mutations are conspicuously high in patients with a history of smoking (12/14) and occur frequently in squamous cell carcinoma (12/60, 20%). In lung cancer, no tumor harbored both NRF2 and KEAP1 mutations concurrently, and these alterations occur in tumors without EGFR mutation (Fig. 1A, Table 1, and Table S2). A homozygous Nrf2 mutant allele was observed in LK2 cells and one primary tumor (LK-40) and an increase in the gene copy number of NRF2 (3 copies) was also seen in LK2 cells (Fig. 1C). Prognostic analysis of lung squamous cell carcinomas carrying a mutation in NRF2 revealed a significant correlation with poor survival (P = 0.0247 by Log-rank analysis; Fig. 1D). Strikingly, the mutations were clustered in exon 2 and coded for amino acid changes in either the DLG or ETGE motifs of the regulatory Neh2 (Nrf2-ECH homology 2) domain (Fig. 1A) (19). Fig. 1. In a new window Download PPT Fig. 1. NRF2 mutations in human cancer. (A) Functional domains of Nrf2 protein showing distribution and types of NRF2 mutations. SQ, lung squamous cell carcinoma; AD, adenocarcinoma; LCNEC, large cell neuroendocrine carcinoma; HN, head and neck cancer; cell, lung cancer cell lines. (B) Representative DNA chromatograph of NRF2 mutations in human cancer cell lines with corresponding normal sequences in the bottom panel. (C) Copy number alteration map [shown by log2 ratio (tumor/normal)] of chromosome 2 in the tumors with mutated NRF2 and having lost the wild-type allele, detected by array-based comparative genomic hybridization. The arrow indicates the chromosomal location of NRF2. (D) Kaplan–Meier plot showing disease-free survival (DFS) of squamous cell lung carcinoma patients segregated according to the presence or absence of NRF2 mutations.  In this window In a new window Table 1. NRF2 mutation in cancer cell lines and cancer patients  NRF2 Mutations Impair the Two-site Substrate Recognition Mechanism of Keap1. The double glycine repeat and the C-terminal region of Keap1 (hereafter refer as to Keap1-DC) recognize both the DLG and ETGE motifs of Nrf2 (20, 21). To evaluate the effect that mutations in the Nrf2 protein may have on its physical association with Keap1, we used isothermal calorimetry (ITC) to probe the interaction between purified recombinant Neh2 protein mutants (either Δ1–33Neh2/ETGE mutants or full length Neh2 DLG mutants), and the Keap1-DC or the full-length Keap1 protein. All of the mutations that occur in the ETGE motif (77DxETGE82), such as D77V, E79Q or T80K (Fig. 2 A–C), compromised the association with Keap1-DC. In particular, the T80K mutation displayed no observable affinity. The wild-type Neh2 protein exhibited a biphasic-binding curve with Keap1 (Fig. 2D), which has been shown to be a dimer in solution (20), indicating that two binding sites (DLG and ETGE) in the Neh2 domain are capable of interacting with Keap1. However, mutants bearing the missense alterations W24C or L30F within the DLG motif (23LxxQDxDLG31) regress to monophasic binding patterns (Fig. 2 E and F). The residual binding affinity most likely is contributed by interaction with the ETGE motif alone. These data unequivocally demonstrate that Nrf2 mutations occurring in either the DLG or ETGE motif can affect binding to the Keap1 dimer. Fig. 2. In a new window Download PPT Fig. 2. NRF2 mutations impair the two-site substrate recognition mechanism of Keap1. ITC titration profiles of Keap1-DC with Neh2[Δ1–33,D77V] (A), Neh2[Δ1–33,E79Q] (B), Neh2[Δ1–33,T80K] (C), and of Keap1 with Neh2 (D), Neh2[W24C] (E), Neh2[L30F] (F), raw thermograms (upper graph) and fitted binding isotherms (lower graph). Stoichiometry (n) and association constant (Ka) are as indicated. (G–I), binding site of the human Keap1-DC and wild-type ETGE peptide structure (G) and modeled structures of Keap1-DC with E79Q (H) or E82D (I) mutant ETGE peptide. Stick representation showing side chains of interacting residues from Keap1 (aqua) and ETGE peptide (pink). Dotted lines indicate predicted hydrogen bonds. Figures were drawn with PyMOL (www.pymol.org).  Using the x-ray crystal structure of human Keap1-DC in association with the ETGE peptide as template (PDB code 2FLU) (22) and by means of the MODELLER 9v1 program (23), we modeled some of the amino acid substitutions found in cancer patients or cell lines to access their impact on the structure-function relationship of Keap1 with Nrf2. The model detailing the structure of the E79Q mutation (Figs. 1A and 2H) suggests that compared with wild-type protein, there is a loss of electrostatic interaction with R483 and S508 of Keap1 (Fig. 2G), which likely accounts for the 1,000-fold reduction in binding affinity with Keap1 seen earlier (Fig. 2B). Similarly, the E82D mutation (Fig. 1A and Fig. 2I), despite both being acidic residues, seems to eliminate the contact with S363, R380, and S508 of Keap1, possibly because of the shorter side chain of Asp.  NRF2 Mutations Disturb Proper Nrf2-Keap1 Binding and Inhibit Keap1-Mediated Degradation of Nrf2 In Vivo. We next examined the interaction between mutant Nrf2 and Keap1 in vivo. Three of the Nrf2 mutations that were detected in cancer biopsies were selected (E79K, T80K, and L30F) and the proteins expressed in 293T cells, together with Keap1. All three mutant proteins showed enhanced doublet bands (Fig. 3A, lanes 4–6). Immunoprecipitation analysis revealed that the E79K and T80K mutations both had substantially reduced ability to interact with Keap1 (Fig. 3B). Protein harboring the L30F mutation retained the ability to associate with Keap1 to a similar degree as wild type, presumably because of the presence of an intact ETGE motif, whereas the weaker binding DLG motif alone could not sufficiently support a stable Nrf2/Keap1 interaction. Fig. 3. In a new window Download PPT Fig. 3. NRF2 mutations disturb proper Nrf2-Keap1 binding, inhibit Keap1-mediated degradation, promote transcriptional activity, and enhance nuclear localization of Nrf2 in vivo. (A) Expression level of the transfected Flag-tagged Keap1 and Myc-tagged wild-type or mutant Nrf2 (E79K, T80K, and L30F) in 293T cells. (B) Keap1-Nrf2 association was evaluated by immunoprecipitation of transfected 293T cells using anti-Flag antibody and immunoblotted by anti-Myc antibody. (C) Ubiquitination efficiency of wild-type or mutant Myc-tagged Nrf2 in vivo. (D) Immunoblot (left) showing degradation rate of Nrf2 protein from 15 to 60 min after cycloheximide chase. Intensities (%) relative to time 0 are plotted (right). Data represent mean ± SD of three independent runs. Dotted line indicates 50% expression of control. (E) Transactivation activities of Nrf2 mutants. Data represent mean ± SD. of three independent runs (*, P < 0.001). (F) Rescue analysis in KEAP1- or NRF2-mutated cancer cells. H1650, A549, LK2, and EBC-1 cells bearing both wild-type KEAP1 and NRF2 or mutations in one of the genes are indicated. Relative expressions of Nrf2 target genes peroxiredoxin1 (PRDX1), multidrug resistance protein 3 (MRP3), and NAD(P)H quinine oxidoreductase-1 (NQO1) to β-ACTIN with or without cotransfected KEAP1 plasmid are shown. (G) Nuclear accumulation of mutant Nrf2 proteins when coexpressed with Keap1 (left). Ratio of cytoplasmic/nuclear localization of Nrf2 proteins was shown as bar chart (right).  Because Keap1 plays a central role in ubiquitination and proteasomal degradation of Nrf2, we proceeded to analyze the ubiquitination status of mutant Nrf2. Wild-type and mutant Nrf2 were coexpressed with Keap1 and hemagglutinin (HA)-tagged ubiquitin, and the levels of Nrf2 ubiquitination were determined after treatment with MG132, a proteasomal inhibitor. Wild-type Nrf2 was efficiently polyubiquitinated whereas mutant Nrf2 proteins were only weakly polyubiquitinated (Fig. 3C). In particular, the L30F mutant was strongly resistant to ubiquitination. We also investigated whether mutant Nrf2 proteins are more stable than wild-type Nrf2. Wild-type and mutant Nrf2 were expressed in 293T cells and treated with cycloheximide to prevent new protein synthesis. Wild-type Nrf2 protein decreased rapidly (t½ = 16 min) as reported (7). By contrast, mutant Nrf2 proteins were degraded more slowly, having half-lives of approximately twice that of wild type (t½ = 28–35 min; Fig. 3D).  NRF2 Mutations Promote Transcriptional Activity and Nuclear Localization of Nrf2. We next determined the transcriptional activation ability of mutant Nrf2 by analyzing luciferase activity from a reporter plasmid carrying a promoter region antioxidant-responsive element (ARE), the canonical Nrf2 recognition motif. Mutant Nrf2 proteins were significantly more active (P < 0.001 by unpaired t test) than wild-type Nrf2 (Fig. 3E). Coexpression of Keap1 markedly decreased luciferase activity induced by wild-type Nrf2 whereas the activity of mutant Nrf2 proteins was resistant to Keap1 inhibition (Fig. 3E). We also tested this reporter construct in cancer cell lines that carry genetic alterations in the Nrf2-Keap1 pathway (A549, LK2, and EBC1) and found that endogenous Nrf2 mutants also exhibited transactivation activity that was resistant to suppression by Keap1 coexpression, especially in LK2 cells, which carry a homozygous L79K mutation (Fig. S2). We further analyzed the endogenous expression of various Nrf2 target genes (NQO1, PRDX1 and MRP3) (15). The expression of these target genes was increased in cells with either altered KEAP1 (A549) or NRF2 (LK2 and EBC1), but remained at low basal levels in a cell line that had functional Nrf2 regulation (H1650). Introduction of Keap1 cDNA decreased expression of these genes in KEAP1 mutated A549 cells but failed to accomplish a similar down-regulation in cancer cells with mutated NRF2 (Fig. 3F). Immunofluorescent analysis also revealed that mutant Nrf2 proteins preferentially accumulated in the nucleus compared with wild type when coexpressed with Keap1 (Fig. 3G). Down-regulation of NRF2 Restores Sensitivity to Oxidative Stress and Chemotherapeutic Agent. Finally, we examined the effects of down-regulating Nrf2 in cancer cell lines where the Nrf2-pathway had become deregulated. Suppression of Nrf2 protein production by short interfering RNA (siRNA) reduced the expression of Nrf2-target genes (Fig. 4A). Because invasive cancer cells frequently encounter various oxidative stresses in their microenvironment (24), we examined whether activating mutations to Nrf2 affords these cells with resistance to oxidative stress. NRF2- or KEAP1-mutated cancer cells exhibited resistance to hydrogen peroxide treatment, and Nrf2 down-regulation in these cells significantly reduced cell viability (≈1.6-fold decrease in both cells) to a level similar to what was observed in Nrf2-pathway intact cancer cells (H1650) (Fig. 4B). Nrf2 downstream targets include enzymes for metabolizing (such as PRDX1) and exporting (MRP3) anti-cancer agents. Therefore, we also tested the effect of Nrf2-siRNA on drug sensitivity. We chose to examine the cytotoxic effect of cisplatin, a drug that is routinely used for treating lung cancers. Nrf2 down-regulation, in cancer cells harboring genetic alterations in the Nrf2-Keap1 pathway, significantly increased sensitivity to this compound (Fig. 4C). Fig. 4. In a new window Download PPT Fig. 4. Down-regulation of NRF2 restores sensitivity to oxidative stress and chemotherapeutic agent. (A) Down-regulation of Nrf2 expression by siRNA. Nonsilencing control or NRF2 siRNA was transfected into cancer cells with either KEAP1 (A549) or NRF2 (EBC-1) mutation. Cell lysates were electrophoresed and immunoblotted with anti-Nrf2 or anti-β-actin (loading control) antibody (top). Relative expressions of Nrf2 target genes (PRDX1, MRP3, and NQO1) to β-ACTIN are shown in the histogram (bottom). (B) Cell viability of siRNA treated A549, EBC-1, and H1650 cells exposed to oxidative stress. Data present percentage of cell viability after a 5-h treatment with H2O2 relative to nontreated control (mean ± SD of three independent runs) (C) A549 and EBC-1 cells were treated with control or NRF2 siRNA and various concentrations of cisplatin. Data present percentage of cell viability after 48 h relative to vehicle (DMSO)-treated control (mean ± SD of three independent runs). (*, P < 0.05; **, P < 0.01; ***, P < 0.001)  Previous Section Next Section Discussion Nrf2 degradation is governed by the Cullin3-Keap1 system, which is unique in terms of the adaptor-substrate recognition whereby the two Neh2 recognition motifs (DLG and ETGE) of a Nrf2 monomer bind to a Keap1 homodimer with different affinities; known as the two-site substrate recognition/hinge and latch model (Fig. S3). Unlike some other substrates of proteasomal degradation system, such as inhibitory kappa B (IKB) in inflammatory response and hypoxia-inducible transcription factor (HIF), which have to be post-translationally modified before recruiting to their respective E3 ligase complexes (25), Nrf2 is being constitutively recognized by Keap1 through negative-charged residues of two conserved degradation signal sequences in the substrate. Therefore, the structural integrity of both the Hinge (ETGE) and Latch (DLG) within the Neh2 domain is crucial for maintaining contemporaneous two-site binding for proper Nrf2 turnover and the narrow window of Nrf2-ARE mediated gene expression (21, 26). X-ray crystal structures of the Keap1-DC-DLG peptide and Keap1-DC-ETGE-peptide complexes have depicted that the side chains of Q26, D27, and D29 of the DLG motif (23LxxQDxDLG31) and E79 and E82 of the ETGE motif (77DxETGE82) have direct interactions with the Keap1-DC. Whereas T80 has both inter- and intramolecular interactions, Q75 has intramolecular contact (14, 21, 22). Cancer-related mutations within the ETGE motif are primarily substitutions of these essential residues for Keap1 recognition whereas mutations in the DLG motif are either changes in the conserved (D29 and L30) or nonconserved residues (W24, I28, V32, and R34). These findings suggest that the weaker Keap1-binding DLG region is more vulnerable to structural changes and fits into its anticipated role of the “latch” for the Nrf2-repression gate in the stress response. Collectively the NRF2 somatic mutations found in human cancers indubitably validate the necessity for simultaneous recognition of these two N-terminal degrons in Nrf2 for its timely proteolysis and regulation in vivo.  The correlation of mutation in the ubiquitin-proteasome pathway with cancer growth has long been recognized (27, 28). Altered degradation pattern of transcription factors and tumor suppressor proteins are found in cancers of different pathologies and origin, where oncogenic changes often arise from mutations in the subunits of E3 ligases. For example, von Hippel-Lindau tumor suppressor protein (pVHL) and BRAC1, together with BARD1 (BRAC1-associated RING domain 1), are components of E3 ubiquitin ligases (29–31). VHL ubiquitin ligase complex facilitates protein turnover of HIFα for angiogenesis or hypoxic response regulation, whereas BRAC1/BARD1 system targets a variety of substrates that involve in regulation of transcription, chromatin remodeling, DNA repair, and centrosome dynamics (32, 33). Mutations of pVHL or BRAC1 predispose tumorigenesis in patients probably by either enhancing adaptive capability of cancer cells to their microenvironment or by jeopardizing proper transcriptional regulations, DNA repair response, and genome stability, respectively (27, 28, 33). Similarly, recent studies have shown different somatic mutations of Keap1 (14–16), the substrate binding module of a Cullin3-based E3 ligase, in lung cancer patients that led to aberrant expression of ARE genes. Cancer-related somatic mutations in the degradation signals of Nrf2, presented in this study, for E3 ligase recognition again reinforce the importance of functional ubiquitin-proteasome system.  Additionally our results demonstrated that gain-of-function mutations of NRF2 were observed in approximately 11% of lung cancer. Together with previous reports (14–16), constitutive activation of Nrf2 (caused by KEAP1 or NRF2 mutations) occurs in >20% of lung cancer cases. Our analysis concurrently revealed frequent NRF2 mutation in head and neck cancers, suggesting perturbations of the Nrf2 pathway in a broad range of cancers, particularly those arising in tissues exposed to oxygen.  Intriguingly KEAP1-deficient mice, which show constitutive activation of Nrf2 in the whole body, develop postnatal hyperkeratosis without accompanying by increased cell proliferation in skin and upper digestive tract (34). Thus, preferential NRF2 mutations among patients diagnosed with squamous cell carcinoma and a history of smoking could be explained by stress-related Nrf2-dependent intervention to squamous cell lineage differentiation. However, malignant development induced by deviated Nrf2-mediated gene expression profile appears to be more complicated because a number of genes involved in cell proliferation were found to be transcriptionally modulated by Nrf2 through glutathione as the effector (35) and the growth rate of immortalized Keap1 null MEF cells also increased on culture dishes (16). Nevertheless, the existence of two tumors with both NRF2 alleles mutated and the association of poor prognosis suggest a direct role of Nrf2 mutation in tumor progression. In terms of stress cellular defense, temporary induction of cytoprotective enzymes by Nrf2 activation is beneficial to the repair of oxidative damage and cancer prevention before the accumulation of any genetic lesions at the early stage of carcinogenesis (36, 37). Although constitutive expression in a tumorigenic situation could be devastating because it could provide a survival advantage to invasive and metastatic cancer cells, such as the adaptation to microenvironment and the evolvement of chemoresistance in cancer cells that are known to occur in tumor hypoxia (38, 39). In addition, cross-talk between tumor hypoxia and induction of Nrf2 has also been suggested recently (40). It is also noteworthy that inhibition of the Nrf2 pathway in these cancer cells enhanced sensitivity to the anti-cancer reagent cisplatin. Taken together, our results set a new paradigm for cancer therapy in which the inhibition of Nrf2 could be used to enhance chemotherapeutic sensitivity. They further suggest that mutation in the Keap1-Nrf2 system may represent a novel oncogenic mechanism leading to malignancy (Fig. S3).  Previous Section Next Section Materials and Methods DNA Sample Collection. DNA was extracted from cell lines using a DNA extraction kit (Qiagen). All tumor samples were subjected to laser-capture microdissection using LM200 (Arcuturus). PCR Amplification and Genetic Analysis of NRF2. All coding exons of NRF2 were amplified by PCR using High Fidelity Taq polymerase (Roche) and appropriate primers (Table S3). All PCR products were purified and analyzed by sequencing. A high-density oligonucleotide array (Human genomic CGH microarray 44B, Agilent Technologies) was used. Isothermal Calorimetry. ITC titration experiments were performed at 25°C on a VP-ITC system (MicroCal Inc.). All proteins were prepared as described in ref. 21. All runs were performed in triplicate. Binding data were analyzed using the computer program Origin version 5.0, supplied by MicroCal Inc. Data processing was carried out as reported (21). Molecular and Cell Biology Analysis. Anti-Myc (Cell signaling), anti-Flag (Sigma), and anti-HA (MBL) antibodies, and MG132 (Calbiochem), cycloheximide (Nacalai), and cisplatin (WAKO) were purchased, whereas anti-Nrf2 polyclonal antibody was produced by immunization with Nrf2 peptide. Immunoprecipitation, immunoblotting, and immunofluorescent analyses were performed as described (7, 8) with slight modifications. Statistical Analysis. The Kaplan–Meier method was used to estimate the probability of disease-free survival. Log-rank analysis was used to assess the significance of the differences between tumors with and without NRF2 mutation. Long-term diazepam (DZ) treatment (5 mg/kg, i.p. for 15-30 consecutive days) reduced GABAergic activity in hypothalamus and cerebellum with the alteration of GABA synthesis and its release process. This long-term DZ-induced inhibition in GABAergic activity in hypothalamus and cerebellum was not altered when DZ-treated rats were exposed (2 h/day) to lower environmental temperature (LET; 12 degrees C) for 7-30 consecutive days. Although repeated exposure to LET did not affect the GABA system in cerebellum and hypothalamus, striatal GABAergic activity was significantly enhanced under similar conditions. This LET-induced (repeated exposure) stimulation in striatal GABAergic activity was prevented when rats were treated with DZ along with repeated LET exposure (2 h/day) under long-term conditions.\n",
      "348\n",
      "E-cadherin is involved in the formation of cell-junctions and the maintenance of epithelial integrity. Direct evidence of E-cadherin mutations triggering tumorigenesis has come from the finding of inactivating germline mutations of the gene (CDH1) in hereditary diffuse gastric cancer (HDGC). We screened a series of 66 young gastric cancer probands for germline CDH1 mutations, and two novel missense alterations together with an intronic variant were identified. We then analysed the functional significance of the two exonic missense variants found here as well as a third germline missense variant that we previously identified in a HGDC family. cDNAs encoding either the wild-type protein or mutant forms of E-cadherin were stably transfected into CHO (Chinese hamster ovary) E-cadherin-negative cells. Transfected cell-lines were characterized in terms of aggregation, motility and invasion. We show that a proportion of apparently sporadic early-onset diffuse gastric carcinomas are associated with germline alterations of the E-cadherin gene. We also demonstrate that a proportion of missense variants are associated with significant functional consequences, suggesting that our cell model can be used as an adjunct in deciding on the potential pathogenic role of identified E-cadherin germline alterations.\n"
     ]
    }
   ],
   "source": [
    "i = 0\n",
    "for x in data:\n",
    "    if(i < 10):\n",
    "        print x\n",
    "        print data[x]\n",
    "        i = i + 1"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
